 1 file changed, 20 insertions(+), 20 deletions(-)

diff --git 1.32/levels/skellige/gameplay/interior triggers/buildings.w2l.txt 4.00/levels/skellige/gameplay/interior triggers/buildings.w2l.txt
index a6a8b45..4cbefdd 100644
--- 1.32/levels/skellige/gameplay/interior triggers/buildings.w2l.txt	
+++ 4.00/levels/skellige/gameplay/interior triggers/buildings.w2l.txt	
@@ -2124,7 +2124,7 @@
             boundingBox (Box)
                 Min (Vector)
                     X (Float) : 2580.3520
-                    Y (Float) : 339.3699
+                    Y (Float) : 339.3700
                     Z (Float) : 25.6380
                     W (Float) : 1.0000
                 Max (Vector)
@@ -2162,12 +2162,12 @@
             worldPoints (array:Vector)
                 0 (Vector)
                     X (Float) : 2580.3520
-                    Y (Float) : 339.3699
+                    Y (Float) : 339.3700
                     Z (Float) : 25.6380
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : 2619.4450
-                    Y (Float) : 339.3699
+                    Y (Float) : 339.3700
                     Z (Float) : 25.6380
                     W (Float) : 1.0000
                 2 (Vector)
@@ -2311,7 +2311,7 @@
             name (String) : "InteriorTrigger"
             boundingBox (Box)
                 Min (Vector)
-                    X (Float) : -230.6346
+                    X (Float) : -230.6347
                     Y (Float) : 730.6396
                     Z (Float) : 78.9982
                     W (Float) : 1.0000
@@ -2359,13 +2359,13 @@
                     W (Float) : 1.0000
             worldPoints (array:Vector)
                 0 (Vector)
-                    X (Float) : -230.6346
+                    X (Float) : -230.6347
                     Y (Float) : 805.5101
                     Z (Float) : 78.9982
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : -207.4363
-                    Y (Float) : 764.5454
+                    Y (Float) : 764.5453
                     Z (Float) : 78.9982
                     W (Float) : 1.0000
                 2 (Vector)
@@ -2374,7 +2374,7 @@
                     Z (Float) : 78.9982
                     W (Float) : 1.0000
                 3 (Vector)
-                    X (Float) : -165.9571
+                    X (Float) : -165.9570
                     Y (Float) : 730.6396
                     Z (Float) : 78.9982
                     W (Float) : 1.0000
@@ -2516,7 +2516,7 @@
                 Min (Vector)
                     X (Float) : -69.0156
                     Y (Float) : 686.3386
-                    Z (Float) : 28.2462
+                    Z (Float) : 28.2463
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : -34.0614
@@ -2569,7 +2569,7 @@
                 3 (Vector)
                     X (Float) : -69.0156
                     Y (Float) : 718.4253
-                    Z (Float) : 28.2462
+                    Z (Float) : 28.2463
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
     CR4InteriorAreaEntity #55 (CR4InteriorAreaEntity) : kaer_trolde_bridge_04
@@ -2981,12 +2981,12 @@
             boundingBox (Box)
                 Min (Vector)
                     X (Float) : -205.2606
-                    Y (Float) : 653.7970
+                    Y (Float) : 653.7971
                     Z (Float) : 88.7971
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : -152.8998
-                    Y (Float) : 696.3853
+                    Y (Float) : 696.3852
                     Z (Float) : 95.7746
                     W (Float) : 1.0000
             height (Float) : 1.0000
@@ -3024,7 +3024,7 @@
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : -161.1438
-                    Y (Float) : 653.7970
+                    Y (Float) : 653.7971
                     Z (Float) : 88.7971
                     W (Float) : 1.0000
                 2 (Vector)
@@ -3034,7 +3034,7 @@
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : -197.0166
-                    Y (Float) : 696.3853
+                    Y (Float) : 696.3852
                     Z (Float) : 88.7971
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
@@ -3443,7 +3443,7 @@
             boundingBox (Box)
                 Min (Vector)
                     X (Float) : -196.2641
-                    Y (Float) : 697.4785
+                    Y (Float) : 697.4786
                     Z (Float) : 82.9381
                     W (Float) : 1.0000
                 Max (Vector)
@@ -3481,7 +3481,7 @@
             worldPoints (array:Vector)
                 0 (Vector)
                     X (Float) : -189.1241
-                    Y (Float) : 697.4785
+                    Y (Float) : 697.4786
                     Z (Float) : 83.3799
                     W (Float) : 1.0000
                 1 (Vector)
@@ -3496,7 +3496,7 @@
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : -196.2641
-                    Y (Float) : 723.4333
+                    Y (Float) : 723.4332
                     Z (Float) : 84.9753
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
@@ -3727,7 +3727,7 @@
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : -176.9323
-                    Y (Float) : 697.8374
+                    Y (Float) : 697.8373
                     Z (Float) : 88.4232
                     W (Float) : 1.0000
             height (Float) : 1.0000
@@ -3775,7 +3775,7 @@
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : -199.9992
-                    Y (Float) : 697.8374
+                    Y (Float) : 697.8373
                     Z (Float) : 81.2924
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
@@ -3859,7 +3859,7 @@
                 1 (Vector)
                     X (Float) : 137.6453
                     Y (Float) : 434.8105
-                    Z (Float) : 12.6101
+                    Z (Float) : 12.6100
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : 86.8845
@@ -3869,7 +3869,7 @@
                 3 (Vector)
                     X (Float) : 39.2732
                     Y (Float) : 495.2424
-                    Z (Float) : 3.7282
+                    Z (Float) : 3.7283
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
             isDarkPlace (Bool) : true
