 1 file changed, 73 insertions(+), 73 deletions(-)

diff --git 1.32/levels/skellige/gameplay/interior triggers/caves.w2l.txt 4.00/levels/skellige/gameplay/interior triggers/caves.w2l.txt
index e8ff857..738ec40 100644
--- 1.32/levels/skellige/gameplay/interior triggers/caves.w2l.txt	
+++ 4.00/levels/skellige/gameplay/interior triggers/caves.w2l.txt	
@@ -291,7 +291,7 @@
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : -345.2827
-                    Y (Float) : -753.4220
+                    Y (Float) : -753.4221
                     Z (Float) : 46.7844
                     W (Float) : 1.0000
                 2 (Vector)
@@ -311,7 +311,7 @@
                     W (Float) : 1.0000
                 5 (Vector)
                     X (Float) : -243.3719
-                    Y (Float) : -767.4664
+                    Y (Float) : -767.4663
                     Z (Float) : 46.7844
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 2596 bytes
@@ -448,12 +448,12 @@
             boundingBox (Box)
                 Min (Vector)
                     X (Float) : 133.0743
-                    Y (Float) : 630.9476
+                    Y (Float) : 630.9477
                     Z (Float) : 86.5903
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : 245.0149
-                    Y (Float) : 710.7476
+                    Y (Float) : 710.7475
                     Z (Float) : 101.2263
                     W (Float) : 1.0000
             height (Float) : 1.0000
@@ -491,7 +491,7 @@
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : 230.6373
-                    Y (Float) : 630.9476
+                    Y (Float) : 630.9477
                     Z (Float) : 86.5903
                     W (Float) : 1.0000
                 2 (Vector)
@@ -501,7 +501,7 @@
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : 147.4519
-                    Y (Float) : 710.7476
+                    Y (Float) : 710.7475
                     Z (Float) : 86.5903
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
@@ -962,7 +962,7 @@
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : -65.3019
-                    Y (Float) : 374.1596
+                    Y (Float) : 374.1597
                     Z (Float) : -8.5273
                     W (Float) : 1.0000
                 3 (Vector)
@@ -1109,7 +1109,7 @@
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : 202.0530
-                    Y (Float) : 698.8967
+                    Y (Float) : 698.8966
                     Z (Float) : 30.4864
                     W (Float) : 1.0000
             height (Float) : 1.0000
@@ -1147,17 +1147,17 @@
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : 202.0530
-                    Y (Float) : 645.8063
+                    Y (Float) : 645.8064
                     Z (Float) : 23.0879
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : 182.5498
-                    Y (Float) : 698.8967
+                    Y (Float) : 698.8966
                     Z (Float) : 22.4437
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : 87.7194
-                    Y (Float) : 664.0644
+                    Y (Float) : 664.0643
                     Z (Float) : 22.8196
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
@@ -1247,7 +1247,7 @@
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : 1197.6980
-                    Y (Float) : -715.2501
+                    Y (Float) : -715.2500
                     Z (Float) : -1.3174
                     W (Float) : 1.0000
                 3 (Vector)
@@ -1295,12 +1295,12 @@
                 Min (Vector)
                     X (Float) : 825.0605
                     Y (Float) : -347.6413
-                    Z (Float) : 6.4959
+                    Z (Float) : 6.4960
                     W (Float) : 1.0000
                 Max (Vector)
-                    X (Float) : 899.5201
+                    X (Float) : 899.5200
                     Y (Float) : -301.3622
-                    Z (Float) : 40.5334
+                    Z (Float) : 40.5333
                     W (Float) : 1.0000
             height (Float) : 1.0000
             color (Color)
@@ -1336,19 +1336,19 @@
                     Z (Float) : 24.7369
                     W (Float) : 1.0000
                 1 (Vector)
-                    X (Float) : 899.5201
+                    X (Float) : 899.5200
                     Y (Float) : -347.6413
                     Z (Float) : 18.4445
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : 899.5200
                     Y (Float) : -303.0481
-                    Z (Float) : 6.4959
+                    Z (Float) : 6.4960
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : 825.0605
                     Y (Float) : -301.3622
-                    Z (Float) : 12.7883
+                    Z (Float) : 12.7884
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
             isDarkPlace (Bool) : true
@@ -1576,12 +1576,12 @@
             boundingBox (Box)
                 Min (Vector)
                     X (Float) : -1691.3290
-                    Y (Float) : -880.5601
+                    Y (Float) : -880.5600
                     Z (Float) : 70.0587
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : -1609.5170
-                    Y (Float) : -798.7485
+                    Y (Float) : -798.7486
                     Z (Float) : 78.4376
                     W (Float) : 1.0000
             height (Float) : 1.0000
@@ -1619,7 +1619,7 @@
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : -1649.1580
-                    Y (Float) : -880.5601
+                    Y (Float) : -880.5600
                     Z (Float) : 70.0587
                     W (Float) : 1.0000
                 2 (Vector)
@@ -1629,7 +1629,7 @@
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : -1651.6880
-                    Y (Float) : -798.7485
+                    Y (Float) : -798.7486
                     Z (Float) : 70.0587
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
@@ -1671,7 +1671,7 @@
                 Min (Vector)
                     X (Float) : -118.8084
                     Y (Float) : 670.4722
-                    Z (Float) : 55.0298
+                    Z (Float) : 55.0299
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : -0.1778
@@ -1709,7 +1709,7 @@
                 0 (Vector)
                     X (Float) : -39.3959
                     Y (Float) : 670.4722
-                    Z (Float) : 55.0298
+                    Z (Float) : 55.0299
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : -0.1778
@@ -1723,7 +1723,7 @@
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : -118.8084
-                    Y (Float) : 803.3318
+                    Y (Float) : 803.3317
                     Z (Float) : 90.6295
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
@@ -1816,7 +1816,7 @@
                     Z (Float) : 86.7012
                     W (Float) : 1.0000
                 3 (Vector)
-                    X (Float) : -193.0897
+                    X (Float) : -193.0898
                     Y (Float) : -568.8419
                     Z (Float) : 86.7012
                     W (Float) : 1.0000
@@ -1954,12 +1954,12 @@
             boundingBox (Box)
                 Min (Vector)
                     X (Float) : -242.0173
-                    Y (Float) : -736.3970
+                    Y (Float) : -736.3969
                     Z (Float) : 71.7106
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : -126.2479
-                    Y (Float) : -623.5952
+                    Y (Float) : -623.5953
                     Z (Float) : 84.6242
                     W (Float) : 1.0000
             height (Float) : 1.0000
@@ -1997,7 +1997,7 @@
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : -166.0522
-                    Y (Float) : -736.3970
+                    Y (Float) : -736.3969
                     Z (Float) : 71.7106
                     W (Float) : 1.0000
                 2 (Vector)
@@ -2006,8 +2006,8 @@
                     Z (Float) : 71.7106
                     W (Float) : 1.0000
                 3 (Vector)
-                    X (Float) : -202.2129
-                    Y (Float) : -623.5952
+                    X (Float) : -202.2130
+                    Y (Float) : -623.5953
                     Z (Float) : 71.7106
                     W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
@@ -2563,7 +2563,7 @@
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : 1357.8760
-                    Y (Float) : 1837.5310
+                    Y (Float) : 1837.5320
                     Z (Float) : 6.8019
                     W (Float) : 1.0000
                 3 (Vector)
@@ -3414,7 +3414,7 @@
                     Z (Float) : 74.7205
                     W (Float) : 1.0000
                 1 (Vector)
-                    X (Float) : 37.6067
+                    X (Float) : 37.6066
                     Y (Float) : -390.9381
                     Z (Float) : 74.7205
                     W (Float) : 1.0000
@@ -3610,7 +3610,7 @@
                     W (Float) : 1.0000
             worldPoints (array:Vector)
                 0 (Vector)
-                    X (Float) : 171.5660
+                    X (Float) : 171.5661
                     Y (Float) : 593.6775
                     Z (Float) : 15.7467
                     W (Float) : 1.0000
@@ -4084,7 +4084,7 @@
             worldPoints (array:Vector)
                 0 (Vector)
                     X (Float) : -2247.0490
-                    Y (Float) : -495.7883
+                    Y (Float) : -495.7884
                     Z (Float) : 35.2014
                     W (Float) : 1.0000
                 1 (Vector)
@@ -5076,7 +5076,7 @@
                 Max (Vector)
                     X (Float) : 554.5954
                     Y (Float) : 807.3019
-                    Z (Float) : 207.8402
+                    Z (Float) : 207.8401
                     W (Float) : 1.0000
             height (Float) : 1.0000
             color (Color)
@@ -5114,7 +5114,7 @@
                 1 (Vector)
                     X (Float) : 554.5954
                     Y (Float) : 799.3956
-                    Z (Float) : 205.8402
+                    Z (Float) : 205.8401
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : 554.4574
@@ -6102,9 +6102,9 @@
                     Z (Float) : 33.1777
                     W (Float) : 1.0000
                 Max (Vector)
-                    X (Float) : 321.2925
+                    X (Float) : 321.2924
                     Y (Float) : 691.0508
-                    Z (Float) : 88.7920
+                    Z (Float) : 88.7919
                     W (Float) : 1.0000
             height (Float) : 1.0000
             color (Color)
@@ -6140,12 +6140,12 @@
                     Z (Float) : 33.1777
                     W (Float) : 1.0000
                 1 (Vector)
-                    X (Float) : 321.2925
+                    X (Float) : 321.2924
                     Y (Float) : 581.1519
                     Z (Float) : 33.1777
                     W (Float) : 1.0000
                 2 (Vector)
-                    X (Float) : 321.2925
+                    X (Float) : 321.2924
                     Y (Float) : 691.0508
                     Z (Float) : 33.1777
                     W (Float) : 1.0000
@@ -6289,7 +6289,7 @@
                 Min (Vector)
                     X (Float) : -13.9379
                     Y (Float) : -1142.2010
-                    Z (Float) : -60.6583
+                    Z (Float) : -60.6582
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : 100.8206
@@ -6327,7 +6327,7 @@
                 0 (Vector)
                     X (Float) : 45.6167
                     Y (Float) : -1142.2010
-                    Z (Float) : -60.6583
+                    Z (Float) : -60.6582
                     W (Float) : 1.0000
                 1 (Vector)
                     X (Float) : 100.8206
@@ -6337,13 +6337,13 @@
                 2 (Vector)
                     X (Float) : 41.2660
                     Y (Float) : -1027.4430
-                    Z (Float) : -48.8215
-                    W (Float) : 1.0000
-                3 (Vector)
-                    X (Float) : -13.9379
-                    Y (Float) : -1082.6480
                     Z (Float) : -48.8216
                     W (Float) : 1.0000
+                3 (Vector)
+                    X (Float) : -13.9379
+                    Y (Float) : -1082.6480
+                    Z (Float) : -48.8215
+                    W (Float) : 1.0000
             Bufferdata (CByteArray2) : 1299 bytes
             isDarkPlace (Bool) : true
             movementLock (EPlayerMovementLockType) : PMLT_Free
@@ -6530,7 +6530,7 @@
                     Z (Float) : -58.1374
                     W (Float) : 1.0000
                 3 (Vector)
-                    X (Float) : 113.4116
+                    X (Float) : 113.4115
                     Y (Float) : -1112.7800
                     Z (Float) : -58.1374
                     W (Float) : 1.0000
@@ -6577,7 +6577,7 @@
                     Z (Float) : -57.7861
                     W (Float) : 1.0000
                 Max (Vector)
-                    X (Float) : 370.8576
+                    X (Float) : 370.8575
                     Y (Float) : -1107.9370
                     Z (Float) : -41.1066
                     W (Float) : 1.0000
@@ -6615,17 +6615,17 @@
                     Z (Float) : -57.7861
                     W (Float) : 1.0000
                 1 (Vector)
-                    X (Float) : 304.4999
+                    X (Float) : 304.5000
                     Y (Float) : -1399.0410
                     Z (Float) : -57.7861
                     W (Float) : 1.0000
                 2 (Vector)
-                    X (Float) : 370.8576
+                    X (Float) : 370.8575
                     Y (Float) : -1151.3900
                     Z (Float) : -57.7861
                     W (Float) : 1.0000
                 3 (Vector)
-                    X (Float) : 208.6898
+                    X (Float) : 208.6897
                     Y (Float) : -1107.9370
                     Z (Float) : -57.7861
                     W (Float) : 1.0000
@@ -6763,12 +6763,12 @@
             boundingBox (Box)
                 Min (Vector)
                     X (Float) : -1949.8490
-                    Y (Float) : -954.9885
+                    Y (Float) : -954.9884
                     Z (Float) : 19.1994
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : -1897.9500
-                    Y (Float) : -875.0291
+                    Y (Float) : -875.0292
                     Z (Float) : 30.7944
                     W (Float) : 1.0000
             height (Float) : 1.0000
@@ -6801,7 +6801,7 @@
             worldPoints (array:Vector)
                 0 (Vector)
                     X (Float) : -1916.6240
-                    Y (Float) : -954.9885
+                    Y (Float) : -954.9884
                     Z (Float) : 19.1994
                     W (Float) : 1.0000
                 1 (Vector)
@@ -6811,7 +6811,7 @@
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : -1931.1750
-                    Y (Float) : -875.0291
+                    Y (Float) : -875.0292
                     Z (Float) : 19.1994
                     W (Float) : 1.0000
                 3 (Vector)
@@ -7281,7 +7281,7 @@
                 2 (Vector)
                     X (Float) : 133.3969
                     Y (Float) : 478.3453
-                    Z (Float) : 18.6717
+                    Z (Float) : 18.6716
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : 74.7613
@@ -7327,11 +7327,11 @@
             boundingBox (Box)
                 Min (Vector)
                     X (Float) : 799.5140
-                    Y (Float) : 506.6263
+                    Y (Float) : 506.6264
                     Z (Float) : 61.6984
                     W (Float) : 1.0000
                 Max (Vector)
-                    X (Float) : 870.6880
+                    X (Float) : 870.6879
                     Y (Float) : 588.9446
                     Z (Float) : 83.0170
                     W (Float) : 1.0000
@@ -7365,11 +7365,11 @@
             worldPoints (array:Vector)
                 0 (Vector)
                     X (Float) : 830.3336
-                    Y (Float) : 506.6263
+                    Y (Float) : 506.6264
                     Z (Float) : 61.6984
                     W (Float) : 1.0000
                 1 (Vector)
-                    X (Float) : 870.6880
+                    X (Float) : 870.6879
                     Y (Float) : 549.6904
                     Z (Float) : 61.6984
                     W (Float) : 1.0000
@@ -7469,18 +7469,18 @@
                     W (Float) : 1.0000
             worldPoints (array:Vector)
                 0 (Vector)
-                    X (Float) : 660.4732
+                    X (Float) : 660.4731
                     Y (Float) : 661.7648
                     Z (Float) : 67.7803
                     W (Float) : 1.0000
                 1 (Vector)
-                    X (Float) : 644.8834
+                    X (Float) : 644.8833
                     Y (Float) : 653.8819
                     Z (Float) : 58.0698
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : 641.4481
-                    Y (Float) : 613.4763
+                    Y (Float) : 613.4762
                     Z (Float) : 66.1618
                     W (Float) : 1.0000
                 3 (Vector)
@@ -7672,7 +7672,7 @@
                     Z (Float) : 224.8170
                     W (Float) : 1.0000
                 1 (Vector)
-                    X (Float) : -206.1742
+                    X (Float) : -206.1743
                     Y (Float) : 915.4556
                     Z (Float) : 224.8170
                     W (Float) : 1.0000
@@ -7682,7 +7682,7 @@
                     Z (Float) : 224.8170
                     W (Float) : 1.0000
                 3 (Vector)
-                    X (Float) : -231.7658
+                    X (Float) : -231.7657
                     Y (Float) : 954.7484
                     Z (Float) : 224.8170
                     W (Float) : 1.0000
@@ -7822,7 +7822,7 @@
                     Z (Float) : 6.5716
                     W (Float) : 1.0000
                 Max (Vector)
-                    X (Float) : 519.3385
+                    X (Float) : 519.3384
                     Y (Float) : -1382.1990
                     Z (Float) : 14.8829
                     W (Float) : 1.0000
@@ -7865,7 +7865,7 @@
                     Z (Float) : 6.5716
                     W (Float) : 1.0000
                 2 (Vector)
-                    X (Float) : 519.3385
+                    X (Float) : 519.3384
                     Y (Float) : -1390.5660
                     Z (Float) : 6.5716
                     W (Float) : 1.0000
@@ -7911,7 +7911,7 @@
             name (String) : "InteriorTrigger"
             boundingBox (Box)
                 Min (Vector)
-                    X (Float) : 469.0407
+                    X (Float) : 469.0408
                     Y (Float) : -1391.2240
                     Z (Float) : 2.4570
                     W (Float) : 1.0000
@@ -7964,7 +7964,7 @@
                     Z (Float) : 2.4570
                     W (Float) : 1.0000
                 3 (Vector)
-                    X (Float) : 469.0407
+                    X (Float) : 469.0408
                     Y (Float) : -1379.0820
                     Z (Float) : 2.4570
                     W (Float) : 1.0000
@@ -8045,7 +8045,7 @@
             worldPoints (array:Vector)
                 0 (Vector)
                     X (Float) : -2007.1830
-                    Y (Float) : -938.4609
+                    Y (Float) : -938.4610
                     Z (Float) : 50.1396
                     W (Float) : 1.0000
                 1 (Vector)
@@ -8055,7 +8055,7 @@
                     W (Float) : 1.0000
                 2 (Vector)
                     X (Float) : -1937.1300
-                    Y (Float) : -905.4290
+                    Y (Float) : -905.4289
                     Z (Float) : 48.9479
                     W (Float) : 1.0000
                 3 (Vector)
