 1 file changed, 182 insertions(+), 182 deletions(-)

diff --git 1.32/dlc/bob/data/levels/bob/locations/barrens/hansa_on_swamps/elven ruins.w2l.txt 4.00/dlc/bob/data/levels/bob/locations/barrens/hansa_on_swamps/elven ruins.w2l.txt
index bba5968..7e83d2a 100644
--- 1.32/dlc/bob/data/levels/bob/locations/barrens/hansa_on_swamps/elven ruins.w2l.txt	
+++ 4.00/dlc/bob/data/levels/bob/locations/barrens/hansa_on_swamps/elven ruins.w2l.txt	
@@ -112,7 +112,7 @@
                                 W (Float) : 1.0000
                             Max (Vector)
                                 X (Float) : -110.3227
-                                Y (Float) : 532.4714
+                                Y (Float) : 532.4713
                                 Z (Float) : 5.3518
                                 W (Float) : 1.0000
                         drawableFlags (EDrawableFlags) : DF_IsVisible,DF_CastShadows
@@ -194,12 +194,12 @@
                         boundingBox (Box)
                             Min (Vector)
                                 X (Float) : -349.8533
-                                Y (Float) : 777.1591
+                                Y (Float) : 777.1592
                                 Z (Float) : -0.4994
                                 W (Float) : 1.0000
                             Max (Vector)
                                 X (Float) : -348.6660
-                                Y (Float) : 781.2275
+                                Y (Float) : 781.2276
                                 Z (Float) : 0.4443
                                 W (Float) : 1.0000
                         drawableFlags (EDrawableFlags) : DF_IsVisible,DF_CastShadows
@@ -484,9 +484,9 @@
                 22 (CSectorDataResource)
                     box0 (Float) : -0.6893
                     box1 (Float) : -0.4454
-                    box2 (Float) : 0.2227
-                    box3 (Float) : 0.6893
-                    box4 (Float) : -0.3067
+                    box2 (Float) : 0.2202
+                    box3 (Float) : 0.6894
+                    box4 (Float) : -0.3051
                     box5 (Float) : 3.7983
                     pathHash (String) : "dlc/bob/data/environment/architecture/elven_ruins/statue/bob_elven_statue_back_support.w2mesh"
                 23 (CSectorDataResource)
@@ -541,7 +541,7 @@
                     box0 (Float) : -5.1263
                     box1 (Float) : -5.6709
                     box2 (Float) : -3.8869
-                    box3 (Float) : 5.9261
+                    box3 (Float) : 5.9260
                     box4 (Float) : 5.5847
                     box5 (Float) : 1.9157
                     pathHash (String) : "dlc/bob/data/environment/entity_proxy/bob_decoration_set_destroyed_pier_proxy.w2mesh"
@@ -640,7 +640,7 @@
                     radius (Uint16) : 180
                     offset (Uint64) : 0
                     positionX (Float) : -261.3534
-                    positionY (Float) : 626.3169
+                    positionY (Float) : 626.3168
                     positionZ (Float) : 9.9652
                 1 (CSectorDataObject)
                     type (Uint8) : 2
@@ -648,7 +648,7 @@
                     radius (Uint16) : 180
                     offset (Uint64) : 64
                     positionX (Float) : -261.3534
-                    positionY (Float) : 626.3169
+                    positionY (Float) : 626.3168
                     positionZ (Float) : 9.9652
                 2 (CSectorDataObject)
                     type (Uint8) : 1
@@ -704,7 +704,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 560
                     positionX (Float) : -299.3296
-                    positionY (Float) : 608.3821
+                    positionY (Float) : 608.3822
                     positionZ (Float) : 0.4830
                 9 (CSectorDataObject)
                     type (Uint8) : 2
@@ -712,7 +712,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 624
                     positionX (Float) : -299.3296
-                    positionY (Float) : 608.3821
+                    positionY (Float) : 608.3822
                     positionZ (Float) : 0.4830
                 10 (CSectorDataObject)
                     type (Uint8) : 1
@@ -767,7 +767,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1120
-                    positionX (Float) : -233.2413
+                    positionX (Float) : -233.2412
                     positionY (Float) : 624.5997
                     positionZ (Float) : 0.0102
                 17 (CSectorDataObject)
@@ -775,7 +775,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1184
-                    positionX (Float) : -233.2413
+                    positionX (Float) : -233.2412
                     positionY (Float) : 624.5997
                     positionZ (Float) : 0.0102
                 18 (CSectorDataObject)
@@ -832,7 +832,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 1680
                     positionX (Float) : -257.0400
-                    positionY (Float) : 634.5922
+                    positionY (Float) : 634.5921
                     positionZ (Float) : 13.5151
                 25 (CSectorDataObject)
                     type (Uint8) : 2
@@ -840,7 +840,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 1744
                     positionX (Float) : -257.0400
-                    positionY (Float) : 634.5922
+                    positionY (Float) : 634.5921
                     positionZ (Float) : 13.5151
                 26 (CSectorDataObject)
                     type (Uint8) : 1
@@ -848,7 +848,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 1820
                     positionX (Float) : -261.6553
-                    positionY (Float) : 610.3101
+                    positionY (Float) : 610.3100
                     positionZ (Float) : 0.7729
                 27 (CSectorDataObject)
                     type (Uint8) : 2
@@ -856,7 +856,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 1884
                     positionX (Float) : -261.6553
-                    positionY (Float) : 610.3101
+                    positionY (Float) : 610.3100
                     positionZ (Float) : 0.7729
                 28 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1104,7 +1104,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 4060
                     positionX (Float) : -284.4871
-                    positionY (Float) : 640.6901
+                    positionY (Float) : 640.6900
                     positionZ (Float) : 0.2540
                 59 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1112,7 +1112,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 4124
                     positionX (Float) : -284.4871
-                    positionY (Float) : 640.6901
+                    positionY (Float) : 640.6900
                     positionZ (Float) : 0.2540
                 60 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1311,7 +1311,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5880
-                    positionX (Float) : -303.6499
+                    positionX (Float) : -303.6500
                     positionY (Float) : 639.5459
                     positionZ (Float) : 0.3610
                 85 (CSectorDataObject)
@@ -1319,7 +1319,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5944
-                    positionX (Float) : -303.6499
+                    positionX (Float) : -303.6500
                     positionY (Float) : 639.5459
                     positionZ (Float) : 0.3610
                 86 (CSectorDataObject)
@@ -1360,7 +1360,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 6300
                     positionX (Float) : -307.2653
-                    positionY (Float) : 612.8835
+                    positionY (Float) : 612.8834
                     positionZ (Float) : 0.4944
                 91 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1368,7 +1368,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 6364
                     positionX (Float) : -307.2653
-                    positionY (Float) : 612.8835
+                    positionY (Float) : 612.8834
                     positionZ (Float) : 0.4944
                 92 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1392,7 +1392,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 6580
                     positionX (Float) : -233.8830
-                    positionY (Float) : 618.1354
+                    positionY (Float) : 618.1355
                     positionZ (Float) : 0.1092
                 95 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1400,7 +1400,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 6644
                     positionX (Float) : -233.8830
-                    positionY (Float) : 618.1354
+                    positionY (Float) : 618.1355
                     positionZ (Float) : 0.1092
                 96 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1599,7 +1599,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 8400
-                    positionX (Float) : -256.0087
+                    positionX (Float) : -256.0088
                     positionY (Float) : 618.6364
                     positionZ (Float) : 12.5597
                 121 (CSectorDataObject)
@@ -1607,7 +1607,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 8464
-                    positionX (Float) : -256.0087
+                    positionX (Float) : -256.0088
                     positionY (Float) : 618.6364
                     positionZ (Float) : 12.5597
                 122 (CSectorDataObject)
@@ -1696,7 +1696,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 9240
                     positionX (Float) : -286.6601
-                    positionY (Float) : 629.4526
+                    positionY (Float) : 629.4525
                     positionZ (Float) : 0.3775
                 133 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1704,7 +1704,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 9304
                     positionX (Float) : -286.6601
-                    positionY (Float) : 629.4526
+                    positionY (Float) : 629.4525
                     positionZ (Float) : 0.3775
                 134 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1743,7 +1743,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 9660
-                    positionX (Float) : -269.3185
+                    positionX (Float) : -269.3186
                     positionY (Float) : 631.7773
                     positionZ (Float) : 13.4730
                 139 (CSectorDataObject)
@@ -1751,7 +1751,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 9724
-                    positionX (Float) : -269.3185
+                    positionX (Float) : -269.3186
                     positionY (Float) : 631.7773
                     positionZ (Float) : 13.4730
                 140 (CSectorDataObject)
@@ -1904,7 +1904,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 11060
                     positionX (Float) : -286.4737
-                    positionY (Float) : 643.5224
+                    positionY (Float) : 643.5223
                     positionZ (Float) : 0.1153
                 159 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1912,7 +1912,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 11124
                     positionX (Float) : -286.4737
-                    positionY (Float) : 643.5224
+                    positionY (Float) : 643.5223
                     positionZ (Float) : 0.1153
                 160 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1920,7 +1920,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 11200
                     positionX (Float) : -253.9743
-                    positionY (Float) : 624.1243
+                    positionY (Float) : 624.1242
                     positionZ (Float) : 13.0973
                 161 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1928,7 +1928,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 11264
                     positionX (Float) : -253.9743
-                    positionY (Float) : 624.1243
+                    positionY (Float) : 624.1242
                     positionZ (Float) : 13.0973
                 162 (CSectorDataObject)
                     type (Uint8) : 1
@@ -2144,7 +2144,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 13160
                     positionX (Float) : -266.0643
-                    positionY (Float) : 632.5014
+                    positionY (Float) : 632.5013
                     positionZ (Float) : 12.9729
                 189 (CSectorDataObject)
                     type (Uint8) : 2
@@ -2152,7 +2152,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 13224
                     positionX (Float) : -266.0643
-                    positionY (Float) : 632.5014
+                    positionY (Float) : 632.5013
                     positionZ (Float) : 12.9729
                 190 (CSectorDataObject)
                     type (Uint8) : 1
@@ -2207,7 +2207,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 13720
-                    positionX (Float) : -299.7741
+                    positionX (Float) : -299.7740
                     positionY (Float) : 650.0271
                     positionZ (Float) : -0.4617
                 197 (CSectorDataObject)
@@ -2215,7 +2215,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 13784
-                    positionX (Float) : -299.7741
+                    positionX (Float) : -299.7740
                     positionY (Float) : 650.0271
                     positionZ (Float) : -0.4617
                 198 (CSectorDataObject)
@@ -2256,7 +2256,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 14140
                     positionX (Float) : -290.2986
-                    positionY (Float) : 657.5317
+                    positionY (Float) : 657.5316
                     positionZ (Float) : -0.3058
                 203 (CSectorDataObject)
                     type (Uint8) : 2
@@ -2264,7 +2264,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 14204
                     positionX (Float) : -290.2986
-                    positionY (Float) : 657.5317
+                    positionY (Float) : 657.5316
                     positionZ (Float) : -0.3058
                 204 (CSectorDataObject)
                     type (Uint8) : 1
@@ -2351,7 +2351,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 14980
-                    positionX (Float) : -256.8064
+                    positionX (Float) : -256.8065
                     positionY (Float) : 631.7039
                     positionZ (Float) : 13.3429
                 215 (CSectorDataObject)
@@ -2359,7 +2359,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 15044
-                    positionX (Float) : -256.8064
+                    positionX (Float) : -256.8065
                     positionY (Float) : 631.7039
                     positionZ (Float) : 13.3429
                 216 (CSectorDataObject)
@@ -2368,7 +2368,7 @@
                     radius (Uint16) : 80
                     offset (Uint64) : 15120
                     positionX (Float) : -266.9592
-                    positionY (Float) : 630.2846
+                    positionY (Float) : 630.2845
                     positionZ (Float) : 13.2965
                 217 (CSectorDataObject)
                     type (Uint8) : 2
@@ -2376,7 +2376,7 @@
                     radius (Uint16) : 80
                     offset (Uint64) : 15184
                     positionX (Float) : -266.9592
-                    positionY (Float) : 630.2846
+                    positionY (Float) : 630.2845
                     positionZ (Float) : 13.2965
                 218 (CSectorDataObject)
                     type (Uint8) : 1
@@ -2511,7 +2511,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 16380
-                    positionX (Float) : -260.1616
+                    positionX (Float) : -260.1617
                     positionY (Float) : 609.4747
                     positionZ (Float) : 0.7602
                 235 (CSectorDataObject)
@@ -2519,7 +2519,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 16444
-                    positionX (Float) : -260.1616
+                    positionX (Float) : -260.1617
                     positionY (Float) : 609.4747
                     positionZ (Float) : 0.7602
                 236 (CSectorDataObject)
@@ -2560,7 +2560,7 @@
                     radius (Uint16) : 80
                     offset (Uint64) : 16800
                     positionX (Float) : -298.0627
-                    positionY (Float) : 650.3645
+                    positionY (Float) : 650.3644
                     positionZ (Float) : -0.4824
                 241 (CSectorDataObject)
                     type (Uint8) : 2
@@ -2568,7 +2568,7 @@
                     radius (Uint16) : 80
                     offset (Uint64) : 16864
                     positionX (Float) : -298.0627
-                    positionY (Float) : 650.3645
+                    positionY (Float) : 650.3644
                     positionZ (Float) : -0.4824
                 242 (CSectorDataObject)
                     type (Uint8) : 1
@@ -2768,7 +2768,7 @@
                     radius (Uint16) : 150
                     offset (Uint64) : 18620
                     positionX (Float) : -269.0073
-                    positionY (Float) : 629.3094
+                    positionY (Float) : 629.3093
                     positionZ (Float) : 9.8533
                 267 (CSectorDataObject)
                     type (Uint8) : 2
@@ -2776,7 +2776,7 @@
                     radius (Uint16) : 40
                     offset (Uint64) : 18684
                     positionX (Float) : -269.0073
-                    positionY (Float) : 629.3094
+                    positionY (Float) : 629.3093
                     positionZ (Float) : 9.8533
                 268 (CSectorDataObject)
                     type (Uint8) : 1
@@ -2833,7 +2833,7 @@
                     offset (Uint64) : 19180
                     positionX (Float) : -287.9154
                     positionY (Float) : 624.3058
-                    positionZ (Float) : -2.4363
+                    positionZ (Float) : -2.4362
                 275 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
@@ -2841,7 +2841,7 @@
                     offset (Uint64) : 19244
                     positionX (Float) : -287.9154
                     positionY (Float) : 624.3058
-                    positionZ (Float) : -2.4363
+                    positionZ (Float) : -2.4362
                 276 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2879,7 +2879,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 19600
-                    positionX (Float) : -284.8499
+                    positionX (Float) : -284.8498
                     positionY (Float) : 627.1667
                     positionZ (Float) : 3.6720
                 281 (CSectorDataObject)
@@ -2887,7 +2887,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 19664
-                    positionX (Float) : -284.8499
+                    positionX (Float) : -284.8498
                     positionY (Float) : 627.1667
                     positionZ (Float) : 3.6720
                 282 (CSectorDataObject)
@@ -2896,7 +2896,7 @@
                     radius (Uint16) : 140
                     offset (Uint64) : 19740
                     positionX (Float) : -304.0179
-                    positionY (Float) : 625.2908
+                    positionY (Float) : 625.2909
                     positionZ (Float) : 3.2541
                 283 (CSectorDataObject)
                     type (Uint8) : 2
@@ -2904,7 +2904,7 @@
                     radius (Uint16) : 80
                     offset (Uint64) : 19804
                     positionX (Float) : -304.0179
-                    positionY (Float) : 625.2908
+                    positionY (Float) : 625.2909
                     positionZ (Float) : 3.2541
                 284 (CSectorDataObject)
                     type (Uint8) : 1
@@ -2960,7 +2960,7 @@
                     radius (Uint16) : 170
                     offset (Uint64) : 20300
                     positionX (Float) : -262.1707
-                    positionY (Float) : 633.4230
+                    positionY (Float) : 633.4229
                     positionZ (Float) : 10.3710
                 291 (CSectorDataObject)
                     type (Uint8) : 2
@@ -2968,14 +2968,14 @@
                     radius (Uint16) : 170
                     offset (Uint64) : 20364
                     positionX (Float) : -262.1707
-                    positionY (Float) : 633.4230
+                    positionY (Float) : 633.4229
                     positionZ (Float) : 10.3710
                 292 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 20440
-                    positionX (Float) : -234.2706
+                    positionX (Float) : -234.2707
                     positionY (Float) : 625.9746
                     positionZ (Float) : 3.2788
                 293 (CSectorDataObject)
@@ -2983,7 +2983,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 20504
-                    positionX (Float) : -234.2706
+                    positionX (Float) : -234.2707
                     positionY (Float) : 625.9746
                     positionZ (Float) : 3.2788
                 294 (CSectorDataObject)
@@ -3247,16 +3247,16 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 22820
-                    positionX (Float) : -304.6277
-                    positionY (Float) : 629.9316
+                    positionX (Float) : -304.6276
+                    positionY (Float) : 629.9315
                     positionZ (Float) : 0.2999
                 327 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 22884
-                    positionX (Float) : -304.6277
-                    positionY (Float) : 629.9316
+                    positionX (Float) : -304.6276
+                    positionY (Float) : 629.9315
                     positionZ (Float) : 0.2999
                 328 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3312,7 +3312,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 23380
                     positionX (Float) : -259.7595
-                    positionY (Float) : 615.8832
+                    positionY (Float) : 615.8831
                     positionZ (Float) : 3.1343
                 335 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3320,7 +3320,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 23444
                     positionX (Float) : -259.7595
-                    positionY (Float) : 615.8832
+                    positionY (Float) : 615.8831
                     positionZ (Float) : 3.1343
                 336 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3376,7 +3376,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 23940
                     positionX (Float) : -257.7107
-                    positionY (Float) : 601.9672
+                    positionY (Float) : 601.9673
                     positionZ (Float) : 2.1351
                 343 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3384,7 +3384,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 24004
                     positionX (Float) : -257.7107
-                    positionY (Float) : 601.9672
+                    positionY (Float) : 601.9673
                     positionZ (Float) : 2.1351
                 344 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3456,7 +3456,7 @@
                     radius (Uint16) : 135
                     offset (Uint64) : 24640
                     positionX (Float) : -265.2823
-                    positionY (Float) : 618.0028
+                    positionY (Float) : 618.0029
                     positionZ (Float) : 9.0057
                 353 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3464,7 +3464,7 @@
                     radius (Uint16) : 135
                     offset (Uint64) : 24704
                     positionX (Float) : -265.2823
-                    positionY (Float) : 618.0028
+                    positionY (Float) : 618.0029
                     positionZ (Float) : 9.0057
                 354 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3504,7 +3504,7 @@
                     radius (Uint16) : 135
                     offset (Uint64) : 25060
                     positionX (Float) : -253.9595
-                    positionY (Float) : 631.7614
+                    positionY (Float) : 631.7615
                     positionZ (Float) : 9.8890
                 359 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3512,7 +3512,7 @@
                     radius (Uint16) : 135
                     offset (Uint64) : 25124
                     positionX (Float) : -253.9595
-                    positionY (Float) : 631.7614
+                    positionY (Float) : 631.7615
                     positionZ (Float) : 9.8890
                 360 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3584,7 +3584,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 25760
                     positionX (Float) : -231.0154
-                    positionY (Float) : 628.6985
+                    positionY (Float) : 628.6986
                     positionZ (Float) : -2.7937
                 369 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3592,7 +3592,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 25824
                     positionX (Float) : -231.0154
-                    positionY (Float) : 628.6985
+                    positionY (Float) : 628.6986
                     positionZ (Float) : -2.7937
                 370 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3744,7 +3744,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 27160
                     positionX (Float) : -270.1961
-                    positionY (Float) : 668.2652
+                    positionY (Float) : 668.2651
                     positionZ (Float) : -0.1639
                 389 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3752,14 +3752,14 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 27224
                     positionX (Float) : -270.1961
-                    positionY (Float) : 668.2652
+                    positionY (Float) : 668.2651
                     positionZ (Float) : -0.1639
                 390 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 150
                     offset (Uint64) : 27300
-                    positionX (Float) : -268.9691
+                    positionX (Float) : -268.9690
                     positionY (Float) : 632.3722
                     positionZ (Float) : 10.0527
                 391 (CSectorDataObject)
@@ -3767,7 +3767,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 40
                     offset (Uint64) : 27364
-                    positionX (Float) : -268.9691
+                    positionX (Float) : -268.9690
                     positionY (Float) : 632.3722
                     positionZ (Float) : 10.0527
                 392 (CSectorDataObject)
@@ -3792,7 +3792,7 @@
                     radius (Uint16) : 135
                     offset (Uint64) : 27580
                     positionX (Float) : -269.6505
-                    positionY (Float) : 630.0590
+                    positionY (Float) : 630.0589
                     positionZ (Float) : 9.8925
                 395 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3800,7 +3800,7 @@
                     radius (Uint16) : 135
                     offset (Uint64) : 27644
                     positionX (Float) : -269.6505
-                    positionY (Float) : 630.0590
+                    positionY (Float) : 630.0589
                     positionZ (Float) : 9.8925
                 396 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3808,7 +3808,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 27720
                     positionX (Float) : -254.5827
-                    positionY (Float) : 577.9363
+                    positionY (Float) : 577.9364
                     positionZ (Float) : -1.9266
                 397 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3816,7 +3816,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 27784
                     positionX (Float) : -254.5827
-                    positionY (Float) : 577.9363
+                    positionY (Float) : 577.9364
                     positionZ (Float) : -1.9266
                 398 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4032,7 +4032,7 @@
                     radius (Uint16) : 150
                     offset (Uint64) : 29680
                     positionX (Float) : -268.6905
-                    positionY (Float) : 620.0841
+                    positionY (Float) : 620.0842
                     positionZ (Float) : 9.1806
                 425 (CSectorDataObject)
                     type (Uint8) : 2
@@ -4040,7 +4040,7 @@
                     radius (Uint16) : 40
                     offset (Uint64) : 29744
                     positionX (Float) : -268.6905
-                    positionY (Float) : 620.0841
+                    positionY (Float) : 620.0842
                     positionZ (Float) : 9.1806
                 426 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4079,7 +4079,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 30100
-                    positionX (Float) : -261.1650
+                    positionX (Float) : -261.1649
                     positionY (Float) : 609.3081
                     positionZ (Float) : 6.7132
                 431 (CSectorDataObject)
@@ -4087,7 +4087,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 30164
-                    positionX (Float) : -261.1650
+                    positionX (Float) : -261.1649
                     positionY (Float) : 609.3081
                     positionZ (Float) : 6.7132
                 432 (CSectorDataObject)
@@ -4128,7 +4128,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 30520
                     positionX (Float) : -258.2847
-                    positionY (Float) : 619.6047
+                    positionY (Float) : 619.6046
                     positionZ (Float) : 6.6207
                 437 (CSectorDataObject)
                     type (Uint8) : 2
@@ -4136,7 +4136,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 30584
                     positionX (Float) : -258.2847
-                    positionY (Float) : 619.6047
+                    positionY (Float) : 619.6046
                     positionZ (Float) : 6.6207
                 438 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4208,7 +4208,7 @@
                     radius (Uint16) : 150
                     offset (Uint64) : 31220
                     positionX (Float) : -255.1455
-                    positionY (Float) : 630.0250
+                    positionY (Float) : 630.0251
                     positionZ (Float) : 9.7988
                 447 (CSectorDataObject)
                     type (Uint8) : 2
@@ -4216,7 +4216,7 @@
                     radius (Uint16) : 40
                     offset (Uint64) : 31284
                     positionX (Float) : -255.1455
-                    positionY (Float) : 630.0250
+                    positionY (Float) : 630.0251
                     positionZ (Float) : 9.7988
                 448 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4224,7 +4224,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 31360
                     positionX (Float) : -229.5336
-                    positionY (Float) : 626.4795
+                    positionY (Float) : 626.4794
                     positionZ (Float) : 1.3247
                 449 (CSectorDataObject)
                     type (Uint8) : 2
@@ -4232,7 +4232,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 31424
                     positionX (Float) : -229.5336
-                    positionY (Float) : 626.4795
+                    positionY (Float) : 626.4794
                     positionZ (Float) : 1.3247
                 450 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4256,7 +4256,7 @@
                     radius (Uint16) : 150
                     offset (Uint64) : 31640
                     positionX (Float) : -269.5508
-                    positionY (Float) : 630.9884
+                    positionY (Float) : 630.9883
                     positionZ (Float) : 9.9667
                 453 (CSectorDataObject)
                     type (Uint8) : 2
@@ -4264,7 +4264,7 @@
                     radius (Uint16) : 40
                     offset (Uint64) : 31704
                     positionX (Float) : -269.5508
-                    positionY (Float) : 630.9884
+                    positionY (Float) : 630.9883
                     positionZ (Float) : 9.9667
                 454 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4368,7 +4368,7 @@
                     radius (Uint16) : 140
                     offset (Uint64) : 32620
                     positionX (Float) : -264.3075
-                    positionY (Float) : 645.2887
+                    positionY (Float) : 645.2888
                     positionZ (Float) : 3.3888
                 467 (CSectorDataObject)
                     type (Uint8) : 2
@@ -4376,7 +4376,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 32684
                     positionX (Float) : -264.3075
-                    positionY (Float) : 645.2887
+                    positionY (Float) : 645.2888
                     positionZ (Float) : 3.3888
                 468 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4415,7 +4415,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 33040
-                    positionX (Float) : -278.9707
+                    positionX (Float) : -278.9706
                     positionY (Float) : 600.1776
                     positionZ (Float) : 0.8822
                 473 (CSectorDataObject)
@@ -4423,7 +4423,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 33104
-                    positionX (Float) : -278.9707
+                    positionX (Float) : -278.9706
                     positionY (Float) : 600.1776
                     positionZ (Float) : 0.8822
                 474 (CSectorDataObject)
@@ -4431,7 +4431,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 33180
-                    positionX (Float) : -285.0469
+                    positionX (Float) : -285.0468
                     positionY (Float) : 641.9937
                     positionZ (Float) : 0.1387
                 475 (CSectorDataObject)
@@ -4439,7 +4439,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 33244
-                    positionX (Float) : -285.0469
+                    positionX (Float) : -285.0468
                     positionY (Float) : 641.9937
                     positionZ (Float) : 0.1387
                 476 (CSectorDataObject)
@@ -4992,7 +4992,7 @@
                     radius (Uint16) : 60
                     offset (Uint64) : 38080
                     positionX (Float) : -264.9063
-                    positionY (Float) : 637.3076
+                    positionY (Float) : 637.3075
                     positionZ (Float) : 7.4188
                 545 (CSectorDataObject)
                     type (Uint8) : 2
@@ -5000,14 +5000,14 @@
                     radius (Uint16) : 60
                     offset (Uint64) : 38144
                     positionX (Float) : -264.9063
-                    positionY (Float) : 637.3076
+                    positionY (Float) : 637.3075
                     positionZ (Float) : 7.4188
                 546 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 38220
-                    positionX (Float) : -257.3836
+                    positionX (Float) : -257.3837
                     positionY (Float) : 615.0662
                     positionZ (Float) : 5.7846
                 547 (CSectorDataObject)
@@ -5015,7 +5015,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 38284
-                    positionX (Float) : -257.3836
+                    positionX (Float) : -257.3837
                     positionY (Float) : 615.0662
                     positionZ (Float) : 5.7846
                 548 (CSectorDataObject)
@@ -5103,33 +5103,33 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 39060
-                    positionX (Float) : -267.8805
-                    positionY (Float) : 627.8960
-                    positionZ (Float) : 15.5867
+                    positionX (Float) : -267.8795
+                    positionY (Float) : 627.8958
+                    positionZ (Float) : 15.5850
                 559 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 39124
-                    positionX (Float) : -267.8805
-                    positionY (Float) : 627.8960
-                    positionZ (Float) : 15.5867
+                    positionX (Float) : -267.8795
+                    positionY (Float) : 627.8958
+                    positionZ (Float) : 15.5850
                 560 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 39200
-                    positionX (Float) : -262.1245
-                    positionY (Float) : 621.1458
-                    positionZ (Float) : 12.6202
+                    positionX (Float) : -262.1255
+                    positionY (Float) : 621.1444
+                    positionZ (Float) : 12.6213
                 561 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 39264
-                    positionX (Float) : -262.1245
-                    positionY (Float) : 621.1458
-                    positionZ (Float) : 12.6202
+                    positionX (Float) : -262.1255
+                    positionY (Float) : 621.1444
+                    positionZ (Float) : 12.6213
                 562 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -5167,17 +5167,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 39620
-                    positionX (Float) : -254.6542
-                    positionY (Float) : 623.9823
-                    positionZ (Float) : 15.2351
+                    positionX (Float) : -254.6552
+                    positionY (Float) : 623.9827
+                    positionZ (Float) : 15.2335
                 567 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 39684
-                    positionX (Float) : -254.6542
-                    positionY (Float) : 623.9823
-                    positionZ (Float) : 15.2351
+                    positionX (Float) : -254.6552
+                    positionY (Float) : 623.9827
+                    positionZ (Float) : 15.2335
                 568 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -5199,65 +5199,65 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 39900
-                    positionX (Float) : -259.2840
-                    positionY (Float) : 632.5423
-                    positionZ (Float) : 15.8490
+                    positionX (Float) : -259.2844
+                    positionY (Float) : 632.5414
+                    positionZ (Float) : 15.8472
                 571 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 39964
-                    positionX (Float) : -259.2840
-                    positionY (Float) : 632.5423
-                    positionZ (Float) : 15.8490
+                    positionX (Float) : -259.2844
+                    positionY (Float) : 632.5414
+                    positionZ (Float) : 15.8472
                 572 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 40040
-                    positionX (Float) : -257.9699
-                    positionY (Float) : 619.8098
-                    positionZ (Float) : 14.9871
+                    positionX (Float) : -257.9703
+                    positionY (Float) : 619.8108
+                    positionZ (Float) : 14.9855
                 573 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 40104
-                    positionX (Float) : -257.9699
-                    positionY (Float) : 619.8098
-                    positionZ (Float) : 14.9871
+                    positionX (Float) : -257.9703
+                    positionY (Float) : 619.8108
+                    positionZ (Float) : 14.9855
                 574 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 40180
-                    positionX (Float) : -264.5421
-                    positionY (Float) : 632.0027
-                    positionZ (Float) : 15.8453
+                    positionX (Float) : -264.5417
+                    positionY (Float) : 632.0020
+                    positionZ (Float) : 15.8435
                 575 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 40244
-                    positionX (Float) : -264.5421
-                    positionY (Float) : 632.0027
-                    positionZ (Float) : 15.8453
+                    positionX (Float) : -264.5417
+                    positionY (Float) : 632.0020
+                    positionZ (Float) : 15.8435
                 576 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 40320
-                    positionX (Float) : -255.2495
-                    positionY (Float) : 629.1746
-                    positionZ (Float) : 15.5780
+                    positionX (Float) : -255.2504
+                    positionY (Float) : 629.1743
+                    positionZ (Float) : 15.5764
                 577 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 40384
-                    positionX (Float) : -255.2495
-                    positionY (Float) : 629.1746
-                    positionZ (Float) : 15.5780
+                    positionX (Float) : -255.2504
+                    positionY (Float) : 629.1743
+                    positionZ (Float) : 15.5764
                 578 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -5279,7 +5279,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 40600
-                    positionX (Float) : -264.4241
+                    positionX (Float) : -264.4240
                     positionY (Float) : 631.6512
                     positionZ (Float) : 15.4215
                 581 (CSectorDataObject)
@@ -5287,7 +5287,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 40664
-                    positionX (Float) : -264.4241
+                    positionX (Float) : -264.4240
                     positionY (Float) : 631.6512
                     positionZ (Float) : 15.4215
                 582 (CSectorDataObject)
@@ -5344,7 +5344,7 @@
                     radius (Uint16) : 315
                     offset (Uint64) : 41160
                     positionX (Float) : -273.9149
-                    positionY (Float) : 631.9185
+                    positionY (Float) : 631.9186
                     positionZ (Float) : 6.0504
                 589 (CSectorDataObject)
                     type (Uint8) : 2
@@ -5352,7 +5352,7 @@
                     radius (Uint16) : 315
                     offset (Uint64) : 41224
                     positionX (Float) : -273.9149
-                    positionY (Float) : 631.9185
+                    positionY (Float) : 631.9186
                     positionZ (Float) : 6.0504
                 590 (CSectorDataObject)
                     type (Uint8) : 1
@@ -5408,7 +5408,7 @@
                     radius (Uint16) : 300
                     offset (Uint64) : 41720
                     positionX (Float) : -254.4599
-                    positionY (Float) : 625.4322
+                    positionY (Float) : 625.4323
                     positionZ (Float) : 31.3436
                 597 (CSectorDataObject)
                     type (Uint8) : 2
@@ -5416,7 +5416,7 @@
                     radius (Uint16) : 300
                     offset (Uint64) : 41784
                     positionX (Float) : -254.4599
-                    positionY (Float) : 625.4322
+                    positionY (Float) : 625.4323
                     positionZ (Float) : 31.3436
                 598 (CSectorDataObject)
                     type (Uint8) : 1
@@ -5504,7 +5504,7 @@
                     radius (Uint16) : 315
                     offset (Uint64) : 42560
                     positionX (Float) : -272.0853
-                    positionY (Float) : 618.0937
+                    positionY (Float) : 618.0936
                     positionZ (Float) : 3.7418
                 609 (CSectorDataObject)
                     type (Uint8) : 2
@@ -5512,14 +5512,14 @@
                     radius (Uint16) : 315
                     offset (Uint64) : 42624
                     positionX (Float) : -272.0853
-                    positionY (Float) : 618.0937
+                    positionY (Float) : 618.0936
                     positionZ (Float) : 3.7418
                 610 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 300
                     offset (Uint64) : 42700
-                    positionX (Float) : -266.5297
+                    positionX (Float) : -266.5296
                     positionY (Float) : 629.9087
                     positionZ (Float) : 19.9545
                 611 (CSectorDataObject)
@@ -5527,7 +5527,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 300
                     offset (Uint64) : 42764
-                    positionX (Float) : -266.5297
+                    positionX (Float) : -266.5296
                     positionY (Float) : 629.9087
                     positionZ (Float) : 19.9545
                 612 (CSectorDataObject)
@@ -5648,7 +5648,7 @@
                     radius (Uint16) : 315
                     offset (Uint64) : 43820
                     positionX (Float) : -239.1320
-                    positionY (Float) : 640.2071
+                    positionY (Float) : 640.2072
                     positionZ (Float) : -6.6920
                 627 (CSectorDataObject)
                     type (Uint8) : 2
@@ -5656,7 +5656,7 @@
                     radius (Uint16) : 315
                     offset (Uint64) : 43884
                     positionX (Float) : -239.1320
-                    positionY (Float) : 640.2071
+                    positionY (Float) : 640.2072
                     positionZ (Float) : -6.6920
                 628 (CSectorDataObject)
                     type (Uint8) : 1
@@ -5744,7 +5744,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 44660
                     positionX (Float) : -184.2154
-                    positionY (Float) : 564.9380
+                    positionY (Float) : 564.9381
                     positionZ (Float) : 6.9844
                 639 (CSectorDataObject)
                     type (Uint8) : 2
@@ -5752,7 +5752,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 44724
                     positionX (Float) : -184.2154
-                    positionY (Float) : 564.9380
+                    positionY (Float) : 564.9381
                     positionZ (Float) : 6.9844
                 640 (CSectorDataObject)
                     type (Uint8) : 1
@@ -6015,7 +6015,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 47040
-                    positionX (Float) : -288.1796
+                    positionX (Float) : -288.1797
                     positionY (Float) : 562.0573
                     positionZ (Float) : -2.3761
                 673 (CSectorDataObject)
@@ -6023,7 +6023,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 47104
-                    positionX (Float) : -288.1796
+                    positionX (Float) : -288.1797
                     positionY (Float) : 562.0573
                     positionZ (Float) : -2.3761
                 674 (CSectorDataObject)
@@ -6159,7 +6159,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 48300
-                    positionX (Float) : -273.0929
+                    positionX (Float) : -273.0928
                     positionY (Float) : 561.1105
                     positionZ (Float) : 0.0417
                 691 (CSectorDataObject)
@@ -6167,7 +6167,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 48364
-                    positionX (Float) : -273.0929
+                    positionX (Float) : -273.0928
                     positionY (Float) : 561.1105
                     positionZ (Float) : 0.0417
                 692 (CSectorDataObject)
@@ -6175,7 +6175,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 48440
-                    positionX (Float) : -264.4636
+                    positionX (Float) : -264.4635
                     positionY (Float) : 559.0079
                     positionZ (Float) : 1.1266
                 693 (CSectorDataObject)
@@ -6183,7 +6183,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 48504
-                    positionX (Float) : -264.4636
+                    positionX (Float) : -264.4635
                     positionY (Float) : 559.0079
                     positionZ (Float) : 1.1266
                 694 (CSectorDataObject)
@@ -6287,16 +6287,16 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 49420
-                    positionX (Float) : -161.3635
-                    positionY (Float) : 727.3539
+                    positionX (Float) : -161.3634
+                    positionY (Float) : 727.3538
                     positionZ (Float) : -0.8944
                 707 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 49484
-                    positionX (Float) : -161.3635
-                    positionY (Float) : 727.3539
+                    positionX (Float) : -161.3634
+                    positionY (Float) : 727.3538
                     positionZ (Float) : -0.8944
                 708 (CSectorDataObject)
                     type (Uint8) : 1
@@ -7408,7 +7408,7 @@
                     radius (Uint16) : 60
                     offset (Uint64) : 59148
                     positionX (Float) : -353.0233
-                    positionY (Float) : 774.3588
+                    positionY (Float) : 774.3589
                     positionZ (Float) : -0.4072
                 847 (CSectorDataObject)
                     type (Uint8) : 1
@@ -9085,7 +9085,7 @@
                         bz (Float) : 0.6553
                         by (Float) : 0.0140
                         cz (Float) : -0.0675
-                        cy (Float) : -0.1302
+                        cy (Float) : -0.1301
                         cz (Float) : 1.2001
                     position (SVector3D)
                         X (Float) : -268.2272
@@ -9103,7 +9103,7 @@
                         bz (Float) : 0.6553
                         by (Float) : 0.0140
                         cz (Float) : -0.0675
-                        cy (Float) : -0.1302
+                        cy (Float) : -0.1301
                         cz (Float) : 1.2001
                     position (SVector3D)
                         X (Float) : -268.2272
@@ -12079,7 +12079,7 @@
                         by (Float) : 0.1297
                         cz (Float) : -0.1294
                         cy (Float) : -0.0088
-                        cz (Float) : 0.9916
+                        cz (Float) : 0.9915
                     position (SVector3D)
                         X (Float) : -284.9917
                         Y (Float) : 616.7651
@@ -12097,7 +12097,7 @@
                         by (Float) : 0.1297
                         cz (Float) : -0.1294
                         cy (Float) : -0.0088
-                        cz (Float) : 0.9916
+                        cz (Float) : 0.9915
                     position (SVector3D)
                         X (Float) : -284.9917
                         Y (Float) : 616.7651
@@ -16624,7 +16624,7 @@
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.5540
                         ay (Float) : 0.8310
-                        az (Float) : 0.0502
+                        az (Float) : 0.0501
                         by (Float) : -0.8322
                         bz (Float) : 0.5512
                         by (Float) : 0.0594
@@ -16642,7 +16642,7 @@
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.5540
                         ay (Float) : 0.8310
-                        az (Float) : 0.0502
+                        az (Float) : 0.0501
                         by (Float) : -0.8322
                         bz (Float) : 0.5512
                         by (Float) : 0.0594
@@ -20805,7 +20805,7 @@
                 636 (SBlockData)
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.8726
-                        ay (Float) : -0.4785
+                        ay (Float) : -0.4784
                         az (Float) : 0.0983
                         by (Float) : 0.4678
                         bz (Float) : 0.8765
@@ -20823,7 +20823,7 @@
                 637 (SBlockData)
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.8726
-                        ay (Float) : -0.4785
+                        ay (Float) : -0.4784
                         az (Float) : 0.0983
                         by (Float) : 0.4678
                         bz (Float) : 0.8765
@@ -24275,7 +24275,7 @@
                         cz (Float) : -0.0139
                     position (SVector3D)
                         X (Float) : -108.7825
-                        Y (Float) : 529.5852
+                        Y (Float) : 529.5851
                         Z (Float) : 4.5333
                     streamingRadius (Uint16) : 0
                     flags (Uint16) : 1029
@@ -25904,7 +25904,7 @@
                         unk1 (CBytes) : 8 bytes
                 886 (SBlockData)
                     rotationMatrix (CMatrix3x3)
-                        ax (Float) : 0.0833
+                        ax (Float) : 0.0834
                         ay (Float) : -4.3441
                         az (Float) : 0.0000
                         by (Float) : 4.3437
