 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git 1.32/characters/models/crowd_npc/nml_ baron_thug_lvl2/torso/model/t3ad_01_mb__nml_ baron_thug_lvl2.w2mesh.txt 4.00/characters/models/crowd_npc/nml_ baron_thug_lvl2/torso/model/t3ad_01_mb__nml_ baron_thug_lvl2.w2mesh.txt
index 0382252..1fbb431 100644
--- 1.32/characters/models/crowd_npc/nml_ baron_thug_lvl2/torso/model/t3ad_01_mb__nml_ baron_thug_lvl2.w2mesh.txt	
+++ 4.00/characters/models/crowd_npc/nml_ baron_thug_lvl2/torso/model/t3ad_01_mb__nml_ baron_thug_lvl2.w2mesh.txt	
@@ -9,7 +9,7 @@
                     X (Float) : -0.6450
                     Y (Float) : -0.2964
                     Z (Float) : 0.5869
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.6420
                     Y (Float) : 0.2158
@@ -279,6 +279,8 @@
                 vertexBufferSize (Uint32) : 510896
                 indexBufferSize (Uint32) : 94110
                 indexBufferOffset (Uint32) : 510976
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             soundInfo (ptr:SMeshSoundInfo) : SMeshSoundInfo #4
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
