 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git 1.32/levels/novigrad/nml_surface/spawning pools/cave_invisible_obj.w2l.txt 4.00/levels/novigrad/nml_surface/spawning pools/cave_invisible_obj.w2l.txt
index 435ac39..8b71c52 100644
--- 1.32/levels/novigrad/nml_surface/spawning pools/cave_invisible_obj.w2l.txt	
+++ 4.00/levels/novigrad/nml_surface/spawning pools/cave_invisible_obj.w2l.txt	
@@ -286,7 +286,7 @@
                     offset (Uint64) : 2240
                     positionX (Float) : 1102.3470
                     positionY (Float) : -736.4464
-                    positionZ (Float) : -3.7730
+                    positionZ (Float) : -3.7731
                 33 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
@@ -294,7 +294,7 @@
                     offset (Uint64) : 2304
                     positionX (Float) : 1102.3470
                     positionY (Float) : -736.4464
-                    positionZ (Float) : -3.7730
+                    positionZ (Float) : -3.7731
                 34 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -317,7 +317,7 @@
                     radius (Uint16) : 40
                     offset (Uint64) : 2520
                     positionX (Float) : 1117.6280
-                    positionY (Float) : -740.8576
+                    positionY (Float) : -740.8577
                     positionZ (Float) : -0.5512
                 37 (CSectorDataObject)
                     type (Uint8) : 2
@@ -325,7 +325,7 @@
                     radius (Uint16) : 40
                     offset (Uint64) : 2584
                     positionX (Float) : 1117.6280
-                    positionY (Float) : -740.8576
+                    positionY (Float) : -740.8577
                     positionZ (Float) : -0.5512
                 38 (CSectorDataObject)
                     type (Uint8) : 1
@@ -854,7 +854,7 @@
                         bz (Float) : 3.7905
                         by (Float) : -0.0484
                         cz (Float) : -0.0387
-                        cy (Float) : 0.1312
+                        cy (Float) : 0.1311
                         cz (Float) : 7.4780
                     position (SVector3D)
                         X (Float) : 1081.7230
@@ -872,7 +872,7 @@
                         bz (Float) : 3.7905
                         by (Float) : -0.0484
                         cz (Float) : -0.0387
-                        cy (Float) : 0.1312
+                        cy (Float) : 0.1311
                         cz (Float) : 7.4780
                     position (SVector3D)
                         X (Float) : 1081.7230
@@ -970,7 +970,7 @@
                         collisionGroup (Uint64) : 550725311
                 14 (SBlockData)
                     rotationMatrix (CMatrix3x3)
-                        ax (Float) : -10.4342
+                        ax (Float) : -10.4341
                         ay (Float) : -2.4033
                         az (Float) : -0.0120
                         by (Float) : 1.3810
@@ -988,7 +988,7 @@
                     occlusionSystemID (Uint32) : 43598682
                 15 (SBlockData)
                     rotationMatrix (CMatrix3x3)
-                        ax (Float) : -10.4342
+                        ax (Float) : -10.4341
                         ay (Float) : -2.4033
                         az (Float) : -0.0120
                         by (Float) : 1.3810
@@ -1095,7 +1095,7 @@
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : -9.1567
                         ay (Float) : -5.5497
-                        az (Float) : 0.0498
+                        az (Float) : 0.0497
                         by (Float) : 2.5015
                         bz (Float) : -4.1266
                         by (Float) : 0.0853
@@ -1113,7 +1113,7 @@
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : -9.1567
                         ay (Float) : -5.5497
-                        az (Float) : 0.0498
+                        az (Float) : 0.0497
                         by (Float) : 2.5015
                         bz (Float) : -4.1266
                         by (Float) : 0.0853
@@ -1955,7 +1955,7 @@
                 62 (SBlockData)
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 1.1851
-                        ay (Float) : -3.5714
+                        ay (Float) : -3.5713
                         az (Float) : 0.0687
                         by (Float) : 3.0736
                         bz (Float) : 1.0199
@@ -1973,7 +1973,7 @@
                 63 (SBlockData)
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 1.1851
-                        ay (Float) : -3.5714
+                        ay (Float) : -3.5713
                         az (Float) : 0.0687
                         by (Float) : 3.0736
                         bz (Float) : 1.0199
