 1 file changed, 34 insertions(+), 34 deletions(-)

diff --git 1.32/levels/novigrad/nml_surface/spawning pools/decals.w2l.txt 4.00/levels/novigrad/nml_surface/spawning pools/decals.w2l.txt
index a1c51de..26692fe 100644
--- 1.32/levels/novigrad/nml_surface/spawning pools/decals.w2l.txt	
+++ 4.00/levels/novigrad/nml_surface/spawning pools/decals.w2l.txt	
@@ -66,7 +66,7 @@
                 0 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 78
+                    radius (Uint16) : 103
                     offset (Uint64) : 0
                     positionX (Float) : 1068.6760
                     positionY (Float) : -746.3533
@@ -74,7 +74,7 @@
                 1 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 105
+                    radius (Uint16) : 138
                     offset (Uint64) : 76
                     positionX (Float) : 1070.8330
                     positionY (Float) : -749.2812
@@ -82,7 +82,7 @@
                 2 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 104
+                    radius (Uint16) : 137
                     offset (Uint64) : 152
                     positionX (Float) : 1080.2060
                     positionY (Float) : -750.3453
@@ -90,7 +90,7 @@
                 3 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 98
+                    radius (Uint16) : 130
                     offset (Uint64) : 228
                     positionX (Float) : 1072.1820
                     positionY (Float) : -747.9120
@@ -98,7 +98,7 @@
                 4 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 86
+                    radius (Uint16) : 114
                     offset (Uint64) : 304
                     positionX (Float) : 1066.1780
                     positionY (Float) : -746.2008
@@ -106,7 +106,7 @@
                 5 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 84
+                    radius (Uint16) : 111
                     offset (Uint64) : 380
                     positionX (Float) : 1102.2960
                     positionY (Float) : -750.6323
@@ -114,7 +114,7 @@
                 6 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 95
+                    radius (Uint16) : 125
                     offset (Uint64) : 456
                     positionX (Float) : 1096.0750
                     positionY (Float) : -748.0764
@@ -122,7 +122,7 @@
                 7 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 101
+                    radius (Uint16) : 133
                     offset (Uint64) : 532
                     positionX (Float) : 1104.0730
                     positionY (Float) : -748.0186
@@ -130,7 +130,7 @@
                 8 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 146
+                    radius (Uint16) : 193
                     offset (Uint64) : 608
                     positionX (Float) : 1104.2200
                     positionY (Float) : -758.0502
@@ -138,7 +138,7 @@
                 9 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 142
+                    radius (Uint16) : 188
                     offset (Uint64) : 684
                     positionX (Float) : 1067.9390
                     positionY (Float) : -746.1167
@@ -146,7 +146,7 @@
                 10 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 139
+                    radius (Uint16) : 183
                     offset (Uint64) : 760
                     positionX (Float) : 1068.3060
                     positionY (Float) : -736.8690
@@ -154,7 +154,7 @@
                 11 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 112
+                    radius (Uint16) : 147
                     offset (Uint64) : 836
                     positionX (Float) : 1066.7950
                     positionY (Float) : -742.4397
@@ -162,7 +162,7 @@
                 12 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 126
+                    radius (Uint16) : 167
                     offset (Uint64) : 912
                     positionX (Float) : 1127.0090
                     positionY (Float) : -735.4315
@@ -170,7 +170,7 @@
                 13 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 138
+                    radius (Uint16) : 182
                     offset (Uint64) : 988
                     positionX (Float) : 1131.2070
                     positionY (Float) : -733.0197
@@ -178,7 +178,7 @@
                 14 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 132
+                    radius (Uint16) : 174
                     offset (Uint64) : 1064
                     positionX (Float) : 1128.9990
                     positionY (Float) : -744.7726
@@ -186,7 +186,7 @@
                 15 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 125
+                    radius (Uint16) : 166
                     offset (Uint64) : 1140
                     positionX (Float) : 1114.0270
                     positionY (Float) : -752.7183
@@ -194,7 +194,7 @@
                 16 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 125
+                    radius (Uint16) : 165
                     offset (Uint64) : 1216
                     positionX (Float) : 1123.6940
                     positionY (Float) : -736.9393
@@ -202,7 +202,7 @@
                 17 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 124
+                    radius (Uint16) : 164
                     offset (Uint64) : 1292
                     positionX (Float) : 1130.7140
                     positionY (Float) : -743.9962
@@ -210,7 +210,7 @@
                 18 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 141
+                    radius (Uint16) : 187
                     offset (Uint64) : 1368
                     positionX (Float) : 1115.9880
                     positionY (Float) : -746.3299
@@ -218,7 +218,7 @@
                 19 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 119
+                    radius (Uint16) : 157
                     offset (Uint64) : 1444
                     positionX (Float) : 1102.7920
                     positionY (Float) : -741.1115
@@ -226,7 +226,7 @@
                 20 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 139
+                    radius (Uint16) : 183
                     offset (Uint64) : 1520
                     positionX (Float) : 1125.4500
                     positionY (Float) : -740.0703
@@ -234,7 +234,7 @@
                 21 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 125
+                    radius (Uint16) : 166
                     offset (Uint64) : 1596
                     positionX (Float) : 1134.0830
                     positionY (Float) : -748.3791
@@ -242,7 +242,7 @@
                 22 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 141
+                    radius (Uint16) : 187
                     offset (Uint64) : 1672
                     positionX (Float) : 1114.6090
                     positionY (Float) : -749.5439
@@ -250,7 +250,7 @@
                 23 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 149
+                    radius (Uint16) : 197
                     offset (Uint64) : 1748
                     positionX (Float) : 1100.6900
                     positionY (Float) : -727.8839
@@ -258,7 +258,7 @@
                 24 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 149
+                    radius (Uint16) : 197
                     offset (Uint64) : 1824
                     positionX (Float) : 1099.1300
                     positionY (Float) : -731.7268
@@ -266,7 +266,7 @@
                 25 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 153
+                    radius (Uint16) : 203
                     offset (Uint64) : 1900
                     positionX (Float) : 1097.7790
                     positionY (Float) : -734.4515
@@ -274,7 +274,7 @@
                 26 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 112
+                    radius (Uint16) : 148
                     offset (Uint64) : 1976
                     positionX (Float) : 1094.6220
                     positionY (Float) : -735.8410
@@ -282,7 +282,7 @@
                 27 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 106
+                    radius (Uint16) : 140
                     offset (Uint64) : 2052
                     positionX (Float) : 1092.3040
                     positionY (Float) : -745.2953
@@ -290,7 +290,7 @@
                 28 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 109
+                    radius (Uint16) : 143
                     offset (Uint64) : 2128
                     positionX (Float) : 1091.0290
                     positionY (Float) : -748.9311
@@ -298,7 +298,7 @@
                 29 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 121
+                    radius (Uint16) : 159
                     offset (Uint64) : 2204
                     positionX (Float) : 1087.7880
                     positionY (Float) : -745.0078
@@ -306,7 +306,7 @@
                 30 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 113
+                    radius (Uint16) : 149
                     offset (Uint64) : 2280
                     positionX (Float) : 1087.9740
                     positionY (Float) : -741.2766
@@ -314,7 +314,7 @@
                 31 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 114
+                    radius (Uint16) : 151
                     offset (Uint64) : 2356
                     positionX (Float) : 1094.5500
                     positionY (Float) : -741.9600
@@ -322,7 +322,7 @@
                 32 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 152
+                    radius (Uint16) : 201
                     offset (Uint64) : 2432
                     positionX (Float) : 1099.0110
                     positionY (Float) : -755.1144
@@ -330,7 +330,7 @@
                 33 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 151
+                    radius (Uint16) : 200
                     offset (Uint64) : 2508
                     positionX (Float) : 1104.5990
                     positionY (Float) : -745.7087
