 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git 1.32/levels/novigrad/nml_surface/spawning pools/fx.w2l.txt 4.00/levels/novigrad/nml_surface/spawning pools/fx.w2l.txt
index 7d19f14..8ca3f17 100644
--- 1.32/levels/novigrad/nml_surface/spawning pools/fx.w2l.txt	
+++ 4.00/levels/novigrad/nml_surface/spawning pools/fx.w2l.txt	
@@ -188,7 +188,7 @@
                     radius (Uint16) : 40
                     offset (Uint64) : 680
                     positionX (Float) : 1072.6080
-                    positionY (Float) : -747.8504
+                    positionY (Float) : -747.8503
                     positionZ (Float) : -8.0119
                 11 (CSectorDataObject)
                     type (Uint8) : 1
@@ -196,7 +196,7 @@
                     radius (Uint16) : 40
                     offset (Uint64) : 744
                     positionX (Float) : 1102.7740
-                    positionY (Float) : -753.2235
+                    positionY (Float) : -753.2236
                     positionZ (Float) : -7.4105
                 12 (CSectorDataObject)
                     type (Uint8) : 1
