 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house6/women.w2l.txt 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house6/women.w2l.txt
index 7148b19..3c9c632 100644
--- 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house6/women.w2l.txt	
+++ 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house6/women.w2l.txt	
@@ -105,14 +105,14 @@
                         graphPositionY (Int16) : 209
                         boundingBox (Box)
                             Min (Vector)
-                                X (Float) : -397.8978
-                                Y (Float) : -1254.9760
+                                X (Float) : -397.8961
+                                Y (Float) : -1254.9750
                                 Z (Float) : 43.6066
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : -396.4745
-                                Y (Float) : -1253.5710
-                                Z (Float) : 44.2047
+                                X (Float) : -396.4759
+                                Y (Float) : -1253.5730
+                                Z (Float) : 44.2037
                                 W (Float) : 1.0000
                         drawableFlags (EDrawableFlags) : DF_IsVisible,DF_CastShadows
                         mesh (handle:CMesh) : dlc/bob/data/environment/decorations/gameplay/generic/containers/washtubs/gen_washtub_clothes.w2mesh
@@ -250,12 +250,12 @@
                     box5 (Float) : 0.0052
                     pathHash (String) : "environment/decorations/water/puddles/puddle_round_calm.w2mesh"
                 2 (CSectorDataResource)
-                    box0 (Float) : -0.5334
-                    box1 (Float) : -0.5060
+                    box0 (Float) : -0.5323
+                    box1 (Float) : -0.5050
                     box2 (Float) : -0.0033
-                    box3 (Float) : 0.5372
-                    box4 (Float) : 0.5201
-                    box5 (Float) : 0.5948
+                    box3 (Float) : 0.5356
+                    box4 (Float) : 0.5194
+                    box5 (Float) : 0.5938
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/containers/washtubs/gen_washtub_clothes.w2mesh"
             Objects (CBufferVLQInt32:CSectorDataObject)
                 0 (CSectorDataObject)
@@ -271,17 +271,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 64
-                    positionX (Float) : -397.1862
+                    positionX (Float) : -397.1860
                     positionY (Float) : -1254.2740
-                    positionZ (Float) : 43.9056
+                    positionZ (Float) : 43.9051
                 2 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 128
-                    positionX (Float) : -397.1862
+                    positionX (Float) : -397.1860
                     positionY (Float) : -1254.2740
-                    positionZ (Float) : 43.9056
+                    positionZ (Float) : 43.9051
             blocksize (CVLQInt32) : 204
             BlockData (CCompressedBuffer:SBlockData)
                 0 (SBlockData)
