 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/beauclair_citizen_corpse04.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/beauclair_citizen_corpse04.w2mesh.txt
index 4c3d027..ee6c018 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/beauclair_citizen_corpse04.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/beauclair_citizen_corpse04.w2mesh.txt	
@@ -7,7 +7,7 @@
                     X (Float) : -0.8497
                     Y (Float) : -0.7039
                     Z (Float) : -0.1192
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.3191
                     Y (Float) : 1.0077
@@ -32,10 +32,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 75305500000000000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : -0.4957
+                    Y (Float) : -0.5942
+                    Z (Float) : 0.1089
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                     2 (Float) : 12.0000
@@ -55,6 +55,8 @@
                 vertexBufferSize (Uint32) : 270368
                 indexBufferSize (Uint32) : 112128
                 indexBufferOffset (Uint32) : 271360
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
