 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/beauclair_citizen_corpse06.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/beauclair_citizen_corpse06.w2mesh.txt
index aa6d79d..5979e01 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/beauclair_citizen_corpse06.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/beauclair_citizen_corpse06.w2mesh.txt	
@@ -7,7 +7,7 @@
                     X (Float) : -0.5024
                     Y (Float) : -0.3338
                     Z (Float) : -0.0772
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.3558
                     Y (Float) : 1.0560
@@ -32,10 +32,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 75313540000000000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : -0.2856
+                    Y (Float) : -0.0884
+                    Z (Float) : 0.7423
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                     2 (Float) : 12.0000
@@ -55,6 +55,8 @@
                 vertexBufferSize (Uint32) : 269888
                 indexBufferSize (Uint32) : 112146
                 indexBufferOffset (Uint32) : 270336
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
