 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/c_01_ma__corpse01.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/c_01_ma__corpse01.w2mesh.txt
index 53cd9a1..922efcf 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/c_01_ma__corpse01.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/c_01_ma__corpse01.w2mesh.txt	
@@ -7,7 +7,7 @@
                     X (Float) : 0.0829
                     Y (Float) : -0.9227
                     Z (Float) : 0.0203
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.2854
                     Y (Float) : -0.7120
@@ -42,6 +42,8 @@
                 vertexBufferSize (Uint32) : 9600
                 indexBufferSize (Uint32) : 4800
                 indexBufferOffset (Uint32) : 10240
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
