 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/c_07_ma__corpse04.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/c_07_ma__corpse04.w2mesh.txt
index 0bd83bf..cbbcdf6 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/c_07_ma__corpse04.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/beauclair_citizen_corpse01/model/c_07_ma__corpse04.w2mesh.txt	
@@ -7,7 +7,7 @@
                     X (Float) : -0.6606
                     Y (Float) : -0.7455
                     Z (Float) : -0.0474
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : -0.3581
                     Y (Float) : -0.4423
@@ -48,6 +48,8 @@
                 vertexBufferSize (Uint32) : 45328
                 indexBufferSize (Uint32) : 23202
                 indexBufferOffset (Uint32) : 46080
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
