 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git 1.32/characters/models/crowd_npc/nml_ baron_thug_lvl2/dress/model/d_01_mb__nml_baron_thug_lvl2.w2mesh.txt 4.00/characters/models/crowd_npc/nml_ baron_thug_lvl2/dress/model/d_01_mb__nml_baron_thug_lvl2.w2mesh.txt
index b0136d4..0af29df 100644
--- 1.32/characters/models/crowd_npc/nml_ baron_thug_lvl2/dress/model/d_01_mb__nml_baron_thug_lvl2.w2mesh.txt	
+++ 4.00/characters/models/crowd_npc/nml_ baron_thug_lvl2/dress/model/d_01_mb__nml_baron_thug_lvl2.w2mesh.txt	
@@ -8,11 +8,11 @@
                     X (Float) : -0.3361
                     Y (Float) : -0.2853
                     Z (Float) : 0.3850
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.3365
                     Y (Float) : 0.2068
-                    Z (Float) : 1.1116
+                    Z (Float) : 1.1102
                     W (Float) : 1.0000
             autoHideDistance (Float) : 60.0000
             generalizedMeshRadius (Float) : 0.4311
@@ -125,6 +125,8 @@
                 vertexBufferSize (Uint32) : 40808
                 indexBufferSize (Uint32) : 14904
                 indexBufferOffset (Uint32) : 40960
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
