 1 file changed, 37 insertions(+), 37 deletions(-)

diff --git 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/vineyard/day/spot8.w2l.txt 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/vineyard/day/spot8.w2l.txt
index 403a81a..f6c4fb9 100644
--- 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/vineyard/day/spot8.w2l.txt	
+++ 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/vineyard/day/spot8.w2l.txt	
@@ -58,14 +58,14 @@
                         graphPositionY (Int16) : 680
                         boundingBox (Box)
                             Min (Vector)
-                                X (Float) : -395.4909
-                                Y (Float) : -1129.3810
-                                Z (Float) : 58.1899
+                                X (Float) : -395.4340
+                                Y (Float) : -1129.3840
+                                Z (Float) : 58.1698
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : -395.1273
-                                Y (Float) : -1129.0740
-                                Z (Float) : 58.4451
+                                X (Float) : -395.0686
+                                Y (Float) : -1129.0750
+                                Z (Float) : 58.4313
                                 W (Float) : 1.0000
                         drawableFlags (EDrawableFlags) : DF_IsVisible,DF_CastShadows
                         mesh (handle:CMesh) : environment/decorations/near_water/ropes/rope_coil.w2mesh
@@ -157,14 +157,14 @@
                         graphPositionY (Int16) : 680
                         boundingBox (Box)
                             Min (Vector)
-                                X (Float) : -395.3502
-                                Y (Float) : -1131.4260
-                                Z (Float) : 58.1899
+                                X (Float) : -395.3810
+                                Y (Float) : -1131.4770
+                                Z (Float) : 58.1698
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : -394.9227
-                                Y (Float) : -1130.9730
-                                Z (Float) : 58.4451
+                                X (Float) : -394.9509
+                                Y (Float) : -1131.0220
+                                Z (Float) : 58.4313
                                 W (Float) : 1.0000
                         drawableFlags (EDrawableFlags) : DF_IsVisible,DF_CastShadows
                         mesh (handle:CMesh) : environment/decorations/near_water/ropes/rope_coil.w2mesh
@@ -193,7 +193,7 @@
                                 Z (Float) : 58.1371
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : -394.7662
+                                X (Float) : -394.7663
                                 Y (Float) : -1130.8090
                                 Z (Float) : 58.6211
                                 W (Float) : 1.0000
@@ -727,14 +727,14 @@
                         graphPositionY (Int16) : 680
                         boundingBox (Box)
                             Min (Vector)
-                                X (Float) : -406.1312
-                                Y (Float) : -1129.4000
-                                Z (Float) : 60.9023
+                                X (Float) : -406.1858
+                                Y (Float) : -1129.4230
+                                Z (Float) : 60.8822
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : -405.7085
-                                Y (Float) : -1128.9900
-                                Z (Float) : 61.1574
+                                X (Float) : -405.7617
+                                Y (Float) : -1129.0100
+                                Z (Float) : 61.1437
                                 W (Float) : 1.0000
                         drawableFlags (EDrawableFlags) : DF_IsVisible,DF_CastShadows
                         mesh (handle:CMesh) : environment/decorations/near_water/ropes/rope_coil.w2mesh
@@ -763,7 +763,7 @@
                                 Z (Float) : 60.8495
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : -405.6698
+                                X (Float) : -405.6697
                                 Y (Float) : -1128.7930
                                 Z (Float) : 61.3335
                                 W (Float) : 1.0000
@@ -801,12 +801,12 @@
                     box5 (Float) : 0.0000
                     pathHash (String) : ""
                 1 (CSectorDataResource)
-                    box0 (Float) : -0.4932
-                    box1 (Float) : -0.7129
-                    box2 (Float) : -0.0240
-                    box3 (Float) : 0.5133
-                    box4 (Float) : 0.5033
-                    box5 (Float) : 0.2690
+                    box0 (Float) : -0.5032
+                    box1 (Float) : -0.5045
+                    box2 (Float) : -0.0498
+                    box3 (Float) : 0.5078
+                    box4 (Float) : 0.7152
+                    box5 (Float) : 0.2556
                     pathHash (String) : "environment/decorations/near_water/ropes/rope_coil.w2mesh"
                 2 (CSectorDataResource)
                     box0 (Float) : -0.3045
@@ -830,9 +830,9 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 30
                     offset (Uint64) : 0
-                    positionX (Float) : -395.3091
-                    positionY (Float) : -1129.2280
-                    positionZ (Float) : 58.3175
+                    positionX (Float) : -395.2513
+                    positionY (Float) : -1129.2290
+                    positionZ (Float) : 58.3006
                 1 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -854,15 +854,15 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 30
                     offset (Uint64) : 204
-                    positionX (Float) : -395.1364
-                    positionY (Float) : -1131.1990
-                    positionZ (Float) : 58.3175
+                    positionX (Float) : -395.1660
+                    positionY (Float) : -1131.2490
+                    positionZ (Float) : 58.3006
                 4 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 40
                     offset (Uint64) : 268
-                    positionX (Float) : -395.1792
+                    positionX (Float) : -395.1793
                     positionY (Float) : -1131.2250
                     positionZ (Float) : 58.3791
                 5 (CSectorDataObject)
@@ -870,7 +870,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 40
                     offset (Uint64) : 332
-                    positionX (Float) : -395.1792
+                    positionX (Float) : -395.1793
                     positionY (Float) : -1131.2250
                     positionZ (Float) : 58.3791
                 6 (CSectorDataObject)
@@ -894,9 +894,9 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 30
                     offset (Uint64) : 548
-                    positionX (Float) : -405.9199
-                    positionY (Float) : -1129.1950
-                    positionZ (Float) : 61.0299
+                    positionX (Float) : -405.9738
+                    positionY (Float) : -1129.2160
+                    positionZ (Float) : 61.0129
                 9 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
