 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house1/women.w2l.txt 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house1/women.w2l.txt
index b7f3b42..11746aa 100644
--- 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house1/women.w2l.txt	
+++ 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house1/women.w2l.txt	
@@ -68,14 +68,14 @@
                         graphPositionY (Int16) : 209
                         boundingBox (Box)
                             Min (Vector)
-                                X (Float) : -415.8834
-                                Y (Float) : -1295.0980
+                                X (Float) : -415.8817
+                                Y (Float) : -1295.0970
                                 Z (Float) : 55.0153
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : -414.4601
-                                Y (Float) : -1293.6940
-                                Z (Float) : 55.6135
+                                X (Float) : -414.4615
+                                Y (Float) : -1293.6960
+                                Z (Float) : 55.6124
                                 W (Float) : 1.0000
                         drawableFlags (EDrawableFlags) : DF_IsVisible,DF_CastShadows
                         mesh (handle:CMesh) : dlc/bob/data/environment/decorations/gameplay/generic/containers/washtubs/gen_washtub_clothes.w2mesh
@@ -218,12 +218,12 @@
                     box5 (Float) : 0.0052
                     pathHash (String) : "environment/decorations/water/puddles/puddle_round_calm.w2mesh"
                 2 (CSectorDataResource)
-                    box0 (Float) : -0.5334
-                    box1 (Float) : -0.5060
+                    box0 (Float) : -0.5323
+                    box1 (Float) : -0.5050
                     box2 (Float) : -0.0033
-                    box3 (Float) : 0.5372
-                    box4 (Float) : 0.5201
-                    box5 (Float) : 0.5948
+                    box3 (Float) : 0.5356
+                    box4 (Float) : 0.5194
+                    box5 (Float) : 0.5938
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/containers/washtubs/gen_washtub_clothes.w2mesh"
             Objects (CBufferVLQInt32:CSectorDataObject)
                 0 (CSectorDataObject)
@@ -239,17 +239,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 64
-                    positionX (Float) : -415.1718
+                    positionX (Float) : -415.1716
                     positionY (Float) : -1294.3960
-                    positionZ (Float) : 55.3144
+                    positionZ (Float) : 55.3139
                 2 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 128
-                    positionX (Float) : -415.1718
+                    positionX (Float) : -415.1716
                     positionY (Float) : -1294.3960
-                    positionZ (Float) : 55.3144
+                    positionZ (Float) : 55.3139
             blocksize (CVLQInt32) : 204
             BlockData (CCompressedBuffer:SBlockData)
                 0 (SBlockData)
