 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git 1.32/levels/kaer_morhen/kaer_morhen_valley/pyre/burial pyre.w2l.txt 4.00/levels/kaer_morhen/kaer_morhen_valley/pyre/burial pyre.w2l.txt
index 5147f1b..2b10ac8 100644
--- 1.32/levels/kaer_morhen/kaer_morhen_valley/pyre/burial pyre.w2l.txt	
+++ 4.00/levels/kaer_morhen/kaer_morhen_valley/pyre/burial pyre.w2l.txt	
@@ -25,7 +25,7 @@
                 2 (CSectorDataResource)
                     box0 (Float) : -0.6064
                     box1 (Float) : -0.5550
-                    box2 (Float) : -0.0347
+                    box2 (Float) : -0.0291
                     box3 (Float) : 0.6739
                     box4 (Float) : 0.5449
                     box5 (Float) : 0.6546
@@ -68,17 +68,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 64
-                    positionX (Float) : 62.4494
-                    positionY (Float) : -93.4178
-                    positionZ (Float) : 180.0178
+                    positionX (Float) : 62.4482
+                    positionY (Float) : -93.4171
+                    positionZ (Float) : 180.0217
                 2 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 128
-                    positionX (Float) : 62.4494
-                    positionY (Float) : -93.4178
-                    positionZ (Float) : 180.0178
+                    positionX (Float) : 62.4482
+                    positionY (Float) : -93.4171
+                    positionZ (Float) : 180.0217
                 3 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
