 1 file changed, 57 insertions(+), 57 deletions(-)

diff --git 1.32/levels/skellige/water_pois/hangman's isle/hangman's isle.w2l.txt 4.00/levels/skellige/water_pois/hangman's isle/hangman's isle.w2l.txt
index a150d24..10b8a06 100644
--- 1.32/levels/skellige/water_pois/hangman's isle/hangman's isle.w2l.txt	
+++ 4.00/levels/skellige/water_pois/hangman's isle/hangman's isle.w2l.txt	
@@ -38,28 +38,28 @@
                     box5 (Float) : 0.0000
                     pathHash (String) : ""
                 1 (CSectorDataResource)
-                    box0 (Float) : -4.0707
-                    box1 (Float) : -9.6331
-                    box2 (Float) : -5.0070
-                    box3 (Float) : 3.4737
-                    box4 (Float) : 8.4144
-                    box5 (Float) : 5.0106
+                    box0 (Float) : -4.0844
+                    box1 (Float) : -9.6377
+                    box2 (Float) : -5.0777
+                    box3 (Float) : 3.4594
+                    box4 (Float) : 8.4121
+                    box5 (Float) : 5.0213
                     pathHash (String) : "environment/terrain_surroundings/common/cave/cave_big/cave_wall_long_no_floor.w2mesh"
                 2 (CSectorDataResource)
-                    box0 (Float) : -3.3086
-                    box1 (Float) : -7.7629
-                    box2 (Float) : -4.4512
-                    box3 (Float) : 3.3052
+                    box0 (Float) : -3.3344
+                    box1 (Float) : -7.7690
+                    box2 (Float) : -4.4372
+                    box3 (Float) : 3.3123
                     box4 (Float) : 7.7543
-                    box5 (Float) : 4.4480
+                    box5 (Float) : 4.4383
                     pathHash (String) : "environment/terrain_surroundings/common/cave/cave_big/cave_wall_big_no_floor.w2mesh"
                 3 (CSectorDataResource)
-                    box0 (Float) : -3.8709
-                    box1 (Float) : -4.1971
-                    box2 (Float) : -0.7037
-                    box3 (Float) : 3.7181
-                    box4 (Float) : 4.9346
-                    box5 (Float) : 0.5998
+                    box0 (Float) : -3.8449
+                    box1 (Float) : -4.1831
+                    box2 (Float) : -0.4626
+                    box3 (Float) : 3.7264
+                    box4 (Float) : 4.9094
+                    box5 (Float) : 0.5735
                     pathHash (String) : "environment/terrain_surroundings/common/rock_debris.w2mesh"
                 4 (CSectorDataResource)
                     box0 (Float) : -4.1814
@@ -155,97 +155,97 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 120
                     offset (Uint64) : 0
-                    positionX (Float) : 1587.7380
-                    positionY (Float) : -51.5568
-                    positionZ (Float) : 2.1702
+                    positionX (Float) : 1587.7300
+                    positionY (Float) : -51.5782
+                    positionZ (Float) : 2.1550
                 1 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 120
                     offset (Uint64) : 64
-                    positionX (Float) : 1587.7380
-                    positionY (Float) : -51.5568
-                    positionZ (Float) : 2.1702
+                    positionX (Float) : 1587.7300
+                    positionY (Float) : -51.5782
+                    positionZ (Float) : 2.1550
                 2 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 120
                     offset (Uint64) : 140
-                    positionX (Float) : 1580.7320
-                    positionY (Float) : -50.9223
-                    positionZ (Float) : 2.2582
+                    positionX (Float) : 1580.7340
+                    positionY (Float) : -50.9303
+                    positionZ (Float) : 2.2641
                 3 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 120
                     offset (Uint64) : 204
-                    positionX (Float) : 1580.7320
-                    positionY (Float) : -50.9223
-                    positionZ (Float) : 2.2582
+                    positionX (Float) : 1580.7340
+                    positionY (Float) : -50.9303
+                    positionZ (Float) : 2.2641
                 4 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 280
-                    positionX (Float) : 1583.9070
-                    positionY (Float) : -53.9369
-                    positionZ (Float) : 0.2007
+                    positionX (Float) : 1583.8990
+                    positionY (Float) : -53.9593
+                    positionZ (Float) : 0.3166
                 5 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 344
-                    positionX (Float) : 1583.9070
-                    positionY (Float) : -53.9369
-                    positionZ (Float) : 0.2007
+                    positionX (Float) : 1583.8990
+                    positionY (Float) : -53.9593
+                    positionZ (Float) : 0.3166
                 6 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 420
-                    positionX (Float) : 1579.2870
-                    positionY (Float) : -50.8580
-                    positionZ (Float) : 0.7787
+                    positionX (Float) : 1579.2790
+                    positionY (Float) : -50.9180
+                    positionZ (Float) : 0.8577
                 7 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 484
-                    positionX (Float) : 1579.2870
-                    positionY (Float) : -50.8580
-                    positionZ (Float) : 0.7787
+                    positionX (Float) : 1579.2790
+                    positionY (Float) : -50.9180
+                    positionZ (Float) : 0.8577
                 8 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 560
-                    positionX (Float) : 1585.5070
-                    positionY (Float) : -52.7616
-                    positionZ (Float) : 1.9493
+                    positionX (Float) : 1585.4410
+                    positionY (Float) : -52.8209
+                    positionZ (Float) : 2.0194
                 9 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 624
-                    positionX (Float) : 1585.5070
-                    positionY (Float) : -52.7616
-                    positionZ (Float) : 1.9493
+                    positionX (Float) : 1585.4410
+                    positionY (Float) : -52.8209
+                    positionZ (Float) : 2.0194
                 10 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 700
-                    positionX (Float) : 1591.4830
-                    positionY (Float) : -55.5829
-                    positionZ (Float) : 2.3273
+                    positionX (Float) : 1591.4640
+                    positionY (Float) : -55.6525
+                    positionZ (Float) : 2.4092
                 11 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 764
-                    positionX (Float) : 1591.4830
-                    positionY (Float) : -55.5829
-                    positionZ (Float) : 2.3273
+                    positionX (Float) : 1591.4640
+                    positionY (Float) : -55.6525
+                    positionZ (Float) : 2.4092
                 12 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -829,7 +829,7 @@
                     offset (Uint64) : 5880
                     positionX (Float) : 1606.3590
                     positionY (Float) : -39.7213
-                    positionZ (Float) : 3.5767
+                    positionZ (Float) : 3.5768
                 85 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
@@ -837,7 +837,7 @@
                     offset (Uint64) : 5944
                     positionX (Float) : 1606.3590
                     positionY (Float) : -39.7213
-                    positionZ (Float) : 3.5767
+                    positionZ (Float) : 3.5768
                 86 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3053,7 +3053,7 @@
                         collisionGroup (Uint64) : 550725311
                 86 (SBlockData)
                     rotationMatrix (CMatrix3x3)
-                        ax (Float) : 0.3262
+                        ax (Float) : 0.3263
                         ay (Float) : 0.1204
                         az (Float) : 0.5908
                         by (Float) : -0.5377
@@ -3071,7 +3071,7 @@
                     occlusionSystemID (Uint32) : 1756994711
                 87 (SBlockData)
                     rotationMatrix (CMatrix3x3)
-                        ax (Float) : 0.3262
+                        ax (Float) : 0.3263
                         ay (Float) : 0.1204
                         az (Float) : 0.5908
                         by (Float) : -0.5377
