 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house8/women.w2l.txt 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house8/women.w2l.txt
index c606f62..6a3e5c0 100644
--- 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house8/women.w2l.txt	
+++ 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/house8/women.w2l.txt	
@@ -160,14 +160,14 @@
                         graphPositionY (Int16) : 209
                         boundingBox (Box)
                             Min (Vector)
-                                X (Float) : -341.2833
-                                Y (Float) : -1304.9160
+                                X (Float) : -341.2821
+                                Y (Float) : -1304.9140
                                 Z (Float) : 29.5608
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : -340.0032
-                                Y (Float) : -1303.6680
-                                Z (Float) : 30.1590
+                                X (Float) : -340.0049
+                                Y (Float) : -1303.6690
+                                Z (Float) : 30.1579
                                 W (Float) : 1.0000
                         drawableFlags (EDrawableFlags) : DF_IsVisible,DF_CastShadows
                         mesh (handle:CMesh) : dlc/bob/data/environment/decorations/gameplay/generic/containers/washtubs/gen_washtub_clothes.w2mesh
@@ -269,12 +269,12 @@
                     box5 (Float) : 0.0052
                     pathHash (String) : "environment/decorations/water/puddles/puddle_round_calm.w2mesh"
                 2 (CSectorDataResource)
-                    box0 (Float) : -0.5334
-                    box1 (Float) : -0.5060
+                    box0 (Float) : -0.5323
+                    box1 (Float) : -0.5050
                     box2 (Float) : -0.0033
-                    box3 (Float) : 0.5372
-                    box4 (Float) : 0.5201
-                    box5 (Float) : 0.5948
+                    box3 (Float) : 0.5356
+                    box4 (Float) : 0.5194
+                    box5 (Float) : 0.5938
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/containers/washtubs/gen_washtub_clothes.w2mesh"
             Objects (CBufferVLQInt32:CSectorDataObject)
                 0 (CSectorDataObject)
@@ -290,17 +290,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 64
-                    positionX (Float) : -340.6432
+                    positionX (Float) : -340.6435
                     positionY (Float) : -1304.2920
-                    positionZ (Float) : 29.8599
+                    positionZ (Float) : 29.8594
                 2 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 128
-                    positionX (Float) : -340.6432
+                    positionX (Float) : -340.6435
                     positionY (Float) : -1304.2920
-                    positionZ (Float) : 29.8599
+                    positionZ (Float) : 29.8594
             blocksize (CVLQInt32) : 204
             BlockData (CCompressedBuffer:SBlockData)
                 0 (SBlockData)
