 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/architecture/streets_elements/rich_inner_arches/bob_rich_inner_arch_rail _01.w2mesh.txt 4.00/dlc/bob/data/environment/architecture/streets_elements/rich_inner_arches/bob_rich_inner_arch_rail _01.w2mesh.txt
index d8c7020..b1d3b28 100644
--- 1.32/dlc/bob/data/environment/architecture/streets_elements/rich_inner_arches/bob_rich_inner_arch_rail _01.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/architecture/streets_elements/rich_inner_arches/bob_rich_inner_arch_rail _01.w2mesh.txt	
@@ -7,7 +7,7 @@
                     X (Float) : -2.3157
                     Y (Float) : -0.6765
                     Z (Float) : -0.3006
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 2.3213
                     Y (Float) : 0.3112
@@ -24,10 +24,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 2860959000000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : 0.0008
+                    Y (Float) : -0.1856
+                    Z (Float) : -0.1994
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                 renderChunks (array:Uint8) : # array:Uint8
                 renderBuffer (DeferredDataBuffer)
@@ -45,6 +45,8 @@
                 vertexBufferSize (Uint32) : 79184
                 indexBufferSize (Uint32) : 12000
                 indexBufferOffset (Uint32) : 79872
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
