 1 file changed, 283 insertions(+), 283 deletions(-)

diff --git 1.32/levels/skellige/faroe/crossroad stone/stone.w2l.txt 4.00/levels/skellige/faroe/crossroad stone/stone.w2l.txt
index d9a4278..5d8759e 100644
--- 1.32/levels/skellige/faroe/crossroad stone/stone.w2l.txt	
+++ 4.00/levels/skellige/faroe/crossroad stone/stone.w2l.txt	
@@ -35,12 +35,12 @@
                     box5 (Float) : 0.0000
                     pathHash (String) : ""
                 1 (CSectorDataResource)
-                    box0 (Float) : -3.8709
-                    box1 (Float) : -4.1971
-                    box2 (Float) : -0.7037
-                    box3 (Float) : 3.7181
-                    box4 (Float) : 4.9346
-                    box5 (Float) : 0.5998
+                    box0 (Float) : -3.8449
+                    box1 (Float) : -4.1831
+                    box2 (Float) : -0.4626
+                    box3 (Float) : 3.7264
+                    box4 (Float) : 4.9094
+                    box5 (Float) : 0.5735
                     pathHash (String) : "environment/terrain_surroundings/common/rock_debris.w2mesh"
                 2 (CSectorDataResource)
                     box0 (Float) : -2.9409
@@ -64,65 +64,65 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 0
-                    positionX (Float) : 2128.3570
-                    positionY (Float) : -2015.7550
-                    positionZ (Float) : 16.6047
+                    positionX (Float) : 2128.3760
+                    positionY (Float) : -2015.7060
+                    positionZ (Float) : 16.6792
                 1 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 64
-                    positionX (Float) : 2128.3570
-                    positionY (Float) : -2015.7550
-                    positionZ (Float) : 16.6047
+                    positionX (Float) : 2128.3760
+                    positionY (Float) : -2015.7060
+                    positionZ (Float) : 16.6792
                 2 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 140
-                    positionX (Float) : 2092.5210
-                    positionY (Float) : -1953.9420
-                    positionZ (Float) : 7.1944
+                    positionX (Float) : 2092.5340
+                    positionY (Float) : -1953.9250
+                    positionZ (Float) : 7.3012
                 3 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 204
-                    positionX (Float) : 2092.5210
-                    positionY (Float) : -1953.9420
-                    positionZ (Float) : 7.1944
+                    positionX (Float) : 2092.5340
+                    positionY (Float) : -1953.9250
+                    positionZ (Float) : 7.3012
                 4 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 280
-                    positionX (Float) : 2091.2340
-                    positionY (Float) : -1963.1980
-                    positionZ (Float) : 8.3449
+                    positionX (Float) : 2091.2370
+                    positionY (Float) : -1963.1330
+                    positionZ (Float) : 8.4316
                 5 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 344
-                    positionX (Float) : 2091.2340
-                    positionY (Float) : -1963.1980
-                    positionZ (Float) : 8.3449
+                    positionX (Float) : 2091.2370
+                    positionY (Float) : -1963.1330
+                    positionZ (Float) : 8.4316
                 6 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 420
-                    positionX (Float) : 2130.2640
-                    positionY (Float) : -2008.0850
-                    positionZ (Float) : 15.1798
+                    positionX (Float) : 2130.3190
+                    positionY (Float) : -2008.0690
+                    positionZ (Float) : 15.2725
                 7 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 484
-                    positionX (Float) : 2130.2640
-                    positionY (Float) : -2008.0850
-                    positionZ (Float) : 15.1798
+                    positionX (Float) : 2130.3190
+                    positionY (Float) : -2008.0690
+                    positionZ (Float) : 15.2725
                 8 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -142,7 +142,7 @@
                 10 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 108
+                    radius (Uint16) : 142
                     offset (Uint64) : 700
                     positionX (Float) : 2106.7420
                     positionY (Float) : -1951.7940
@@ -152,673 +152,673 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 776
-                    positionX (Float) : 2135.5920
-                    positionY (Float) : -1961.7600
-                    positionZ (Float) : 10.9031
+                    positionX (Float) : 2135.6090
+                    positionY (Float) : -1961.6910
+                    positionZ (Float) : 11.0130
                 12 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 840
-                    positionX (Float) : 2135.5920
-                    positionY (Float) : -1961.7600
-                    positionZ (Float) : 10.9031
+                    positionX (Float) : 2135.6090
+                    positionY (Float) : -1961.6910
+                    positionZ (Float) : 11.0130
                 13 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 916
-                    positionX (Float) : 2103.8090
-                    positionY (Float) : -1968.0800
-                    positionZ (Float) : 11.0024
+                    positionX (Float) : 2103.8840
+                    positionY (Float) : -1968.0640
+                    positionZ (Float) : 11.0798
                 14 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 980
-                    positionX (Float) : 2103.8090
-                    positionY (Float) : -1968.0800
-                    positionZ (Float) : 11.0024
+                    positionX (Float) : 2103.8840
+                    positionY (Float) : -1968.0640
+                    positionZ (Float) : 11.0798
                 15 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1056
-                    positionX (Float) : 2109.4350
-                    positionY (Float) : -1960.1600
-                    positionZ (Float) : 4.3597
+                    positionX (Float) : 2109.4380
+                    positionY (Float) : -1960.0970
+                    positionZ (Float) : 4.4482
                 16 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1120
-                    positionX (Float) : 2109.4350
-                    positionY (Float) : -1960.1600
-                    positionZ (Float) : 4.3597
+                    positionX (Float) : 2109.4380
+                    positionY (Float) : -1960.0970
+                    positionZ (Float) : 4.4482
                 17 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1196
-                    positionX (Float) : 2116.0100
-                    positionY (Float) : -1988.6620
-                    positionZ (Float) : 13.7167
+                    positionX (Float) : 2116.0300
+                    positionY (Float) : -1988.6080
+                    positionZ (Float) : 13.8090
                 18 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1260
-                    positionX (Float) : 2116.0100
-                    positionY (Float) : -1988.6620
-                    positionZ (Float) : 13.7167
+                    positionX (Float) : 2116.0300
+                    positionY (Float) : -1988.6080
+                    positionZ (Float) : 13.8090
                 19 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1336
-                    positionX (Float) : 2128.5430
-                    positionY (Float) : -2000.8490
-                    positionZ (Float) : 13.4699
+                    positionX (Float) : 2128.5740
+                    positionY (Float) : -2000.7980
+                    positionZ (Float) : 13.5611
                 20 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1400
-                    positionX (Float) : 2128.5430
-                    positionY (Float) : -2000.8490
-                    positionZ (Float) : 13.4699
+                    positionX (Float) : 2128.5740
+                    positionY (Float) : -2000.7980
+                    positionZ (Float) : 13.5611
                 21 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1476
-                    positionX (Float) : 2090.1840
-                    positionY (Float) : -1955.9250
-                    positionZ (Float) : 7.9013
+                    positionX (Float) : 2090.1770
+                    positionY (Float) : -1955.8890
+                    positionZ (Float) : 8.0039
                 22 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1540
-                    positionX (Float) : 2090.1840
-                    positionY (Float) : -1955.9250
-                    positionZ (Float) : 7.9013
+                    positionX (Float) : 2090.1770
+                    positionY (Float) : -1955.8890
+                    positionZ (Float) : 8.0039
                 23 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1616
-                    positionX (Float) : 2077.4620
-                    positionY (Float) : -1957.0420
-                    positionZ (Float) : 8.9533
+                    positionX (Float) : 2077.4960
+                    positionY (Float) : -1957.0200
+                    positionZ (Float) : 9.0546
                 24 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1680
-                    positionX (Float) : 2077.4620
-                    positionY (Float) : -1957.0420
-                    positionZ (Float) : 8.9533
+                    positionX (Float) : 2077.4960
+                    positionY (Float) : -1957.0200
+                    positionZ (Float) : 9.0546
                 25 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1756
-                    positionX (Float) : 2067.6150
-                    positionY (Float) : -1969.4260
-                    positionZ (Float) : 10.3672
+                    positionX (Float) : 2067.6180
+                    positionY (Float) : -1969.3990
+                    positionZ (Float) : 10.4727
                 26 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1820
-                    positionX (Float) : 2067.6150
-                    positionY (Float) : -1969.4260
-                    positionZ (Float) : 10.3672
+                    positionX (Float) : 2067.6180
+                    positionY (Float) : -1969.3990
+                    positionZ (Float) : 10.4727
                 27 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1896
-                    positionX (Float) : 2071.7800
-                    positionY (Float) : -1983.1750
-                    positionZ (Float) : 10.8093
+                    positionX (Float) : 2071.7760
+                    positionY (Float) : -1983.1080
+                    positionZ (Float) : 10.8949
                 28 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 1960
-                    positionX (Float) : 2071.7800
-                    positionY (Float) : -1983.1750
-                    positionZ (Float) : 10.8093
+                    positionX (Float) : 2071.7760
+                    positionY (Float) : -1983.1080
+                    positionZ (Float) : 10.8949
                 29 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2036
-                    positionX (Float) : 2064.5630
-                    positionY (Float) : -1981.0190
-                    positionZ (Float) : 11.1843
+                    positionX (Float) : 2064.5740
+                    positionY (Float) : -1980.9960
+                    positionZ (Float) : 11.2902
                 30 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2100
-                    positionX (Float) : 2064.5630
-                    positionY (Float) : -1981.0190
-                    positionZ (Float) : 11.1843
+                    positionX (Float) : 2064.5740
+                    positionY (Float) : -1980.9960
+                    positionZ (Float) : 11.2902
                 31 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2176
-                    positionX (Float) : 2035.2400
-                    positionY (Float) : -2006.1080
-                    positionZ (Float) : 20.3944
+                    positionX (Float) : 2035.2050
+                    positionY (Float) : -2006.0530
+                    positionZ (Float) : 20.4813
                 32 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2240
-                    positionX (Float) : 2035.2400
-                    positionY (Float) : -2006.1080
-                    positionZ (Float) : 20.3944
+                    positionX (Float) : 2035.2050
+                    positionY (Float) : -2006.0530
+                    positionZ (Float) : 20.4813
                 33 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2316
-                    positionX (Float) : 2054.3270
-                    positionY (Float) : -2009.6050
-                    positionZ (Float) : 20.3184
+                    positionX (Float) : 2054.3040
+                    positionY (Float) : -2009.5830
+                    positionZ (Float) : 20.4223
                 34 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2380
-                    positionX (Float) : 2054.3270
-                    positionY (Float) : -2009.6050
-                    positionZ (Float) : 20.3184
+                    positionX (Float) : 2054.3040
+                    positionY (Float) : -2009.5830
+                    positionZ (Float) : 20.4223
                 35 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2456
-                    positionX (Float) : 2088.6200
-                    positionY (Float) : -2014.3180
-                    positionZ (Float) : 22.6923
+                    positionX (Float) : 2088.6050
+                    positionY (Float) : -2014.3400
+                    positionZ (Float) : 22.7980
                 36 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2520
-                    positionX (Float) : 2088.6200
-                    positionY (Float) : -2014.3180
-                    positionZ (Float) : 22.6923
+                    positionX (Float) : 2088.6050
+                    positionY (Float) : -2014.3400
+                    positionZ (Float) : 22.7980
                 37 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2596
-                    positionX (Float) : 2093.4560
-                    positionY (Float) : -2016.7190
-                    positionZ (Float) : 23.1233
+                    positionX (Float) : 2093.4530
+                    positionY (Float) : -2016.7670
+                    positionZ (Float) : 23.2007
                 38 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2660
-                    positionX (Float) : 2093.4560
-                    positionY (Float) : -2016.7190
-                    positionZ (Float) : 23.1233
+                    positionX (Float) : 2093.4530
+                    positionY (Float) : -2016.7670
+                    positionZ (Float) : 23.2007
                 39 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2736
-                    positionX (Float) : 2092.8490
-                    positionY (Float) : -2018.4130
-                    positionZ (Float) : 22.7827
+                    positionX (Float) : 2092.8610
+                    positionY (Float) : -2018.3980
+                    positionZ (Float) : 22.8718
                 40 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2800
-                    positionX (Float) : 2092.8490
-                    positionY (Float) : -2018.4130
-                    positionZ (Float) : 22.7827
+                    positionX (Float) : 2092.8610
+                    positionY (Float) : -2018.3980
+                    positionZ (Float) : 22.8718
                 41 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2876
-                    positionX (Float) : 2107.6130
-                    positionY (Float) : -2019.1880
-                    positionZ (Float) : 20.5731
+                    positionX (Float) : 2107.6170
+                    positionY (Float) : -2019.2080
+                    positionZ (Float) : 20.6620
                 42 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 2940
-                    positionX (Float) : 2107.6130
-                    positionY (Float) : -2019.1880
-                    positionZ (Float) : 20.5731
+                    positionX (Float) : 2107.6170
+                    positionY (Float) : -2019.2080
+                    positionZ (Float) : 20.6620
                 43 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3016
-                    positionX (Float) : 2115.1470
-                    positionY (Float) : -2016.3720
-                    positionZ (Float) : 21.2797
+                    positionX (Float) : 2115.1560
+                    positionY (Float) : -2016.3940
+                    positionZ (Float) : 21.3678
                 44 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3080
-                    positionX (Float) : 2115.1470
-                    positionY (Float) : -2016.3720
-                    positionZ (Float) : 21.2797
+                    positionX (Float) : 2115.1560
+                    positionY (Float) : -2016.3940
+                    positionZ (Float) : 21.3678
                 45 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3156
-                    positionX (Float) : 2126.8110
-                    positionY (Float) : -2023.2600
-                    positionZ (Float) : 20.9066
+                    positionX (Float) : 2126.8120
+                    positionY (Float) : -2023.2160
+                    positionZ (Float) : 20.9864
                 46 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3220
-                    positionX (Float) : 2126.8110
-                    positionY (Float) : -2023.2600
-                    positionZ (Float) : 20.9066
+                    positionX (Float) : 2126.8120
+                    positionY (Float) : -2023.2160
+                    positionZ (Float) : 20.9864
                 47 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3296
-                    positionX (Float) : 2097.9480
-                    positionY (Float) : -1952.6730
-                    positionZ (Float) : 6.3281
+                    positionX (Float) : 2098.0000
+                    positionY (Float) : -1952.6460
+                    positionZ (Float) : 6.4204
                 48 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3360
-                    positionX (Float) : 2097.9480
-                    positionY (Float) : -1952.6730
-                    positionZ (Float) : 6.3281
+                    positionX (Float) : 2098.0000
+                    positionY (Float) : -1952.6460
+                    positionZ (Float) : 6.4204
                 49 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3436
-                    positionX (Float) : 2156.9310
-                    positionY (Float) : -2012.4300
-                    positionZ (Float) : 3.9937
+                    positionX (Float) : 2157.0110
+                    positionY (Float) : -2012.4470
+                    positionZ (Float) : 4.0652
                 50 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3500
-                    positionX (Float) : 2156.9310
-                    positionY (Float) : -2012.4300
-                    positionZ (Float) : 3.9937
+                    positionX (Float) : 2157.0110
+                    positionY (Float) : -2012.4470
+                    positionZ (Float) : 4.0652
                 51 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3576
-                    positionX (Float) : 2160.4090
-                    positionY (Float) : -2005.3150
-                    positionZ (Float) : 4.0915
+                    positionX (Float) : 2160.4780
+                    positionY (Float) : -2005.3240
+                    positionZ (Float) : 4.1756
                 52 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3640
-                    positionX (Float) : 2160.4090
-                    positionY (Float) : -2005.3150
-                    positionZ (Float) : 4.0915
+                    positionX (Float) : 2160.4780
+                    positionY (Float) : -2005.3240
+                    positionZ (Float) : 4.1756
                 53 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3716
-                    positionX (Float) : 2162.7610
-                    positionY (Float) : -1995.6550
-                    positionZ (Float) : 3.8614
+                    positionX (Float) : 2162.8180
+                    positionY (Float) : -1995.6490
+                    positionZ (Float) : 3.9543
                 54 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3780
-                    positionX (Float) : 2162.7610
-                    positionY (Float) : -1995.6550
-                    positionZ (Float) : 3.8614
+                    positionX (Float) : 2162.8180
+                    positionY (Float) : -1995.6490
+                    positionZ (Float) : 3.9543
                 55 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3856
-                    positionX (Float) : 2171.2660
-                    positionY (Float) : -1981.6160
-                    positionZ (Float) : 4.2121
+                    positionX (Float) : 2171.1960
+                    positionY (Float) : -1981.6150
+                    positionZ (Float) : 4.2956
                 56 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3920
-                    positionX (Float) : 2171.2660
-                    positionY (Float) : -1981.6160
-                    positionZ (Float) : 4.2121
+                    positionX (Float) : 2171.1960
+                    positionY (Float) : -1981.6150
+                    positionZ (Float) : 4.2956
                 57 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 3996
-                    positionX (Float) : 2158.4720
-                    positionY (Float) : -1971.1470
-                    positionZ (Float) : 3.9016
+                    positionX (Float) : 2158.5230
+                    positionY (Float) : -1971.0910
+                    positionZ (Float) : 4.0085
                 58 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4060
-                    positionX (Float) : 2158.4720
-                    positionY (Float) : -1971.1470
-                    positionZ (Float) : 3.9016
+                    positionX (Float) : 2158.5230
+                    positionY (Float) : -1971.0910
+                    positionZ (Float) : 4.0085
                 59 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4136
-                    positionX (Float) : 2153.0590
-                    positionY (Float) : -1972.7070
-                    positionZ (Float) : 7.4922
+                    positionX (Float) : 2153.1220
+                    positionY (Float) : -1972.6590
+                    positionZ (Float) : 7.5973
                 60 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4200
-                    positionX (Float) : 2153.0590
-                    positionY (Float) : -1972.7070
-                    positionZ (Float) : 7.4922
+                    positionX (Float) : 2153.1220
+                    positionY (Float) : -1972.6590
+                    positionZ (Float) : 7.5973
                 61 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4276
-                    positionX (Float) : 2159.7500
-                    positionY (Float) : -1991.3300
-                    positionZ (Float) : 5.9688
+                    positionX (Float) : 2159.8130
+                    positionY (Float) : -1991.3120
+                    positionZ (Float) : 6.0557
                 62 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4340
-                    positionX (Float) : 2159.7500
-                    positionY (Float) : -1991.3300
-                    positionZ (Float) : 5.9688
+                    positionX (Float) : 2159.8130
+                    positionY (Float) : -1991.3120
+                    positionZ (Float) : 6.0557
                 63 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4416
-                    positionX (Float) : 2170.8880
-                    positionY (Float) : -2003.6260
-                    positionZ (Float) : 2.9708
+                    positionX (Float) : 2170.8260
+                    positionY (Float) : -2003.6280
+                    positionZ (Float) : 3.0605
                 64 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4480
-                    positionX (Float) : 2170.8880
-                    positionY (Float) : -2003.6260
-                    positionZ (Float) : 2.9708
+                    positionX (Float) : 2170.8260
+                    positionY (Float) : -2003.6280
+                    positionZ (Float) : 3.0605
                 65 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4556
-                    positionX (Float) : 2162.1130
-                    positionY (Float) : -1981.2190
-                    positionZ (Float) : 4.6747
+                    positionX (Float) : 2162.1580
+                    positionY (Float) : -1981.1830
+                    positionZ (Float) : 4.7675
                 66 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4620
-                    positionX (Float) : 2162.1130
-                    positionY (Float) : -1981.2190
-                    positionZ (Float) : 4.6747
+                    positionX (Float) : 2162.1580
+                    positionY (Float) : -1981.1830
+                    positionZ (Float) : 4.7675
                 67 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4696
-                    positionX (Float) : 2171.8410
-                    positionY (Float) : -1973.8660
-                    positionZ (Float) : 3.8633
+                    positionX (Float) : 2171.7650
+                    positionY (Float) : -1973.8810
+                    positionZ (Float) : 3.9389
                 68 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4760
-                    positionX (Float) : 2171.8410
-                    positionY (Float) : -1973.8660
-                    positionZ (Float) : 3.8633
+                    positionX (Float) : 2171.7650
+                    positionY (Float) : -1973.8810
+                    positionZ (Float) : 3.9389
                 69 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4836
-                    positionX (Float) : 2168.3180
-                    positionY (Float) : -1967.6860
-                    positionZ (Float) : 2.4333
+                    positionX (Float) : 2168.2660
+                    positionY (Float) : -1967.7380
+                    positionZ (Float) : 2.5137
                 70 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4900
-                    positionX (Float) : 2168.3180
-                    positionY (Float) : -1967.6860
-                    positionZ (Float) : 2.4333
+                    positionX (Float) : 2168.2660
+                    positionY (Float) : -1967.7380
+                    positionZ (Float) : 2.5137
                 71 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 4976
-                    positionX (Float) : 2165.2560
-                    positionY (Float) : -1960.0500
-                    positionZ (Float) : 2.5972
+                    positionX (Float) : 2165.2260
+                    positionY (Float) : -1960.0850
+                    positionZ (Float) : 2.6957
                 72 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5040
-                    positionX (Float) : 2165.2560
-                    positionY (Float) : -1960.0500
-                    positionZ (Float) : 2.5972
+                    positionX (Float) : 2165.2260
+                    positionY (Float) : -1960.0850
+                    positionZ (Float) : 2.6957
                 73 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5116
-                    positionX (Float) : 2154.1920
-                    positionY (Float) : -1965.8240
-                    positionZ (Float) : 4.6201
+                    positionX (Float) : 2154.2390
+                    positionY (Float) : -1965.7800
+                    positionZ (Float) : 4.7345
                 74 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5180
-                    positionX (Float) : 2154.1920
-                    positionY (Float) : -1965.8240
-                    positionZ (Float) : 4.6201
+                    positionX (Float) : 2154.2390
+                    positionY (Float) : -1965.7800
+                    positionZ (Float) : 4.7345
                 75 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5256
-                    positionX (Float) : 2181.2320
-                    positionY (Float) : -1924.1860
-                    positionZ (Float) : 6.3167
+                    positionX (Float) : 2181.2010
+                    positionY (Float) : -1924.1240
+                    positionZ (Float) : 6.4281
                 76 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5320
-                    positionX (Float) : 2181.2320
-                    positionY (Float) : -1924.1860
-                    positionZ (Float) : 6.3167
+                    positionX (Float) : 2181.2010
+                    positionY (Float) : -1924.1240
+                    positionZ (Float) : 6.4281
                 77 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5396
-                    positionX (Float) : 2182.4920
-                    positionY (Float) : -1916.3370
-                    positionZ (Float) : 2.8935
+                    positionX (Float) : 2182.4660
+                    positionY (Float) : -1916.3020
+                    positionZ (Float) : 3.0173
                 78 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5460
-                    positionX (Float) : 2182.4920
-                    positionY (Float) : -1916.3370
-                    positionZ (Float) : 2.8935
+                    positionX (Float) : 2182.4660
+                    positionY (Float) : -1916.3020
+                    positionZ (Float) : 3.0173
                 79 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5536
-                    positionX (Float) : 2190.7550
-                    positionY (Float) : -1924.0210
-                    positionZ (Float) : 7.5854
+                    positionX (Float) : 2190.7040
+                    positionY (Float) : -1923.9830
+                    positionZ (Float) : 7.7004
                 80 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5600
-                    positionX (Float) : 2190.7550
-                    positionY (Float) : -1924.0210
-                    positionZ (Float) : 7.5854
+                    positionX (Float) : 2190.7040
+                    positionY (Float) : -1923.9830
+                    positionZ (Float) : 7.7004
                 81 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5676
-                    positionX (Float) : 2174.6540
-                    positionY (Float) : -1923.9290
-                    positionZ (Float) : 5.0935
+                    positionX (Float) : 2174.6060
+                    positionY (Float) : -1923.8890
+                    positionZ (Float) : 5.2091
                 82 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5740
-                    positionX (Float) : 2174.6540
-                    positionY (Float) : -1923.9290
-                    positionZ (Float) : 5.0935
+                    positionX (Float) : 2174.6060
+                    positionY (Float) : -1923.8890
+                    positionZ (Float) : 5.2091
                 83 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5816
-                    positionX (Float) : 2214.9350
-                    positionY (Float) : -1913.0590
-                    positionZ (Float) : 11.6771
+                    positionX (Float) : 2214.8770
+                    positionY (Float) : -1913.0730
+                    positionZ (Float) : 11.7943
                 84 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5880
-                    positionX (Float) : 2214.9350
-                    positionY (Float) : -1913.0590
-                    positionZ (Float) : 11.6771
+                    positionX (Float) : 2214.8770
+                    positionY (Float) : -1913.0730
+                    positionZ (Float) : 11.7943
                 85 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 5956
-                    positionX (Float) : 2194.5010
-                    positionY (Float) : -1901.7320
-                    positionZ (Float) : 6.0634
+                    positionX (Float) : 2194.4540
+                    positionY (Float) : -1901.7740
+                    positionZ (Float) : 6.1784
                 86 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 6020
-                    positionX (Float) : 2194.5010
-                    positionY (Float) : -1901.7320
-                    positionZ (Float) : 6.0634
+                    positionX (Float) : 2194.4540
+                    positionY (Float) : -1901.7740
+                    positionZ (Float) : 6.1784
                 87 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 6096
-                    positionX (Float) : 2202.3930
-                    positionY (Float) : -1906.7640
-                    positionZ (Float) : 7.5164
+                    positionX (Float) : 2202.4030
+                    positionY (Float) : -1906.8010
+                    positionZ (Float) : 7.6421
                 88 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 6160
-                    positionX (Float) : 2202.3930
-                    positionY (Float) : -1906.7640
-                    positionZ (Float) : 7.5164
+                    positionX (Float) : 2202.4030
+                    positionY (Float) : -1906.8010
+                    positionZ (Float) : 7.6421
                 89 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 6236
-                    positionX (Float) : 2189.1260
-                    positionY (Float) : -1895.8160
-                    positionZ (Float) : 5.2146
+                    positionX (Float) : 2189.0930
+                    positionY (Float) : -1895.8390
+                    positionZ (Float) : 5.3398
                 90 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 6300
-                    positionX (Float) : 2189.1260
-                    positionY (Float) : -1895.8160
-                    positionZ (Float) : 5.2146
+                    positionX (Float) : 2189.0930
+                    positionY (Float) : -1895.8390
+                    positionZ (Float) : 5.3398
                 91 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 6376
-                    positionX (Float) : 2208.3370
-                    positionY (Float) : -1919.4410
-                    positionZ (Float) : 8.7883
+                    positionX (Float) : 2208.2860
+                    positionY (Float) : -1919.4620
+                    positionZ (Float) : 8.9074
                 92 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 6440
-                    positionX (Float) : 2208.3370
-                    positionY (Float) : -1919.4410
-                    positionZ (Float) : 8.7883
+                    positionX (Float) : 2208.2860
+                    positionY (Float) : -1919.4620
+                    positionZ (Float) : 8.9074
                 93 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 6516
-                    positionX (Float) : 2206.9340
-                    positionY (Float) : -1911.0980
-                    positionZ (Float) : 8.3520
+                    positionX (Float) : 2206.8770
+                    positionY (Float) : -1911.1120
+                    positionZ (Float) : 8.4693
                 94 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 6580
-                    positionX (Float) : 2206.9340
-                    positionY (Float) : -1911.0980
-                    positionZ (Float) : 8.3520
+                    positionX (Float) : 2206.8770
+                    positionY (Float) : -1911.1120
+                    positionZ (Float) : 8.4693
             blocksize (CVLQInt32) : 6656
             BlockData (CCompressedBuffer:SBlockData)
                 0 (SBlockData)
