 1 file changed, 37 insertions(+), 7 deletions(-)

diff --git 1.32/environment/vegetation/brushes/michal stelmachowicz/grass1.vbrush.txt 4.00/environment/vegetation/brushes/michal stelmachowicz/grass1.vbrush.txt
index b3deb3b..957e827 100644
--- 1.32/environment/vegetation/brushes/michal stelmachowicz/grass1.vbrush.txt	
+++ 4.00/environment/vegetation/brushes/michal stelmachowicz/grass1.vbrush.txt	
@@ -4,18 +4,48 @@
                 0 (ptr:CVegetationBrushEntry) : CVegetationBrushEntry #1
                 1 (ptr:CVegetationBrushEntry) : CVegetationBrushEntry #2
                 2 (ptr:CVegetationBrushEntry) : CVegetationBrushEntry #3
+                3 (ptr:CVegetationBrushEntry) : CVegetationBrushEntry #4
+                4 (ptr:CVegetationBrushEntry) : CVegetationBrushEntry #5
+                5 (ptr:CVegetationBrushEntry) : CVegetationBrushEntry #6
+                6 (ptr:CVegetationBrushEntry) : CVegetationBrushEntry #7
     CVegetationBrushEntry #1 (CVegetationBrushEntry) : 
         CVegetationBrushEntry (CVegetationBrushEntry)
-            resource (handle:CSRTBaseTree) : environment/vegetation/grass/grass_common/common_lowest.srt
-            sizeVar (Float) : 0.2000
+            resource (handle:CSRTBaseTree) : environment/vegetation/flowers_new/flowers_new/grass4.srt
+            size (Float) : 0.6000
+            sizeVar (Float) : 0.6000
             radiusScale (Float) : 3.0000
     CVegetationBrushEntry #2 (CVegetationBrushEntry) : 
         CVegetationBrushEntry (CVegetationBrushEntry)
-            resource (handle:CSRTBaseTree) : environment/vegetation/grass/grass_common/low_thick.srt
-            sizeVar (Float) : 0.2000
-            radiusScale (Float) : 3.0000
+            resource (handle:CSRTBaseTree) : environment/vegetation/flowers_new/flowers_new/crownbeard.srt
+            size (Float) : 0.7000
+            sizeVar (Float) : 0.3000
+            radiusScale (Float) : 4.0000
     CVegetationBrushEntry #3 (CVegetationBrushEntry) : 
         CVegetationBrushEntry (CVegetationBrushEntry)
-            resource (handle:CSRTBaseTree) : environment/vegetation/grass/grass_common/low_thick_b_cluster.srt
-            sizeVar (Float) : 1.0000
+            resource (handle:CSRTBaseTree) : environment/vegetation/flowers_new/flowers_new/weed_red_high.srt
+            size (Float) : 0.5000
+            sizeVar (Float) : 0.5000
+            radiusScale (Float) : 20.0000
+    CVegetationBrushEntry #4 (CVegetationBrushEntry) : 
+        CVegetationBrushEntry (CVegetationBrushEntry)
+            resource (handle:CSRTBaseTree) : environment/vegetation/flowers_new/flowers_new/red_flower.srt
+            size (Float) : 1.2000
+            sizeVar (Float) : 0.4000
+            radiusScale (Float) : 10.0000
+    CVegetationBrushEntry #5 (CVegetationBrushEntry) : 
+        CVegetationBrushEntry (CVegetationBrushEntry)
+            resource (handle:CSRTBaseTree) : environment/vegetation/grass/grass_common/flowers_yellow_medium.srt
+            size (Float) : 0.8000
+            radiusScale (Float) : 10.0000
+    CVegetationBrushEntry #6 (CVegetationBrushEntry) : 
+        CVegetationBrushEntry (CVegetationBrushEntry)
+            resource (handle:CSRTBaseTree) : environment/vegetation/grass/grass_common_bob/green_low.srt
+            size (Float) : 0.4000
+            sizeVar (Float) : 0.8000
             radiusScale (Float) : 3.0000
+    CVegetationBrushEntry #7 (CVegetationBrushEntry) : 
+        CVegetationBrushEntry (CVegetationBrushEntry)
+            resource (handle:CSRTBaseTree) : environment/vegetation/grass/grass_common_bob/green_med.srt
+            size (Float) : 0.5000
+            sizeVar (Float) : 0.3000
+            radiusScale (Float) : 4.0000
