 1 file changed, 471 insertions(+), 471 deletions(-)

diff --git 1.32/levels/novigrad/novigrad/passiflora/passiflora garden.w2l.txt 4.00/levels/novigrad/novigrad/passiflora/passiflora garden.w2l.txt
index a80e5e6..dd28246 100644
--- 1.32/levels/novigrad/novigrad/passiflora/passiflora garden.w2l.txt	
+++ 4.00/levels/novigrad/novigrad/passiflora/passiflora garden.w2l.txt	
@@ -300,7 +300,7 @@
                                 W (Float) : 1.0000
                             Max (Vector)
                                 X (Float) : 683.7921
-                                Y (Float) : 2068.0890
+                                Y (Float) : 2068.0880
                                 Z (Float) : 35.7147
                                 W (Float) : 1.0000
                         drawableFlags (EDrawableFlags) : DF_IsVisible,DF_CastShadows
@@ -1644,12 +1644,12 @@
                     box5 (Float) : 0.0000
                     pathHash (String) : ""
                 1 (CSectorDataResource)
-                    box0 (Float) : -0.2313
-                    box1 (Float) : -0.2313
-                    box2 (Float) : -0.6577
-                    box3 (Float) : 0.2310
-                    box4 (Float) : 0.2309
-                    box5 (Float) : 0.6549
+                    box0 (Float) : -0.2333
+                    box1 (Float) : -0.2333
+                    box2 (Float) : -0.6549
+                    box3 (Float) : 0.2300
+                    box4 (Float) : 0.2300
+                    box5 (Float) : 0.6611
                     pathHash (String) : "environment/architecture/human/redania/novigrad/passiflora_brothel/very_rich_balcony_set/nov_very_rich_balcony_rail_support.w2mesh"
                 2 (CSectorDataResource)
                     box0 (Float) : -2.0174
@@ -1678,10 +1678,10 @@
                 5 (CSectorDataResource)
                     box0 (Float) : -2.2765
                     box1 (Float) : -5.8205
-                    box2 (Float) : -0.0928
+                    box2 (Float) : -0.0989
                     box3 (Float) : 1.4020
-                    box4 (Float) : 0.0002
-                    box5 (Float) : 0.0928
+                    box4 (Float) : 0.0375
+                    box5 (Float) : 0.0933
                     pathHash (String) : "environment/architecture/human/redania/novigrad/novigrad_street_assets/rich/rich_street_pavement_a_20_degrees_outer.w2mesh"
                 6 (CSectorDataResource)
                     box0 (Float) : -2.2470
@@ -1972,12 +1972,12 @@
                     box5 (Float) : 0.0250
                     pathHash (String) : "environment/decorations/wooden_parts/wooden_floor_planks_c.w2mesh"
                 42 (CSectorDataResource)
-                    box0 (Float) : -0.4632
-                    box1 (Float) : -0.2574
-                    box2 (Float) : -0.0316
-                    box3 (Float) : 0.2608
-                    box4 (Float) : 0.2984
-                    box5 (Float) : 0.1970
+                    box0 (Float) : -0.4625
+                    box1 (Float) : -0.2564
+                    box2 (Float) : -0.0075
+                    box3 (Float) : 0.2473
+                    box4 (Float) : 0.2775
+                    box5 (Float) : 0.1920
                     pathHash (String) : "environment/decorations/near_water/harbour_props/rope_piles_a.w2mesh"
                 43 (CSectorDataResource)
                     box0 (Float) : -0.3203
@@ -2012,12 +2012,12 @@
                     box5 (Float) : 0.1965
                     pathHash (String) : "environment/architecture/human/redania/novigrad/novigrad_street_assets/rich/rich_street_well_open_passiflora.w2mesh"
                 47 (CSectorDataResource)
-                    box0 (Float) : -0.9199
-                    box1 (Float) : -0.6196
-                    box2 (Float) : -0.0104
-                    box3 (Float) : 0.9310
-                    box4 (Float) : 0.7439
-                    box5 (Float) : 0.3062
+                    box0 (Float) : -0.8822
+                    box1 (Float) : -0.5755
+                    box2 (Float) : -0.0759
+                    box3 (Float) : 0.8707
+                    box4 (Float) : 0.6670
+                    box5 (Float) : 0.2923
                     pathHash (String) : "environment/terrain_surroundings/common/rock_debris_section_moss_small.w2mesh"
                 48 (CSectorDataResource)
                     box0 (Float) : -0.6337
@@ -2052,19 +2052,19 @@
                     box5 (Float) : 0.4131
                     pathHash (String) : "environment/architecture/human/common/bridges/old_village_bridge/old_village_bridge_wall_20_degrees.w2mesh"
                 52 (CSectorDataResource)
-                    box0 (Float) : -1.4605
-                    box1 (Float) : -0.7068
-                    box2 (Float) : -0.3330
-                    box3 (Float) : 1.4605
-                    box4 (Float) : 0.7068
-                    box5 (Float) : 0.3330
+                    box0 (Float) : -1.4630
+                    box1 (Float) : -0.7134
+                    box2 (Float) : -0.3444
+                    box3 (Float) : 1.4669
+                    box4 (Float) : 0.7472
+                    box5 (Float) : 0.3492
                     pathHash (String) : "environment/architecture/human/common/stairs/stone_stairs/stone_stairs_medium.w2mesh"
                 53 (CSectorDataResource)
-                    box0 (Float) : -0.4407
-                    box1 (Float) : -0.4362
+                    box0 (Float) : -0.4405
+                    box1 (Float) : -0.4360
                     box2 (Float) : -0.0003
-                    box3 (Float) : 0.4400
-                    box4 (Float) : 0.4356
+                    box3 (Float) : 0.4398
+                    box4 (Float) : 0.4354
                     box5 (Float) : 1.0997
                     pathHash (String) : "environment/decorations/containers/barrel/barrel_closed_metal.w2mesh"
                 54 (CSectorDataResource)
@@ -2076,15 +2076,15 @@
                     box5 (Float) : -0.3033
                     pathHash (String) : "environment/architecture/human/redania/novigrad/novigrad_street_assets/rich/rich_street_well_circle_passiflora.w2mesh"
                 55 (CSectorDataResource)
-                    box0 (Float) : -2.0000
-                    box1 (Float) : 0.1590
-                    box2 (Float) : 0.0330
-                    box3 (Float) : 2.0000
-                    box4 (Float) : 4.7639
-                    box5 (Float) : 0.0862
+                    box0 (Float) : -2.0593
+                    box1 (Float) : 0.1186
+                    box2 (Float) : -0.0591
+                    box3 (Float) : 2.0594
+                    box4 (Float) : 4.7926
+                    box5 (Float) : 0.1195
                     pathHash (String) : "environment/architecture/human/redania/novigrad/novigrad_harbour_shore_set/nov_harbour_pavement_pt_e.w2mesh"
                 56 (CSectorDataResource)
-                    box0 (Float) : -0.9486
+                    box0 (Float) : -0.9485
                     box1 (Float) : -0.9991
                     box2 (Float) : -0.0810
                     box3 (Float) : 0.9486
@@ -2097,17 +2097,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 0
-                    positionX (Float) : 675.6294
-                    positionY (Float) : 2118.1790
-                    positionZ (Float) : 33.0734
+                    positionX (Float) : 675.6281
+                    positionY (Float) : 2118.1770
+                    positionZ (Float) : 33.0779
                 1 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 64
-                    positionX (Float) : 675.6294
-                    positionY (Float) : 2118.1790
-                    positionZ (Float) : 33.0734
+                    positionX (Float) : 675.6281
+                    positionY (Float) : 2118.1770
+                    positionZ (Float) : 33.0779
                 2 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2129,17 +2129,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 280
-                    positionX (Float) : 687.4613
-                    positionY (Float) : 2092.7010
-                    positionZ (Float) : 33.0755
+                    positionX (Float) : 687.4628
+                    positionY (Float) : 2092.7030
+                    positionZ (Float) : 33.0800
                 5 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 344
-                    positionX (Float) : 687.4613
-                    positionY (Float) : 2092.7010
-                    positionZ (Float) : 33.0755
+                    positionX (Float) : 687.4628
+                    positionY (Float) : 2092.7030
+                    positionZ (Float) : 33.0800
                 6 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2161,17 +2161,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 560
-                    positionX (Float) : 684.7704
-                    positionY (Float) : 2092.4180
-                    positionZ (Float) : 33.0755
+                    positionX (Float) : 684.7716
+                    positionY (Float) : 2092.4200
+                    positionZ (Float) : 33.0800
                 9 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 624
-                    positionX (Float) : 684.7704
-                    positionY (Float) : 2092.4180
-                    positionZ (Float) : 33.0755
+                    positionX (Float) : 684.7716
+                    positionY (Float) : 2092.4200
+                    positionZ (Float) : 33.0800
                 10 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2225,49 +2225,49 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 1120
-                    positionX (Float) : 678.4283
-                    positionY (Float) : 2118.2070
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 678.4269
+                    positionY (Float) : 2118.2050
+                    positionZ (Float) : 31.8466
                 17 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 1184
-                    positionX (Float) : 678.4283
-                    positionY (Float) : 2118.2070
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 678.4269
+                    positionY (Float) : 2118.2050
+                    positionZ (Float) : 31.8466
                 18 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 1260
-                    positionX (Float) : 678.9988
-                    positionY (Float) : 2120.9220
-                    positionZ (Float) : 31.8950
+                    positionX (Float) : 678.9991
+                    positionY (Float) : 2120.9200
+                    positionZ (Float) : 31.8995
                 19 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 1324
-                    positionX (Float) : 678.9988
-                    positionY (Float) : 2120.9220
-                    positionZ (Float) : 31.8950
+                    positionX (Float) : 678.9991
+                    positionY (Float) : 2120.9200
+                    positionZ (Float) : 31.8995
                 20 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 1400
-                    positionX (Float) : 682.0114
-                    positionY (Float) : 2112.2750
-                    positionZ (Float) : 31.9493
+                    positionX (Float) : 682.0273
+                    positionY (Float) : 2112.2650
+                    positionZ (Float) : 31.9463
                 21 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 1464
-                    positionX (Float) : 682.0114
-                    positionY (Float) : 2112.2750
-                    positionZ (Float) : 31.9493
+                    positionX (Float) : 682.0273
+                    positionY (Float) : 2112.2650
+                    positionZ (Float) : 31.9463
                 22 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2289,33 +2289,33 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 1680
-                    positionX (Float) : 686.5372
-                    positionY (Float) : 2109.5100
-                    positionZ (Float) : 31.9140
+                    positionX (Float) : 686.5557
+                    positionY (Float) : 2109.5070
+                    positionZ (Float) : 31.9112
                 25 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 1744
-                    positionX (Float) : 686.5372
-                    positionY (Float) : 2109.5100
-                    positionZ (Float) : 31.9140
+                    positionX (Float) : 686.5557
+                    positionY (Float) : 2109.5070
+                    positionZ (Float) : 31.9112
                 26 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 1820
-                    positionX (Float) : 681.9307
-                    positionY (Float) : 2092.3530
-                    positionZ (Float) : 33.0755
+                    positionX (Float) : 681.9319
+                    positionY (Float) : 2092.3540
+                    positionZ (Float) : 33.0800
                 27 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 1884
-                    positionX (Float) : 681.9307
-                    positionY (Float) : 2092.3530
-                    positionZ (Float) : 33.0755
+                    positionX (Float) : 681.9319
+                    positionY (Float) : 2092.3540
+                    positionZ (Float) : 33.0800
                 28 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2338,32 +2338,32 @@
                     radius (Uint16) : 85
                     offset (Uint64) : 2100
                     positionX (Float) : 693.5980
-                    positionY (Float) : 2113.8090
-                    positionZ (Float) : 31.8421
+                    positionY (Float) : 2113.8110
+                    positionZ (Float) : 31.8466
                 31 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 2164
                     positionX (Float) : 693.5980
-                    positionY (Float) : 2113.8090
-                    positionZ (Float) : 31.8421
+                    positionY (Float) : 2113.8110
+                    positionZ (Float) : 31.8466
                 32 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 2240
-                    positionX (Float) : 693.6241
+                    positionX (Float) : 693.6220
                     positionY (Float) : 2117.4540
-                    positionZ (Float) : 31.8421
+                    positionZ (Float) : 31.8466
                 33 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 2304
-                    positionX (Float) : 693.6241
+                    positionX (Float) : 693.6220
                     positionY (Float) : 2117.4540
-                    positionZ (Float) : 31.8421
+                    positionZ (Float) : 31.8466
                 34 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2385,17 +2385,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 2520
-                    positionX (Float) : 691.9623
-                    positionY (Float) : 2119.8400
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 691.9602
+                    positionY (Float) : 2119.8410
+                    positionZ (Float) : 31.8466
                 37 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 2584
-                    positionX (Float) : 691.9623
-                    positionY (Float) : 2119.8400
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 691.9602
+                    positionY (Float) : 2119.8410
+                    positionZ (Float) : 31.8466
                 38 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2417,17 +2417,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 2800
-                    positionX (Float) : 689.9371
-                    positionY (Float) : 2121.7300
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 689.9351
+                    positionY (Float) : 2121.7290
+                    positionZ (Float) : 31.8466
                 41 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 2864
-                    positionX (Float) : 689.9371
-                    positionY (Float) : 2121.7300
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 689.9351
+                    positionY (Float) : 2121.7290
+                    positionZ (Float) : 31.8466
                 42 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2449,17 +2449,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 3080
-                    positionX (Float) : 687.2199
-                    positionY (Float) : 2122.4820
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 687.2183
+                    positionY (Float) : 2122.4810
+                    positionZ (Float) : 31.8466
                 45 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 3144
-                    positionX (Float) : 687.2199
-                    positionY (Float) : 2122.4820
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 687.2183
+                    positionY (Float) : 2122.4810
+                    positionZ (Float) : 31.8466
                 46 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2481,17 +2481,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 3360
-                    positionX (Float) : 684.3610
-                    positionY (Float) : 2122.7520
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 684.3597
+                    positionY (Float) : 2122.7500
+                    positionZ (Float) : 31.8466
                 49 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 3424
-                    positionX (Float) : 684.3610
-                    positionY (Float) : 2122.7520
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 684.3597
+                    positionY (Float) : 2122.7500
+                    positionZ (Float) : 31.8466
                 50 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2561,49 +2561,49 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 4060
-                    positionX (Float) : 692.1577
-                    positionY (Float) : 2106.8990
-                    positionZ (Float) : 32.4214
+                    positionX (Float) : 692.1560
+                    positionY (Float) : 2106.8970
+                    positionZ (Float) : 32.4259
                 59 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 4124
-                    positionX (Float) : 692.1577
-                    positionY (Float) : 2106.8990
-                    positionZ (Float) : 32.4214
+                    positionX (Float) : 692.1560
+                    positionY (Float) : 2106.8970
+                    positionZ (Float) : 32.4259
                 60 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 4200
-                    positionX (Float) : 693.0710
-                    positionY (Float) : 2103.1960
-                    positionZ (Float) : 32.4031
+                    positionX (Float) : 693.0693
+                    positionY (Float) : 2103.1970
+                    positionZ (Float) : 32.4076
                 61 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 4264
-                    positionX (Float) : 693.0710
-                    positionY (Float) : 2103.1960
-                    positionZ (Float) : 32.4031
+                    positionX (Float) : 693.0693
+                    positionY (Float) : 2103.1970
+                    positionZ (Float) : 32.4076
                 62 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 4340
-                    positionX (Float) : 693.8694
-                    positionY (Float) : 2099.5240
-                    positionZ (Float) : 32.4214
+                    positionX (Float) : 693.8677
+                    positionY (Float) : 2099.5250
+                    positionZ (Float) : 32.4259
                 63 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 4404
-                    positionX (Float) : 693.8694
-                    positionY (Float) : 2099.5240
-                    positionZ (Float) : 32.4214
+                    positionX (Float) : 693.8677
+                    positionY (Float) : 2099.5250
+                    positionZ (Float) : 32.4259
                 64 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2657,49 +2657,49 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 4900
-                    positionX (Float) : 695.7692
+                    positionX (Float) : 695.7672
                     positionY (Float) : 2115.4890
-                    positionZ (Float) : 31.8421
+                    positionZ (Float) : 31.8466
                 71 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 4964
-                    positionX (Float) : 695.7692
+                    positionX (Float) : 695.7672
                     positionY (Float) : 2115.4890
-                    positionZ (Float) : 31.8421
+                    positionZ (Float) : 31.8466
                 72 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5040
-                    positionX (Float) : 692.6128
-                    positionY (Float) : 2111.4390
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 692.6117
+                    positionY (Float) : 2111.4410
+                    positionZ (Float) : 31.8466
                 73 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5104
-                    positionX (Float) : 692.6128
-                    positionY (Float) : 2111.4390
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 692.6117
+                    positionY (Float) : 2111.4410
+                    positionZ (Float) : 31.8466
                 74 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5180
-                    positionX (Float) : 692.3776
-                    positionY (Float) : 2108.8900
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 692.3764
+                    positionY (Float) : 2108.8920
+                    positionZ (Float) : 31.8466
                 75 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5244
-                    positionX (Float) : 692.3776
-                    positionY (Float) : 2108.8900
-                    positionZ (Float) : 31.8421
+                    positionX (Float) : 692.3764
+                    positionY (Float) : 2108.8920
+                    positionZ (Float) : 31.8466
                 76 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2721,33 +2721,33 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5460
-                    positionX (Float) : 693.7517
-                    positionY (Float) : 2095.7890
-                    positionZ (Float) : 32.4214
+                    positionX (Float) : 693.7504
+                    positionY (Float) : 2095.7900
+                    positionZ (Float) : 32.4259
                 79 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5524
-                    positionX (Float) : 693.7517
-                    positionY (Float) : 2095.7890
-                    positionZ (Float) : 32.4214
+                    positionX (Float) : 693.7504
+                    positionY (Float) : 2095.7900
+                    positionZ (Float) : 32.4259
                 80 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5600
-                    positionX (Float) : 702.0140
-                    positionY (Float) : 2107.5750
-                    positionZ (Float) : 29.1945
+                    positionX (Float) : 702.0121
+                    positionY (Float) : 2107.5760
+                    positionZ (Float) : 29.1990
                 81 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5664
-                    positionX (Float) : 702.0140
-                    positionY (Float) : 2107.5750
-                    positionZ (Float) : 29.1945
+                    positionX (Float) : 702.0121
+                    positionY (Float) : 2107.5760
+                    positionZ (Float) : 29.1990
                 82 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2769,17 +2769,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5880
-                    positionX (Float) : 700.5430
+                    positionX (Float) : 700.5409
                     positionY (Float) : 2109.9670
-                    positionZ (Float) : 29.1945
+                    positionZ (Float) : 29.1990
                 85 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 5944
-                    positionX (Float) : 700.5430
+                    positionX (Float) : 700.5409
                     positionY (Float) : 2109.9670
-                    positionZ (Float) : 29.1945
+                    positionZ (Float) : 29.1990
                 86 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2817,17 +2817,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 6300
-                    positionX (Float) : 703.2069
-                    positionY (Float) : 2105.0590
-                    positionZ (Float) : 29.1945
+                    positionX (Float) : 703.2050
+                    positionY (Float) : 2105.0600
+                    positionZ (Float) : 29.1990
                 91 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 6364
-                    positionX (Float) : 703.2069
-                    positionY (Float) : 2105.0590
-                    positionZ (Float) : 29.1945
+                    positionX (Float) : 703.2050
+                    positionY (Float) : 2105.0600
+                    positionZ (Float) : 29.1990
                 92 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3001,17 +3001,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 7904
-                    positionX (Float) : 670.0711
-                    positionY (Float) : 2118.0550
-                    positionZ (Float) : 33.0734
+                    positionX (Float) : 670.0697
+                    positionY (Float) : 2118.0530
+                    positionZ (Float) : 33.0779
                 114 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 7968
-                    positionX (Float) : 670.0711
-                    positionY (Float) : 2118.0550
-                    positionZ (Float) : 33.0734
+                    positionX (Float) : 670.0697
+                    positionY (Float) : 2118.0530
+                    positionZ (Float) : 33.0779
                 115 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3049,17 +3049,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 8324
-                    positionX (Float) : 672.8224
-                    positionY (Float) : 2118.1790
-                    positionZ (Float) : 33.0734
+                    positionX (Float) : 672.8209
+                    positionY (Float) : 2118.1770
+                    positionZ (Float) : 33.0779
                 120 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 8388
-                    positionX (Float) : 672.8224
-                    positionY (Float) : 2118.1790
-                    positionZ (Float) : 33.0734
+                    positionX (Float) : 672.8209
+                    positionY (Float) : 2118.1770
+                    positionZ (Float) : 33.0779
                 121 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3129,17 +3129,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 9024
-                    positionX (Float) : 681.5010
-                    positionY (Float) : 2122.3060
-                    positionZ (Float) : 31.8950
+                    positionX (Float) : 681.5001
+                    positionY (Float) : 2122.3040
+                    positionZ (Float) : 31.8995
                 130 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 9088
-                    positionX (Float) : 681.5010
-                    positionY (Float) : 2122.3060
-                    positionZ (Float) : 31.8950
+                    positionX (Float) : 681.5001
+                    positionY (Float) : 2122.3040
+                    positionZ (Float) : 31.8995
                 131 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3353,17 +3353,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 10984
-                    positionX (Float) : 689.2172
-                    positionY (Float) : 2109.5830
-                    positionZ (Float) : 32.4687
+                    positionX (Float) : 689.2156
+                    positionY (Float) : 2109.5820
+                    positionZ (Float) : 32.4733
                 158 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 11048
-                    positionX (Float) : 689.2172
-                    positionY (Float) : 2109.5830
-                    positionZ (Float) : 32.4687
+                    positionX (Float) : 689.2156
+                    positionY (Float) : 2109.5820
+                    positionZ (Float) : 32.4733
                 159 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3385,17 +3385,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 11264
-                    positionX (Float) : 686.3993
-                    positionY (Float) : 2110.3690
-                    positionZ (Float) : 32.4687
+                    positionX (Float) : 686.3973
+                    positionY (Float) : 2110.3680
+                    positionZ (Float) : 32.4733
                 162 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 11328
-                    positionX (Float) : 686.3993
-                    positionY (Float) : 2110.3690
-                    positionZ (Float) : 32.4687
+                    positionX (Float) : 686.3973
+                    positionY (Float) : 2110.3680
+                    positionZ (Float) : 32.4733
                 163 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3417,17 +3417,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 11544
-                    positionX (Float) : 683.8399
-                    positionY (Float) : 2111.7720
-                    positionZ (Float) : 32.4687
+                    positionX (Float) : 683.8378
+                    positionY (Float) : 2111.7710
+                    positionZ (Float) : 32.4733
                 166 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 11608
-                    positionX (Float) : 683.8399
-                    positionY (Float) : 2111.7720
-                    positionZ (Float) : 32.4687
+                    positionX (Float) : 683.8378
+                    positionY (Float) : 2111.7710
+                    positionZ (Float) : 32.4733
                 167 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3449,17 +3449,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 11824
-                    positionX (Float) : 681.5807
+                    positionX (Float) : 681.5786
                     positionY (Float) : 2113.8240
-                    positionZ (Float) : 32.4687
+                    positionZ (Float) : 32.4733
                 170 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 11888
-                    positionX (Float) : 681.5807
+                    positionX (Float) : 681.5786
                     positionY (Float) : 2113.8240
-                    positionZ (Float) : 32.4687
+                    positionZ (Float) : 32.4733
                 171 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3481,17 +3481,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 12104
-                    positionX (Float) : 687.6198
-                    positionY (Float) : 2095.8050
-                    positionZ (Float) : 33.0755
+                    positionX (Float) : 687.6183
+                    positionY (Float) : 2095.8060
+                    positionZ (Float) : 33.0800
                 174 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 12168
-                    positionX (Float) : 687.6198
-                    positionY (Float) : 2095.8050
-                    positionZ (Float) : 33.0755
+                    positionX (Float) : 687.6183
+                    positionY (Float) : 2095.8060
+                    positionZ (Float) : 33.0800
                 175 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3561,161 +3561,161 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 12804
-                    positionX (Float) : 665.1796
-                    positionY (Float) : 2067.9250
-                    positionZ (Float) : 33.0436
+                    positionX (Float) : 665.1782
+                    positionY (Float) : 2067.9230
+                    positionZ (Float) : 33.0481
                 184 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 12868
-                    positionX (Float) : 665.1796
-                    positionY (Float) : 2067.9250
-                    positionZ (Float) : 33.0436
+                    positionX (Float) : 665.1782
+                    positionY (Float) : 2067.9230
+                    positionZ (Float) : 33.0481
                 185 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 12944
-                    positionX (Float) : 655.4484
-                    positionY (Float) : 2064.9910
-                    positionZ (Float) : 33.8851
+                    positionX (Float) : 655.4476
+                    positionY (Float) : 2064.9900
+                    positionZ (Float) : 33.8896
                 186 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13008
-                    positionX (Float) : 655.4484
-                    positionY (Float) : 2064.9910
-                    positionZ (Float) : 33.8851
+                    positionX (Float) : 655.4476
+                    positionY (Float) : 2064.9900
+                    positionZ (Float) : 33.8896
                 187 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13084
-                    positionX (Float) : 667.9786
-                    positionY (Float) : 2068.1750
-                    positionZ (Float) : 33.0436
+                    positionX (Float) : 667.9773
+                    positionY (Float) : 2068.1730
+                    positionZ (Float) : 33.0481
                 188 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13148
-                    positionX (Float) : 667.9786
-                    positionY (Float) : 2068.1750
-                    positionZ (Float) : 33.0436
+                    positionX (Float) : 667.9773
+                    positionY (Float) : 2068.1730
+                    positionZ (Float) : 33.0481
                 189 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13224
-                    positionX (Float) : 674.6545
-                    positionY (Float) : 2068.3640
-                    positionZ (Float) : 33.9276
+                    positionX (Float) : 674.6531
+                    positionY (Float) : 2068.3630
+                    positionZ (Float) : 33.9321
                 190 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13288
-                    positionX (Float) : 674.6545
-                    positionY (Float) : 2068.3640
-                    positionZ (Float) : 33.9276
+                    positionX (Float) : 674.6531
+                    positionY (Float) : 2068.3630
+                    positionZ (Float) : 33.9321
                 191 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13364
-                    positionX (Float) : 671.8019
-                    positionY (Float) : 2068.3420
-                    positionZ (Float) : 33.2109
+                    positionX (Float) : 671.8005
+                    positionY (Float) : 2068.3410
+                    positionZ (Float) : 33.2154
                 192 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13428
-                    positionX (Float) : 671.8019
-                    positionY (Float) : 2068.3420
-                    positionZ (Float) : 33.2109
+                    positionX (Float) : 671.8005
+                    positionY (Float) : 2068.3410
+                    positionZ (Float) : 33.2154
                 193 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13504
-                    positionX (Float) : 677.6287
-                    positionY (Float) : 2068.1610
-                    positionZ (Float) : 34.4857
+                    positionX (Float) : 677.6272
+                    positionY (Float) : 2068.1600
+                    positionZ (Float) : 34.4902
                 194 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13568
-                    positionX (Float) : 677.6287
-                    positionY (Float) : 2068.1610
-                    positionZ (Float) : 34.4857
+                    positionX (Float) : 677.6272
+                    positionY (Float) : 2068.1600
+                    positionZ (Float) : 34.4902
                 195 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13644
-                    positionX (Float) : 683.7408
-                    positionY (Float) : 2067.6010
-                    positionZ (Float) : 35.3624
+                    positionX (Float) : 683.7391
+                    positionY (Float) : 2067.6000
+                    positionZ (Float) : 35.3669
                 196 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13708
-                    positionX (Float) : 683.7408
-                    positionY (Float) : 2067.6010
-                    positionZ (Float) : 35.3624
+                    positionX (Float) : 683.7391
+                    positionY (Float) : 2067.6000
+                    positionZ (Float) : 35.3669
                 197 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13784
-                    positionX (Float) : 686.6910
-                    positionY (Float) : 2067.1130
-                    positionZ (Float) : 35.6732
+                    positionX (Float) : 686.6893
+                    positionY (Float) : 2067.1120
+                    positionZ (Float) : 35.6777
                 198 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13848
-                    positionX (Float) : 686.6910
-                    positionY (Float) : 2067.1130
-                    positionZ (Float) : 35.6732
+                    positionX (Float) : 686.6893
+                    positionY (Float) : 2067.1120
+                    positionZ (Float) : 35.6777
                 199 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13924
-                    positionX (Float) : 689.6195
-                    positionY (Float) : 2066.5830
-                    positionZ (Float) : 35.9838
+                    positionX (Float) : 689.6176
+                    positionY (Float) : 2066.5820
+                    positionZ (Float) : 35.9883
                 200 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 13988
-                    positionX (Float) : 689.6195
-                    positionY (Float) : 2066.5830
-                    positionZ (Float) : 35.9838
+                    positionX (Float) : 689.6176
+                    positionY (Float) : 2066.5820
+                    positionZ (Float) : 35.9883
                 201 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 14064
-                    positionX (Float) : 680.7272
-                    positionY (Float) : 2068.0070
-                    positionZ (Float) : 34.8965
+                    positionX (Float) : 680.7257
+                    positionY (Float) : 2068.0050
+                    positionZ (Float) : 34.9010
                 202 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 14128
-                    positionX (Float) : 680.7272
-                    positionY (Float) : 2068.0070
-                    positionZ (Float) : 34.8965
+                    positionX (Float) : 680.7257
+                    positionY (Float) : 2068.0050
+                    positionZ (Float) : 34.9010
                 203 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3801,33 +3801,33 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 14904
-                    positionX (Float) : 662.1396
-                    positionY (Float) : 2067.4600
-                    positionZ (Float) : 33.0436
+                    positionX (Float) : 662.1386
+                    positionY (Float) : 2067.4580
+                    positionZ (Float) : 33.0481
                 214 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 14968
-                    positionX (Float) : 662.1396
-                    positionY (Float) : 2067.4600
-                    positionZ (Float) : 33.0436
+                    positionX (Float) : 662.1386
+                    positionY (Float) : 2067.4580
+                    positionZ (Float) : 33.0481
                 215 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 15044
-                    positionX (Float) : 658.4762
-                    positionY (Float) : 2066.5790
-                    positionZ (Float) : 33.8532
+                    positionX (Float) : 658.4755
+                    positionY (Float) : 2066.5770
+                    positionZ (Float) : 33.8578
                 216 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 15108
-                    positionX (Float) : 658.4762
-                    positionY (Float) : 2066.5790
-                    positionZ (Float) : 33.8532
+                    positionX (Float) : 658.4755
+                    positionY (Float) : 2066.5770
+                    positionZ (Float) : 33.8578
                 217 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3881,7 +3881,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 15604
-                    positionX (Float) : 647.3596
+                    positionX (Float) : 647.3597
                     positionY (Float) : 2112.2080
                     positionZ (Float) : 33.1573
                 224 (CSectorDataObject)
@@ -3889,7 +3889,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 15668
-                    positionX (Float) : 647.3596
+                    positionX (Float) : 647.3597
                     positionY (Float) : 2112.2080
                     positionZ (Float) : 33.1573
                 225 (CSectorDataObject)
@@ -4585,7 +4585,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 21764
-                    positionX (Float) : 684.9080
+                    positionX (Float) : 684.9079
                     positionY (Float) : 2072.9530
                     positionZ (Float) : 32.9479
                 312 (CSectorDataObject)
@@ -4593,7 +4593,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 21828
-                    positionX (Float) : 684.9080
+                    positionX (Float) : 684.9079
                     positionY (Float) : 2072.9530
                     positionZ (Float) : 32.9479
                 313 (CSectorDataObject)
@@ -4602,7 +4602,7 @@
                     radius (Uint16) : 50
                     offset (Uint64) : 21904
                     positionX (Float) : 687.2046
-                    positionY (Float) : 2091.3940
+                    positionY (Float) : 2091.3950
                     positionZ (Float) : 32.5967
                 314 (CSectorDataObject)
                     type (Uint8) : 2
@@ -4610,7 +4610,7 @@
                     radius (Uint16) : 50
                     offset (Uint64) : 21968
                     positionX (Float) : 687.2046
-                    positionY (Float) : 2091.3940
+                    positionY (Float) : 2091.3950
                     positionZ (Float) : 32.5967
                 315 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4633,7 +4633,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 22184
-                    positionX (Float) : 678.8069
+                    positionX (Float) : 678.8068
                     positionY (Float) : 2087.2480
                     positionZ (Float) : 32.6495
                 318 (CSectorDataObject)
@@ -4641,7 +4641,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 22248
-                    positionX (Float) : 678.8069
+                    positionX (Float) : 678.8068
                     positionY (Float) : 2087.2480
                     positionZ (Float) : 32.6495
                 319 (CSectorDataObject)
@@ -4649,17 +4649,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 22324
-                    positionX (Float) : 681.1335
-                    positionY (Float) : 2088.7340
-                    positionZ (Float) : 33.2688
+                    positionX (Float) : 681.1320
+                    positionY (Float) : 2088.7360
+                    positionZ (Float) : 33.2733
                 320 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 22388
-                    positionX (Float) : 681.1335
-                    positionY (Float) : 2088.7340
-                    positionZ (Float) : 33.2688
+                    positionX (Float) : 681.1320
+                    positionY (Float) : 2088.7360
+                    positionZ (Float) : 33.2733
                 321 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -4681,17 +4681,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 22604
-                    positionX (Float) : 681.0222
-                    positionY (Float) : 2091.8510
-                    positionZ (Float) : 33.2688
+                    positionX (Float) : 681.0206
+                    positionY (Float) : 2091.8520
+                    positionZ (Float) : 33.2733
                 324 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 22668
-                    positionX (Float) : 681.0222
-                    positionY (Float) : 2091.8510
-                    positionZ (Float) : 33.2688
+                    positionX (Float) : 681.0206
+                    positionY (Float) : 2091.8520
+                    positionZ (Float) : 33.2733
                 325 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -4713,7 +4713,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 22884
-                    positionX (Float) : 673.3314
+                    positionX (Float) : 673.3315
                     positionY (Float) : 2081.6890
                     positionZ (Float) : 32.6856
                 328 (CSectorDataObject)
@@ -4721,7 +4721,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 22948
-                    positionX (Float) : 673.3314
+                    positionX (Float) : 673.3315
                     positionY (Float) : 2081.6890
                     positionZ (Float) : 32.6856
                 329 (CSectorDataObject)
@@ -5081,7 +5081,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 26104
-                    positionX (Float) : 671.1019
+                    positionX (Float) : 671.1018
                     positionY (Float) : 2081.6090
                     positionZ (Float) : 32.6792
                 374 (CSectorDataObject)
@@ -5089,7 +5089,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 26168
-                    positionX (Float) : 671.1019
+                    positionX (Float) : 671.1018
                     positionY (Float) : 2081.6090
                     positionZ (Float) : 32.6792
                 375 (CSectorDataObject)
@@ -5097,7 +5097,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 26244
-                    positionX (Float) : 664.4597
+                    positionX (Float) : 664.4596
                     positionY (Float) : 2081.6620
                     positionZ (Float) : 32.6366
                 376 (CSectorDataObject)
@@ -5105,7 +5105,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 26308
-                    positionX (Float) : 664.4597
+                    positionX (Float) : 664.4596
                     positionY (Float) : 2081.6620
                     positionZ (Float) : 32.6366
                 377 (CSectorDataObject)
@@ -5209,7 +5209,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 27224
-                    positionX (Float) : 664.1306
+                    positionX (Float) : 664.1307
                     positionY (Float) : 2070.5770
                     positionZ (Float) : 32.6458
                 390 (CSectorDataObject)
@@ -5217,7 +5217,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 27288
-                    positionX (Float) : 664.1306
+                    positionX (Float) : 664.1307
                     positionY (Float) : 2070.5770
                     positionZ (Float) : 32.6458
                 391 (CSectorDataObject)
@@ -5369,17 +5369,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 28624
-                    positionX (Float) : 661.2107
-                    positionY (Float) : 2067.7110
-                    positionZ (Float) : 33.8532
+                    positionX (Float) : 661.2100
+                    positionY (Float) : 2067.7090
+                    positionZ (Float) : 33.8578
                 410 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 28688
-                    positionX (Float) : 661.2107
-                    positionY (Float) : 2067.7110
-                    positionZ (Float) : 33.8532
+                    positionX (Float) : 661.2100
+                    positionY (Float) : 2067.7090
+                    positionZ (Float) : 33.8578
                 411 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -5481,17 +5481,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 29604
-                    positionX (Float) : 662.1660
-                    positionY (Float) : 2070.3750
-                    positionZ (Float) : 33.8532
+                    positionX (Float) : 662.1670
+                    positionY (Float) : 2070.3730
+                    positionZ (Float) : 33.8578
                 424 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 29668
-                    positionX (Float) : 662.1660
-                    positionY (Float) : 2070.3750
-                    positionZ (Float) : 33.8532
+                    positionX (Float) : 662.1670
+                    positionY (Float) : 2070.3730
+                    positionZ (Float) : 33.8578
                 425 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -5657,17 +5657,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 31144
-                    positionX (Float) : 678.3903
-                    positionY (Float) : 2088.5950
-                    positionZ (Float) : 33.2688
+                    positionX (Float) : 678.3917
+                    positionY (Float) : 2088.5970
+                    positionZ (Float) : 33.2733
                 446 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 31208
-                    positionX (Float) : 678.3903
-                    positionY (Float) : 2088.5950
-                    positionZ (Float) : 33.2688
+                    positionX (Float) : 678.3917
+                    positionY (Float) : 2088.5970
+                    positionZ (Float) : 33.2733
                 447 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -5753,7 +5753,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 31984
-                    positionX (Float) : 686.2433
+                    positionX (Float) : 686.2432
                     positionY (Float) : 2086.2400
                     positionZ (Float) : 32.6174
                 458 (CSectorDataObject)
@@ -5761,7 +5761,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 32048
-                    positionX (Float) : 686.2433
+                    positionX (Float) : 686.2432
                     positionY (Float) : 2086.2400
                     positionZ (Float) : 32.6174
                 459 (CSectorDataObject)
@@ -5882,7 +5882,7 @@
                     radius (Uint16) : 50
                     offset (Uint64) : 33104
                     positionX (Float) : 689.1367
-                    positionY (Float) : 2108.2330
+                    positionY (Float) : 2108.2320
                     positionZ (Float) : 31.9710
                 474 (CSectorDataObject)
                     type (Uint8) : 2
@@ -5890,7 +5890,7 @@
                     radius (Uint16) : 50
                     offset (Uint64) : 33168
                     positionX (Float) : 689.1367
-                    positionY (Float) : 2108.2330
+                    positionY (Float) : 2108.2320
                     positionZ (Float) : 31.9710
                 475 (CSectorDataObject)
                     type (Uint8) : 1
@@ -5962,7 +5962,7 @@
                     radius (Uint16) : 75
                     offset (Uint64) : 33804
                     positionX (Float) : 679.1821
-                    positionY (Float) : 2068.0980
+                    positionY (Float) : 2068.0970
                     positionZ (Float) : 34.6169
                 484 (CSectorDataObject)
                     type (Uint8) : 2
@@ -5970,7 +5970,7 @@
                     radius (Uint16) : 75
                     offset (Uint64) : 33868
                     positionX (Float) : 679.1821
-                    positionY (Float) : 2068.0980
+                    positionY (Float) : 2068.0970
                     positionZ (Float) : 34.6169
                 485 (CSectorDataObject)
                     type (Uint8) : 1
@@ -6201,7 +6201,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 35904
-                    positionX (Float) : 646.7627
+                    positionX (Float) : 646.7628
                     positionY (Float) : 2089.3970
                     positionZ (Float) : 33.5943
                 514 (CSectorDataObject)
@@ -6209,7 +6209,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 35968
-                    positionX (Float) : 646.7627
+                    positionX (Float) : 646.7628
                     positionY (Float) : 2089.3970
                     positionZ (Float) : 33.5943
                 515 (CSectorDataObject)
@@ -6217,7 +6217,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 36044
-                    positionX (Float) : 649.0900
+                    positionX (Float) : 649.0901
                     positionY (Float) : 2087.9070
                     positionZ (Float) : 38.5984
                 516 (CSectorDataObject)
@@ -6225,7 +6225,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 36108
-                    positionX (Float) : 649.0900
+                    positionX (Float) : 649.0901
                     positionY (Float) : 2087.9070
                     positionZ (Float) : 38.5984
                 517 (CSectorDataObject)
@@ -6233,7 +6233,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 36184
-                    positionX (Float) : 649.0187
+                    positionX (Float) : 649.0188
                     positionY (Float) : 2090.9610
                     positionZ (Float) : 38.5980
                 518 (CSectorDataObject)
@@ -6241,7 +6241,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 36248
-                    positionX (Float) : 649.0187
+                    positionX (Float) : 649.0188
                     positionY (Float) : 2090.9610
                     positionZ (Float) : 38.5980
                 519 (CSectorDataObject)
@@ -6665,7 +6665,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 39964
-                    positionX (Float) : 650.8527
+                    positionX (Float) : 650.8528
                     positionY (Float) : 2101.4320
                     positionZ (Float) : 32.7520
                 572 (CSectorDataObject)
@@ -6673,7 +6673,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 40028
-                    positionX (Float) : 650.8527
+                    positionX (Float) : 650.8528
                     positionY (Float) : 2101.4320
                     positionZ (Float) : 32.7520
                 573 (CSectorDataObject)
@@ -6681,7 +6681,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 40104
-                    positionX (Float) : 653.9129
+                    positionX (Float) : 653.9130
                     positionY (Float) : 2101.4320
                     positionZ (Float) : 32.7520
                 574 (CSectorDataObject)
@@ -6689,7 +6689,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 40168
-                    positionX (Float) : 653.9129
+                    positionX (Float) : 653.9130
                     positionY (Float) : 2101.4320
                     positionZ (Float) : 32.7520
                 575 (CSectorDataObject)
@@ -6825,9 +6825,9 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 30
                     offset (Uint64) : 41364
-                    positionX (Float) : 669.2175
-                    positionY (Float) : 2060.5750
-                    positionZ (Float) : 32.6949
+                    positionX (Float) : 669.0588
+                    positionY (Float) : 2060.5250
+                    positionZ (Float) : 32.5807
                 592 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6857,7 +6857,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 41620
-                    positionX (Float) : 681.4136
+                    positionX (Float) : 681.4135
                     positionY (Float) : 2061.6980
                     positionZ (Float) : 34.5763
                 596 (CSectorDataObject)
@@ -6865,7 +6865,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 41684
-                    positionX (Float) : 681.4136
+                    positionX (Float) : 681.4135
                     positionY (Float) : 2061.6980
                     positionZ (Float) : 34.5763
                 597 (CSectorDataObject)
@@ -6875,7 +6875,7 @@
                     offset (Uint64) : 41760
                     positionX (Float) : 661.7719
                     positionY (Float) : 2060.7060
-                    positionZ (Float) : 31.7609
+                    positionZ (Float) : 31.7608
                 598 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
@@ -6883,7 +6883,7 @@
                     offset (Uint64) : 41824
                     positionX (Float) : 661.7719
                     positionY (Float) : 2060.7060
-                    positionZ (Float) : 31.7609
+                    positionZ (Float) : 31.7608
                 599 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6913,7 +6913,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 42136
-                    positionX (Float) : 683.9867
+                    positionX (Float) : 683.9868
                     positionY (Float) : 2056.7730
                     positionZ (Float) : 36.5918
                 603 (CSectorDataObject)
@@ -6967,19 +6967,19 @@
                 609 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 40
+                    radius (Uint16) : 45
                     offset (Uint64) : 42652
-                    positionX (Float) : 676.4832
-                    positionY (Float) : 2059.9600
-                    positionZ (Float) : 35.6296
+                    positionX (Float) : 676.4813
+                    positionY (Float) : 2059.9410
+                    positionZ (Float) : 35.6452
                 610 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 40
+                    radius (Uint16) : 45
                     offset (Uint64) : 42716
-                    positionX (Float) : 676.4832
-                    positionY (Float) : 2059.9600
-                    positionZ (Float) : 35.6296
+                    positionX (Float) : 676.4813
+                    positionY (Float) : 2059.9410
+                    positionZ (Float) : 35.6452
                 611 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -7017,7 +7017,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 43072
-                    positionX (Float) : 680.2614
+                    positionX (Float) : 680.2613
                     positionY (Float) : 2073.3270
                     positionZ (Float) : 32.6209
                 616 (CSectorDataObject)
@@ -7025,7 +7025,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 43136
-                    positionX (Float) : 680.2614
+                    positionX (Float) : 680.2613
                     positionY (Float) : 2073.3270
                     positionZ (Float) : 32.6209
                 617 (CSectorDataObject)
@@ -7273,7 +7273,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 45312
-                    positionX (Float) : 650.8852
+                    positionX (Float) : 650.8853
                     positionY (Float) : 2100.4020
                     positionZ (Float) : 32.8323
                 648 (CSectorDataObject)
@@ -7281,7 +7281,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 45376
-                    positionX (Float) : 650.8852
+                    positionX (Float) : 650.8853
                     positionY (Float) : 2100.4020
                     positionZ (Float) : 32.8323
                 649 (CSectorDataObject)
@@ -7305,7 +7305,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 45592
-                    positionX (Float) : 650.8484
+                    positionX (Float) : 650.8485
                     positionY (Float) : 2102.2510
                     positionZ (Float) : 32.8323
                 652 (CSectorDataObject)
@@ -7313,7 +7313,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 45656
-                    positionX (Float) : 650.8484
+                    positionX (Float) : 650.8485
                     positionY (Float) : 2102.2510
                     positionZ (Float) : 32.8323
                 653 (CSectorDataObject)
@@ -7321,7 +7321,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 45732
-                    positionX (Float) : 653.8892
+                    positionX (Float) : 653.8893
                     positionY (Float) : 2102.2510
                     positionZ (Float) : 32.8323
                 654 (CSectorDataObject)
@@ -7329,7 +7329,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 75
                     offset (Uint64) : 45796
-                    positionX (Float) : 653.8892
+                    positionX (Float) : 653.8893
                     positionY (Float) : 2102.2510
                     positionZ (Float) : 32.8323
                 655 (CSectorDataObject)
@@ -7433,17 +7433,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 46712
-                    positionX (Float) : 690.0652
-                    positionY (Float) : 2067.4990
-                    positionZ (Float) : 35.9959
+                    positionX (Float) : 690.0631
+                    positionY (Float) : 2067.4980
+                    positionZ (Float) : 36.0004
                 668 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 85
                     offset (Uint64) : 46776
-                    positionX (Float) : 690.0652
-                    positionY (Float) : 2067.4990
-                    positionZ (Float) : 35.9959
+                    positionX (Float) : 690.0631
+                    positionY (Float) : 2067.4980
+                    positionZ (Float) : 36.0004
                 669 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -7593,17 +7593,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 30
                     offset (Uint64) : 48100
-                    positionX (Float) : 692.4006
-                    positionY (Float) : 2113.7810
-                    positionZ (Float) : 31.4538
+                    positionX (Float) : 692.3873
+                    positionY (Float) : 2113.7650
+                    positionZ (Float) : 31.4147
                 688 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 30
                     offset (Uint64) : 48164
-                    positionX (Float) : 692.4006
-                    positionY (Float) : 2113.7810
-                    positionZ (Float) : 31.4538
+                    positionX (Float) : 692.3873
+                    positionY (Float) : 2113.7650
+                    positionZ (Float) : 31.4147
                 689 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -7850,7 +7850,7 @@
                     radius (Uint16) : 75
                     offset (Uint64) : 50316
                     positionX (Float) : 645.2209
-                    positionY (Float) : 2105.4660
+                    positionY (Float) : 2105.4650
                     positionZ (Float) : 33.0178
                 720 (CSectorDataObject)
                     type (Uint8) : 1
@@ -7937,41 +7937,41 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 30
                     offset (Uint64) : 51044
-                    positionX (Float) : 688.7287
-                    positionY (Float) : 2063.8170
+                    positionX (Float) : 688.4604
+                    positionY (Float) : 2063.7020
                     positionZ (Float) : 35.2080
                 731 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 51108
-                    positionX (Float) : 686.4286
-                    positionY (Float) : 2099.9960
-                    positionZ (Float) : 32.2489
+                    positionX (Float) : 686.4164
+                    positionY (Float) : 2099.9840
+                    positionZ (Float) : 32.2507
                 732 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 51172
-                    positionX (Float) : 686.4286
-                    positionY (Float) : 2099.9960
-                    positionZ (Float) : 32.2489
+                    positionX (Float) : 686.4164
+                    positionY (Float) : 2099.9840
+                    positionZ (Float) : 32.2507
                 733 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 51248
-                    positionX (Float) : 687.6935
-                    positionY (Float) : 2098.4280
-                    positionZ (Float) : 32.2878
+                    positionX (Float) : 687.6780
+                    positionY (Float) : 2098.4220
+                    positionZ (Float) : 32.2896
                 734 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 51312
-                    positionX (Float) : 687.6935
-                    positionY (Float) : 2098.4280
-                    positionZ (Float) : 32.2878
+                    positionX (Float) : 687.6780
+                    positionY (Float) : 2098.4220
+                    positionZ (Float) : 32.2896
                 735 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -7983,23 +7983,23 @@
                 736 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 60
                     offset (Uint64) : 51452
                     positionX (Float) : 654.8414
-                    positionY (Float) : 2091.8140
+                    positionY (Float) : 2091.8130
                     positionZ (Float) : 33.4313
                 737 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 60
                     offset (Uint64) : 51516
                     positionX (Float) : 654.8414
-                    positionY (Float) : 2091.8140
+                    positionY (Float) : 2091.8130
                     positionZ (Float) : 33.4313
                 738 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 60
                     offset (Uint64) : 51592
                     positionX (Float) : 654.9779
                     positionY (Float) : 2092.2890
@@ -8007,7 +8007,7 @@
                 739 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 60
                     offset (Uint64) : 51656
                     positionX (Float) : 654.9779
                     positionY (Float) : 2092.2890
@@ -8015,23 +8015,23 @@
                 740 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 60
                     offset (Uint64) : 51732
                     positionX (Float) : 651.5981
-                    positionY (Float) : 2091.9620
+                    positionY (Float) : 2091.9610
                     positionZ (Float) : 33.4593
                 741 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 60
                     offset (Uint64) : 51796
                     positionX (Float) : 651.5981
-                    positionY (Float) : 2091.9620
+                    positionY (Float) : 2091.9610
                     positionZ (Float) : 33.4593
                 742 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 60
                     offset (Uint64) : 51872
                     positionX (Float) : 654.6178
                     positionY (Float) : 2092.6740
@@ -8039,7 +8039,7 @@
                 743 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 60
                     offset (Uint64) : 51936
                     positionX (Float) : 654.6178
                     positionY (Float) : 2092.6740
@@ -8177,17 +8177,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 53132
-                    positionX (Float) : 671.1016
-                    positionY (Float) : 2061.6310
-                    positionZ (Float) : 32.6547
+                    positionX (Float) : 671.0842
+                    positionY (Float) : 2061.7590
+                    positionZ (Float) : 32.5768
                 761 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 53196
-                    positionX (Float) : 671.1016
-                    positionY (Float) : 2061.6310
-                    positionZ (Float) : 32.6547
+                    positionX (Float) : 671.0842
+                    positionY (Float) : 2061.7590
+                    positionZ (Float) : 32.5768
                 762 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -8415,11 +8415,11 @@
                 790 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 150
+                    radius (Uint16) : 300
                     offset (Uint64) : 55232
                     positionX (Float) : 660.8501
-                    positionY (Float) : 2052.4510
-                    positionZ (Float) : 34.8839
+                    positionY (Float) : 2052.4460
+                    positionZ (Float) : 34.8546
                 791 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -8439,7 +8439,7 @@
                 793 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 80
                     offset (Uint64) : 55436
                     positionX (Float) : 654.5933
                     positionY (Float) : 2018.3730
@@ -15683,7 +15683,7 @@
                 353 (SBlockData)
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.9845
-                        ay (Float) : -0.1753
+                        ay (Float) : -0.1752
                         az (Float) : -0.0032
                         by (Float) : 0.1752
                         bz (Float) : 0.9845
@@ -15701,7 +15701,7 @@
                 354 (SBlockData)
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.9845
-                        ay (Float) : -0.1753
+                        ay (Float) : -0.1752
                         az (Float) : -0.0032
                         by (Float) : 0.1752
                         bz (Float) : 0.9845
@@ -17367,7 +17367,7 @@
                         ay (Float) : 0.0000
                         az (Float) : -1.6397
                         by (Float) : 0.0000
-                        bz (Float) : 1.6398
+                        bz (Float) : 1.6397
                         by (Float) : 0.0000
                         cz (Float) : 1.3891
                         cy (Float) : 0.0000
@@ -17385,7 +17385,7 @@
                         ay (Float) : 0.0000
                         az (Float) : -1.6397
                         by (Float) : 0.0000
-                        bz (Float) : 1.6398
+                        bz (Float) : 1.6397
                         by (Float) : 0.0000
                         cz (Float) : 1.3891
                         cy (Float) : 0.0000
@@ -17408,7 +17408,7 @@
                         ay (Float) : 0.0000
                         az (Float) : -1.6397
                         by (Float) : 0.0000
-                        bz (Float) : 1.6398
+                        bz (Float) : 1.6397
                         by (Float) : 0.0000
                         cz (Float) : 1.3891
                         cy (Float) : 0.0000
@@ -17426,7 +17426,7 @@
                         ay (Float) : 0.0000
                         az (Float) : -1.6397
                         by (Float) : 0.0000
-                        bz (Float) : 1.6398
+                        bz (Float) : 1.6397
                         by (Float) : 0.0000
                         cz (Float) : 1.3891
                         cy (Float) : 0.0000
@@ -17449,7 +17449,7 @@
                         ay (Float) : 0.0000
                         az (Float) : -1.6397
                         by (Float) : 0.0000
-                        bz (Float) : 1.6398
+                        bz (Float) : 1.6397
                         by (Float) : 0.0000
                         cz (Float) : 1.3891
                         cy (Float) : 0.0000
@@ -17467,7 +17467,7 @@
                         ay (Float) : 0.0000
                         az (Float) : -1.6397
                         by (Float) : 0.0000
-                        bz (Float) : 1.6398
+                        bz (Float) : 1.6397
                         by (Float) : 0.0000
                         cz (Float) : 1.3891
                         cy (Float) : 0.0000
@@ -17490,7 +17490,7 @@
                         ay (Float) : 0.0000
                         az (Float) : -0.0105
                         by (Float) : 0.0000
-                        bz (Float) : 1.6398
+                        bz (Float) : 1.6397
                         by (Float) : 0.0000
                         cz (Float) : 0.0089
                         cy (Float) : 0.0000
@@ -17508,7 +17508,7 @@
                         ay (Float) : 0.0000
                         az (Float) : -0.0105
                         by (Float) : 0.0000
-                        bz (Float) : 1.6398
+                        bz (Float) : 1.6397
                         by (Float) : 0.0000
                         cz (Float) : 0.0089
                         cy (Float) : 0.0000
@@ -20571,9 +20571,9 @@
                         cy (Float) : 0.0000
                         cz (Float) : 1.0000
                     position (SVector3D)
-                        X (Float) : 669.1790
-                        Y (Float) : 2061.4590
-                        Z (Float) : 32.5890
+                        X (Float) : 669.0204
+                        Y (Float) : 2061.4100
+                        Z (Float) : 32.4747
                     streamingRadius (Uint16) : 0
                     flags (Uint16) : 4
                     occlusionSystemID (Uint32) : 4037824550
@@ -21359,7 +21359,7 @@
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.1967
                         ay (Float) : -1.0536
-                        az (Float) : 0.0012
+                        az (Float) : 0.0013
                         by (Float) : -1.0536
                         bz (Float) : -0.1967
                         by (Float) : -0.0067
@@ -21377,7 +21377,7 @@
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.1967
                         ay (Float) : -1.0536
-                        az (Float) : 0.0012
+                        az (Float) : 0.0013
                         by (Float) : -1.0536
                         bz (Float) : -0.1967
                         by (Float) : -0.0067
@@ -21400,7 +21400,7 @@
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.1967
                         ay (Float) : -1.0536
-                        az (Float) : 0.0012
+                        az (Float) : 0.0013
                         by (Float) : -1.0536
                         bz (Float) : -0.1967
                         by (Float) : -0.0067
@@ -21418,7 +21418,7 @@
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : 0.1967
                         ay (Float) : -1.0536
-                        az (Float) : 0.0012
+                        az (Float) : 0.0013
                         by (Float) : -1.0536
                         bz (Float) : -0.1967
                         by (Float) : -0.0067
@@ -23405,8 +23405,8 @@
                         cy (Float) : 0.0911
                         cz (Float) : 1.0344
                     position (SVector3D)
-                        X (Float) : 689.3773
-                        Y (Float) : 2063.1510
+                        X (Float) : 689.1090
+                        Y (Float) : 2063.0350
                         Z (Float) : 35.2171
                     streamingRadius (Uint16) : 0
                     flags (Uint16) : 4
@@ -24005,37 +24005,37 @@
                         collisionGroup (Uint64) : 550725311
                 760 (SBlockData)
                     rotationMatrix (CMatrix3x3)
-                        ax (Float) : 0.9958
-                        ay (Float) : -0.0895
-                        az (Float) : 0.0198
-                        by (Float) : 0.0886
-                        bz (Float) : 0.9952
-                        by (Float) : 0.0407
-                        cz (Float) : -0.0233
-                        cy (Float) : -0.0388
-                        cz (Float) : 0.9990
+                        ax (Float) : 0.9910
+                        ay (Float) : -0.0922
+                        az (Float) : 0.0972
+                        by (Float) : 0.0985
+                        bz (Float) : 0.9932
+                        by (Float) : -0.0620
+                        cz (Float) : -0.0908
+                        cy (Float) : 0.0710
+                        cz (Float) : 0.9933
                     position (SVector3D)
-                        X (Float) : 671.1016
-                        Y (Float) : 2061.6190
-                        Z (Float) : 32.6464
+                        X (Float) : 671.0847
+                        Y (Float) : 2061.7470
+                        Z (Float) : 32.5698
                     streamingRadius (Uint16) : 0
                     flags (Uint16) : 4
                     occlusionSystemID (Uint32) : 837594311
                 761 (SBlockData)
                     rotationMatrix (CMatrix3x3)
-                        ax (Float) : 0.9958
-                        ay (Float) : -0.0895
-                        az (Float) : 0.0198
-                        by (Float) : 0.0886
-                        bz (Float) : 0.9952
-                        by (Float) : 0.0407
-                        cz (Float) : -0.0233
-                        cy (Float) : -0.0388
-                        cz (Float) : 0.9990
+                        ax (Float) : 0.9910
+                        ay (Float) : -0.0922
+                        az (Float) : 0.0972
+                        by (Float) : 0.0985
+                        bz (Float) : 0.9932
+                        by (Float) : -0.0620
+                        cz (Float) : -0.0908
+                        cy (Float) : 0.0710
+                        cz (Float) : 0.9933
                     position (SVector3D)
-                        X (Float) : 671.1016
-                        Y (Float) : 2061.6190
-                        Z (Float) : 32.6464
+                        X (Float) : 671.0847
+                        Y (Float) : 2061.7470
+                        Z (Float) : 32.5698
                     streamingRadius (Uint16) : 0
                     flags (Uint16) : 0
                     occlusionSystemID (Uint32) : 0
