 1 file changed, 39 insertions(+), 39 deletions(-)

diff --git 1.32/levels/skellige/spikeroog/udalryk_house/house inside_decal.w2l.txt 4.00/levels/skellige/spikeroog/udalryk_house/house inside_decal.w2l.txt
index e370673..def17b2 100644
--- 1.32/levels/skellige/spikeroog/udalryk_house/house inside_decal.w2l.txt	
+++ 4.00/levels/skellige/spikeroog/udalryk_house/house inside_decal.w2l.txt	
@@ -82,7 +82,7 @@
                 0 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 47
+                    radius (Uint16) : 61
                     offset (Uint64) : 0
                     positionX (Float) : -1738.8190
                     positionY (Float) : 1505.8850
@@ -90,7 +90,7 @@
                 1 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 39
+                    radius (Uint16) : 50
                     offset (Uint64) : 76
                     positionX (Float) : -1739.2070
                     positionY (Float) : 1505.5060
@@ -98,7 +98,7 @@
                 2 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 44
+                    radius (Uint16) : 57
                     offset (Uint64) : 152
                     positionX (Float) : -1739.1500
                     positionY (Float) : 1505.5660
@@ -106,7 +106,7 @@
                 3 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 52
+                    radius (Uint16) : 67
                     offset (Uint64) : 228
                     positionX (Float) : -1739.5560
                     positionY (Float) : 1504.6820
@@ -114,7 +114,7 @@
                 4 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 52
+                    radius (Uint16) : 67
                     offset (Uint64) : 304
                     positionX (Float) : -1739.0540
                     positionY (Float) : 1503.5100
@@ -122,7 +122,7 @@
                 5 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 28
+                    radius (Uint16) : 35
                     offset (Uint64) : 380
                     positionX (Float) : -1738.6170
                     positionY (Float) : 1503.2170
@@ -130,7 +130,7 @@
                 6 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 30
+                    radius (Uint16) : 38
                     offset (Uint64) : 456
                     positionX (Float) : -1738.6230
                     positionY (Float) : 1504.7200
@@ -138,7 +138,7 @@
                 7 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 30
+                    radius (Uint16) : 38
                     offset (Uint64) : 532
                     positionX (Float) : -1738.7930
                     positionY (Float) : 1504.7840
@@ -146,7 +146,7 @@
                 8 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 69
+                    radius (Uint16) : 91
                     offset (Uint64) : 608
                     positionX (Float) : -1739.3640
                     positionY (Float) : 1504.2700
@@ -154,7 +154,7 @@
                 9 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 31
+                    radius (Uint16) : 39
                     offset (Uint64) : 684
                     positionX (Float) : -1739.4200
                     positionY (Float) : 1506.9810
@@ -162,7 +162,7 @@
                 10 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 31
+                    radius (Uint16) : 39
                     offset (Uint64) : 760
                     positionX (Float) : -1739.4190
                     positionY (Float) : 1506.9330
@@ -170,7 +170,7 @@
                 11 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 51
+                    radius (Uint16) : 66
                     offset (Uint64) : 836
                     positionX (Float) : -1739.2240
                     positionY (Float) : 1506.1020
@@ -178,7 +178,7 @@
                 12 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 31
+                    radius (Uint16) : 39
                     offset (Uint64) : 912
                     positionX (Float) : -1739.8720
                     positionY (Float) : 1507.7330
@@ -186,7 +186,7 @@
                 13 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 43
+                    radius (Uint16) : 56
                     offset (Uint64) : 988
                     positionX (Float) : -1739.6560
                     positionY (Float) : 1507.1810
@@ -194,7 +194,7 @@
                 14 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 43
+                    radius (Uint16) : 56
                     offset (Uint64) : 1064
                     positionX (Float) : -1738.7440
                     positionY (Float) : 1504.9850
@@ -202,7 +202,7 @@
                 15 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 23
+                    radius (Uint16) : 29
                     offset (Uint64) : 1140
                     positionX (Float) : -1739.0290
                     positionY (Float) : 1505.7190
@@ -210,7 +210,7 @@
                 16 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 35
+                    radius (Uint16) : 45
                     offset (Uint64) : 1216
                     positionX (Float) : -1738.3150
                     positionY (Float) : 1504.8740
@@ -218,7 +218,7 @@
                 17 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 32
+                    radius (Uint16) : 41
                     offset (Uint64) : 1292
                     positionX (Float) : -1738.5850
                     positionY (Float) : 1504.1590
@@ -226,7 +226,7 @@
                 18 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 22
+                    radius (Uint16) : 28
                     offset (Uint64) : 1368
                     positionX (Float) : -1737.9650
                     positionY (Float) : 1503.9100
@@ -234,7 +234,7 @@
                 19 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 24
+                    radius (Uint16) : 30
                     offset (Uint64) : 1444
                     positionX (Float) : -1738.1260
                     positionY (Float) : 1503.8490
@@ -242,7 +242,7 @@
                 20 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 57
+                    radius (Uint16) : 74
                     offset (Uint64) : 1520
                     positionX (Float) : -1738.3540
                     positionY (Float) : 1503.8280
@@ -250,7 +250,7 @@
                 21 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 51
+                    radius (Uint16) : 67
                     offset (Uint64) : 1596
                     positionX (Float) : -1738.9550
                     positionY (Float) : 1503.5530
@@ -258,7 +258,7 @@
                 22 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 51
+                    radius (Uint16) : 67
                     offset (Uint64) : 1672
                     positionX (Float) : -1739.3010
                     positionY (Float) : 1504.8930
@@ -266,7 +266,7 @@
                 23 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 36
+                    radius (Uint16) : 46
                     offset (Uint64) : 1748
                     positionX (Float) : -1739.6050
                     positionY (Float) : 1506.5480
@@ -274,7 +274,7 @@
                 24 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 128
+                    radius (Uint16) : 169
                     offset (Uint64) : 1824
                     positionX (Float) : -1739.2510
                     positionY (Float) : 1506.3150
@@ -282,7 +282,7 @@
                 25 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 18
+                    radius (Uint16) : 23
                     offset (Uint64) : 1900
                     positionX (Float) : -1739.6630
                     positionY (Float) : 1507.2390
@@ -290,7 +290,7 @@
                 26 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 23
+                    radius (Uint16) : 29
                     offset (Uint64) : 1976
                     positionX (Float) : -1739.7000
                     positionY (Float) : 1507.2800
@@ -298,7 +298,7 @@
                 27 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 36
+                    radius (Uint16) : 46
                     offset (Uint64) : 2052
                     positionX (Float) : -1739.5970
                     positionY (Float) : 1506.5750
@@ -306,7 +306,7 @@
                 28 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 26
+                    radius (Uint16) : 33
                     offset (Uint64) : 2128
                     positionX (Float) : -1739.1570
                     positionY (Float) : 1505.6690
@@ -314,7 +314,7 @@
                 29 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 28
+                    radius (Uint16) : 36
                     offset (Uint64) : 2204
                     positionX (Float) : -1739.3590
                     positionY (Float) : 1505.8400
@@ -322,7 +322,7 @@
                 30 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 26
+                    radius (Uint16) : 33
                     offset (Uint64) : 2280
                     positionX (Float) : -1739.1750
                     positionY (Float) : 1505.4750
@@ -330,7 +330,7 @@
                 31 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 26
+                    radius (Uint16) : 33
                     offset (Uint64) : 2356
                     positionX (Float) : -1739.2210
                     positionY (Float) : 1505.6370
@@ -338,7 +338,7 @@
                 32 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 142
+                    radius (Uint16) : 187
                     offset (Uint64) : 2432
                     positionX (Float) : -1737.3890
                     positionY (Float) : 1508.3690
@@ -346,7 +346,7 @@
                 33 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 120
+                    radius (Uint16) : 159
                     offset (Uint64) : 2508
                     positionX (Float) : -1737.5230
                     positionY (Float) : 1506.9900
@@ -354,7 +354,7 @@
                 34 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 98
+                    radius (Uint16) : 129
                     offset (Uint64) : 2584
                     positionX (Float) : -1736.3970
                     positionY (Float) : 1507.4650
@@ -362,7 +362,7 @@
                 35 (CSectorDataObject)
                     type (Uint8) : 3
                     flags (Uint8) : 0
-                    radius (Uint16) : 119
+                    radius (Uint16) : 157
                     offset (Uint64) : 2660
                     positionX (Float) : -1733.5880
                     positionY (Float) : 1507.0450
@@ -643,7 +643,7 @@
                         by (Float) : 0.4337
                         cz (Float) : 0.2608
                         cy (Float) : 0.1241
-                        cz (Float) : -0.0107
+                        cz (Float) : -0.0106
                     position (SVector3D)
                         X (Float) : -1739.2240
                         Y (Float) : 1506.1020
@@ -1070,7 +1070,7 @@
                         ax (Float) : -0.1384
                         ay (Float) : 0.1119
                         az (Float) : -0.5304
-                        by (Float) : 0.6502
+                        by (Float) : 0.6501
                         bz (Float) : 0.3004
                         by (Float) : -0.1063
                         cz (Float) : 0.1133
@@ -1094,7 +1094,7 @@
                         ax (Float) : -0.0264
                         ay (Float) : 0.1979
                         az (Float) : 0.3978
-                        by (Float) : 0.6502
+                        by (Float) : 0.6501
                         bz (Float) : 0.3004
                         by (Float) : -0.1063
                         cz (Float) : -0.1168
