 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git 1.32/dlc/bob/data/levels/bob/community/sansretour_marshes/fisherman's hut/kids_day.w2l.txt 4.00/dlc/bob/data/levels/bob/community/sansretour_marshes/fisherman's hut/kids_day.w2l.txt
index bd83dc4..2ee2bcb 100644
--- 1.32/dlc/bob/data/levels/bob/community/sansretour_marshes/fisherman's hut/kids_day.w2l.txt	
+++ 4.00/dlc/bob/data/levels/bob/community/sansretour_marshes/fisherman's hut/kids_day.w2l.txt	
@@ -57,7 +57,7 @@
                                 W (Float) : 1.0000
                             Max (Vector)
                                 X (Float) : -110.2539
-                                Y (Float) : -711.5026
+                                Y (Float) : -711.5025
                                 Z (Float) : 0.5570
                                 W (Float) : 1.0000
                         mesh (handle:CMesh) : environment/terrain_surroundings/common/rock_little_round_small_angular.w2mesh
@@ -304,7 +304,7 @@
                                 W (Float) : 1.0000
                             Max (Vector)
                                 X (Float) : -106.4092
-                                Y (Float) : -702.9103
+                                Y (Float) : -702.9102
                                 Z (Float) : 0.3070
                                 W (Float) : 1.0000
                         mesh (handle:CMesh) : environment/terrain_surroundings/common/rock_little_round_small_triangular.w2mesh
@@ -364,7 +364,7 @@
                                 W (Float) : 1.0000
                             Max (Vector)
                                 X (Float) : -106.4588
-                                Y (Float) : -702.9499
+                                Y (Float) : -702.9500
                                 Z (Float) : 0.3268
                                 W (Float) : 1.0000
                         mesh (handle:CMesh) : environment/terrain_surroundings/common/rock_little_round_small_smallest.w2mesh
@@ -389,7 +389,7 @@
                         boundingBox (Box)
                             Min (Vector)
                                 X (Float) : -106.5663
-                                Y (Float) : -703.0380
+                                Y (Float) : -703.0379
                                 Z (Float) : 0.2333
                                 W (Float) : 1.0000
                             Max (Vector)
@@ -455,7 +455,7 @@
                         boundingBox (Box)
                             Min (Vector)
                                 X (Float) : -105.9411
-                                Y (Float) : -705.6348
+                                Y (Float) : -705.6347
                                 Z (Float) : 0.2157
                                 W (Float) : 1.0000
                             Max (Vector)
@@ -972,7 +972,7 @@
                     radius (Uint16) : 50
                     offset (Uint64) : 576
                     positionX (Float) : -106.5093
-                    positionY (Float) : -702.9899
+                    positionY (Float) : -702.9898
                     positionZ (Float) : 0.2835
                 10 (CSectorDataObject)
                     type (Uint8) : 1
