 1 file changed, 201 insertions(+), 201 deletions(-)

diff --git 1.32/dlc/bob/data/levels/bob/locations/barrens/poi_bar_a_08/bandit campfire.w2l.txt 4.00/dlc/bob/data/levels/bob/locations/barrens/poi_bar_a_08/bandit campfire.w2l.txt
index 8b0e27f..be7d978 100644
--- 1.32/dlc/bob/data/levels/bob/locations/barrens/poi_bar_a_08/bandit campfire.w2l.txt	
+++ 4.00/dlc/bob/data/levels/bob/locations/barrens/poi_bar_a_08/bandit campfire.w2l.txt	
@@ -673,7 +673,7 @@
                     box2 (Float) : -2.7022
                     box3 (Float) : 3.3420
                     box4 (Float) : 3.3551
-                    box5 (Float) : 2.6877
+                    box5 (Float) : 2.6876
                     pathHash (String) : "dlc/bob/data/environment/terrain_surroundings/rock_debris/bob_round_rock_a_grass.w2mesh"
                 3 (CSectorDataResource)
                     box0 (Float) : -3.5826
@@ -681,7 +681,7 @@
                     box2 (Float) : -2.7532
                     box3 (Float) : 3.5833
                     box4 (Float) : 3.5831
-                    box5 (Float) : 2.8096
+                    box5 (Float) : 2.8095
                     pathHash (String) : "dlc/bob/data/environment/terrain_surroundings/rock_debris/bob_round_rock_b_grass.w2mesh"
                 4 (CSectorDataResource)
                     box0 (Float) : -3.1039
@@ -761,7 +761,7 @@
                     box2 (Float) : -4.9255
                     box3 (Float) : 1.2406
                     box4 (Float) : 5.2781
-                    box5 (Float) : 0.2089
+                    box5 (Float) : 0.2088
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/creeper/creep_stairs_l.w2mesh"
                 14 (CSectorDataResource)
                     box0 (Float) : -4.4313
@@ -789,11 +789,11 @@
                     pathHash (String) : "environment/blockout_base/box_wood.w2mesh"
                 17 (CSectorDataResource)
                     box0 (Float) : -1.0533
-                    box1 (Float) : -0.8855
-                    box2 (Float) : -0.6575
-                    box3 (Float) : 0.9726
+                    box1 (Float) : -0.8816
+                    box2 (Float) : -0.6546
+                    box3 (Float) : 0.9596
                     box4 (Float) : 1.0301
-                    box5 (Float) : 1.5115
+                    box5 (Float) : 1.5091
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/decoration_set/barrels/meshes/merge_deco_gen_crate_pile_i.w2mesh"
                 18 (CSectorDataResource)
                     box0 (Float) : -0.2132
@@ -993,7 +993,7 @@
                     box2 (Float) : 0.0008
                     box3 (Float) : 0.3390
                     box4 (Float) : 0.3335
-                    box5 (Float) : 0.0376
+                    box5 (Float) : 0.0373
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/containers/barrels/gen_barrel_lid.w2mesh"
                 43 (CSectorDataResource)
                     box0 (Float) : -0.0971
@@ -1028,28 +1028,28 @@
                     box5 (Float) : 0.4300
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/decoration_set/crates/meshes/merge_deco_gen_crate_jars_c.w2mesh"
                 47 (CSectorDataResource)
-                    box0 (Float) : -0.7278
-                    box1 (Float) : -1.3359
-                    box2 (Float) : -0.0053
-                    box3 (Float) : 0.7278
-                    box4 (Float) : 1.3359
-                    box5 (Float) : 0.2923
+                    box0 (Float) : -0.7165
+                    box1 (Float) : -1.3304
+                    box2 (Float) : -0.0067
+                    box3 (Float) : 0.7260
+                    box4 (Float) : 1.3299
+                    box5 (Float) : 0.2893
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/logs/gen_log_platform_hi_res.w2mesh"
                 48 (CSectorDataResource)
-                    box0 (Float) : -0.7319
-                    box1 (Float) : -1.3670
-                    box2 (Float) : 0.0019
-                    box3 (Float) : 0.7319
-                    box4 (Float) : 1.3670
-                    box5 (Float) : 0.8254
+                    box0 (Float) : -0.7339
+                    box1 (Float) : -1.3771
+                    box2 (Float) : 0.0020
+                    box3 (Float) : 0.7127
+                    box4 (Float) : 1.3658
+                    box5 (Float) : 0.8058
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/logs/gen_log_pile_long_hi_res.w2mesh"
                 49 (CSectorDataResource)
-                    box0 (Float) : -0.1489
-                    box1 (Float) : -1.2325
-                    box2 (Float) : 0.0020
-                    box3 (Float) : 0.1489
-                    box4 (Float) : 1.2325
-                    box5 (Float) : 0.2970
+                    box0 (Float) : -0.1689
+                    box1 (Float) : -1.2378
+                    box2 (Float) : -0.0018
+                    box3 (Float) : 0.1490
+                    box4 (Float) : 1.2386
+                    box5 (Float) : 0.2963
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/logs/gen_log_single_hi_res.w2mesh"
                 50 (CSectorDataResource)
                     box0 (Float) : -0.1252
@@ -1073,7 +1073,7 @@
                     box2 (Float) : -0.4886
                     box3 (Float) : 0.3853
                     box4 (Float) : 0.3497
-                    box5 (Float) : 0.4895
+                    box5 (Float) : 0.4894
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/accessories/gen_leather_saddle.w2mesh"
                 53 (CSectorDataResource)
                     box0 (Float) : -1.2969
@@ -1084,20 +1084,20 @@
                     box5 (Float) : 3.4003
                     pathHash (String) : "environment/decorations/near_water/harbour_props/large_wooden_cage_broken.w2mesh"
                 54 (CSectorDataResource)
-                    box0 (Float) : -0.6595
-                    box1 (Float) : -0.6855
-                    box2 (Float) : -0.0996
-                    box3 (Float) : 0.6842
-                    box4 (Float) : 0.6652
-                    box5 (Float) : 2.9787
+                    box0 (Float) : -0.6881
+                    box1 (Float) : -0.7444
+                    box2 (Float) : -0.0063
+                    box3 (Float) : 0.6454
+                    box4 (Float) : 0.6138
+                    box5 (Float) : 3.0268
                     pathHash (String) : "environment/decorations/near_water/harbour_props/metal_cage_c.w2mesh"
                 55 (CSectorDataResource)
-                    box0 (Float) : -0.2779
-                    box1 (Float) : -0.2766
-                    box2 (Float) : -0.0042
-                    box3 (Float) : 0.2773
-                    box4 (Float) : 0.2708
-                    box5 (Float) : 0.4781
+                    box0 (Float) : -0.3074
+                    box1 (Float) : -0.2678
+                    box2 (Float) : -0.0482
+                    box3 (Float) : 0.3085
+                    box4 (Float) : 0.2905
+                    box5 (Float) : 0.4797
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/logs/gen_tree_stump.w2mesh"
                 56 (CSectorDataResource)
                     box0 (Float) : -0.1472
@@ -1172,10 +1172,10 @@
                     box5 (Float) : 1.6608
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/merchant/vehicles/poor_wagon_a.w2mesh"
                 65 (CSectorDataResource)
-                    box0 (Float) : -1.0604
-                    box1 (Float) : -1.1821
+                    box0 (Float) : -1.0595
+                    box1 (Float) : -1.1829
                     box2 (Float) : -0.4727
-                    box3 (Float) : 1.1541
+                    box3 (Float) : 1.1511
                     box4 (Float) : 1.1968
                     box5 (Float) : 1.2912
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/merchant/decoration_set/vehicles/meshes/merge_deco_gen_crate_pile_f.w2mesh"
@@ -1273,7 +1273,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 200
                     offset (Uint64) : 560
-                    positionX (Float) : 914.7519
+                    positionX (Float) : 914.7518
                     positionY (Float) : -699.8779
                     positionZ (Float) : 31.6848
                 9 (CSectorDataObject)
@@ -1281,7 +1281,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 200
                     offset (Uint64) : 624
-                    positionX (Float) : 914.7519
+                    positionX (Float) : 914.7518
                     positionY (Float) : -699.8779
                     positionZ (Float) : 31.6848
                 10 (CSectorDataObject)
@@ -1338,7 +1338,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 1120
                     positionX (Float) : 906.0750
-                    positionY (Float) : -708.3375
+                    positionY (Float) : -708.3374
                     positionZ (Float) : 30.8041
                 17 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1346,7 +1346,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 1184
                     positionX (Float) : 906.0750
-                    positionY (Float) : -708.3375
+                    positionY (Float) : -708.3374
                     positionZ (Float) : 30.8041
                 18 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1370,7 +1370,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 1400
                     positionX (Float) : 924.0269
-                    positionY (Float) : -701.7328
+                    positionY (Float) : -701.7329
                     positionZ (Float) : 31.8932
                 21 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1378,7 +1378,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 1464
                     positionX (Float) : 924.0269
-                    positionY (Float) : -701.7328
+                    positionY (Float) : -701.7329
                     positionZ (Float) : 31.8932
                 22 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1418,16 +1418,16 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 1820
                     positionX (Float) : 909.7239
-                    positionY (Float) : -694.0333
-                    positionZ (Float) : 33.2589
+                    positionY (Float) : -694.0332
+                    positionZ (Float) : 33.2588
                 27 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 200
                     offset (Uint64) : 1884
                     positionX (Float) : 909.7239
-                    positionY (Float) : -694.0333
-                    positionZ (Float) : 33.2589
+                    positionY (Float) : -694.0332
+                    positionZ (Float) : 33.2588
                 28 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -2971,7 +2971,7 @@
                     offset (Uint64) : 15400
                     positionX (Float) : 917.1359
                     positionY (Float) : -713.2446
-                    positionZ (Float) : 32.2510
+                    positionZ (Float) : 32.2511
                 221 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
@@ -2979,7 +2979,7 @@
                     offset (Uint64) : 15464
                     positionX (Float) : 917.1359
                     positionY (Float) : -713.2446
-                    positionZ (Float) : 32.2510
+                    positionZ (Float) : 32.2511
                 222 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3626,7 +3626,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 21140
                     positionX (Float) : 905.8202
-                    positionY (Float) : -718.3700
+                    positionY (Float) : -718.3701
                     positionZ (Float) : 27.2255
                 303 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3634,7 +3634,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 21204
                     positionX (Float) : 905.8202
-                    positionY (Float) : -718.3700
+                    positionY (Float) : -718.3701
                     positionZ (Float) : 27.2255
                 304 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3642,7 +3642,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 21280
                     positionX (Float) : 913.7625
-                    positionY (Float) : -722.5364
+                    positionY (Float) : -722.5365
                     positionZ (Float) : 28.5357
                 305 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3650,7 +3650,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 21344
                     positionX (Float) : 913.7625
-                    positionY (Float) : -722.5364
+                    positionY (Float) : -722.5365
                     positionZ (Float) : 28.5357
                 306 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3658,8 +3658,8 @@
                     radius (Uint16) : 45
                     offset (Uint64) : 21420
                     positionX (Float) : 911.1180
-                    positionY (Float) : -722.5764
-                    positionZ (Float) : 29.3584
+                    positionY (Float) : -722.5763
+                    positionZ (Float) : 29.3583
                 307 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -3713,7 +3713,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 200
                     offset (Uint64) : 21880
-                    positionX (Float) : 910.8144
+                    positionX (Float) : 910.8143
                     positionY (Float) : -689.2683
                     positionZ (Float) : 34.6718
                 314 (CSectorDataObject)
@@ -3721,7 +3721,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 200
                     offset (Uint64) : 21944
-                    positionX (Float) : 910.8144
+                    positionX (Float) : 910.8143
                     positionY (Float) : -689.2683
                     positionZ (Float) : 34.6718
                 315 (CSectorDataObject)
@@ -3746,7 +3746,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 22160
                     positionX (Float) : 928.2042
-                    positionY (Float) : -696.5313
+                    positionY (Float) : -696.5314
                     positionZ (Float) : 32.6632
                 318 (CSectorDataObject)
                     type (Uint8) : 2
@@ -3754,23 +3754,23 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 22224
                     positionX (Float) : 928.2042
-                    positionY (Float) : -696.5313
+                    positionY (Float) : -696.5314
                     positionZ (Float) : 32.6632
                 319 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 200
                     offset (Uint64) : 22300
-                    positionX (Float) : 927.9085
-                    positionY (Float) : -692.7329
+                    positionX (Float) : 927.9086
+                    positionY (Float) : -692.7330
                     positionZ (Float) : 33.1649
                 320 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 200
                     offset (Uint64) : 22364
-                    positionX (Float) : 927.9085
-                    positionY (Float) : -692.7329
+                    positionX (Float) : 927.9086
+                    positionY (Float) : -692.7330
                     positionZ (Float) : 33.1649
                 321 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3793,17 +3793,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 22580
-                    positionX (Float) : 916.8617
-                    positionY (Float) : -711.6578
-                    positionZ (Float) : 29.7394
+                    positionX (Float) : 916.8574
+                    positionY (Float) : -711.6631
+                    positionZ (Float) : 29.7386
                 324 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 50
+                    radius (Uint16) : 60
                     offset (Uint64) : 22644
-                    positionX (Float) : 916.8617
-                    positionY (Float) : -711.6578
-                    positionZ (Float) : 29.7394
+                    positionX (Float) : 916.8574
+                    positionY (Float) : -711.6631
+                    positionZ (Float) : 29.7386
                 325 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -4322,7 +4322,7 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 27116
                     positionX (Float) : 917.7178
-                    positionY (Float) : -709.3748
+                    positionY (Float) : -709.3747
                     positionZ (Float) : 30.1807
                 390 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4407,11 +4407,11 @@
                 400 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 40
+                    radius (Uint16) : 50
                     offset (Uint64) : 27844
-                    positionX (Float) : 914.8882
+                    positionX (Float) : 914.8881
                     positionY (Float) : -717.3314
-                    positionZ (Float) : 28.3432
+                    positionZ (Float) : 28.3431
                 401 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -4433,7 +4433,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 28036
-                    positionX (Float) : 914.4072
+                    positionX (Float) : 914.4071
                     positionY (Float) : -717.5115
                     positionZ (Float) : 28.3513
                 404 (CSectorDataObject)
@@ -4441,7 +4441,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 28100
-                    positionX (Float) : 914.4072
+                    positionX (Float) : 914.4071
                     positionY (Float) : -717.5115
                     positionZ (Float) : 28.3513
                 405 (CSectorDataObject)
@@ -4487,99 +4487,99 @@
                 410 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 28508
-                    positionX (Float) : 911.0400
-                    positionY (Float) : -702.9362
-                    positionZ (Float) : 30.5179
+                    positionX (Float) : 911.0410
+                    positionY (Float) : -702.9318
+                    positionZ (Float) : 30.5206
                 411 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 28572
-                    positionX (Float) : 911.0400
-                    positionY (Float) : -702.9362
-                    positionZ (Float) : 30.5179
+                    positionX (Float) : 911.0410
+                    positionY (Float) : -702.9318
+                    positionZ (Float) : 30.5206
                 412 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 28648
-                    positionX (Float) : 911.3263
-                    positionY (Float) : -702.1412
-                    positionZ (Float) : 30.2437
+                    positionX (Float) : 911.3245
+                    positionY (Float) : -702.1294
+                    positionZ (Float) : 30.2339
                 413 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 28712
-                    positionX (Float) : 911.3263
-                    positionY (Float) : -702.1412
-                    positionZ (Float) : 30.2437
+                    positionX (Float) : 911.3245
+                    positionY (Float) : -702.1294
+                    positionZ (Float) : 30.2339
                 414 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 28788
-                    positionX (Float) : 910.9690
-                    positionY (Float) : -703.1689
-                    positionZ (Float) : 30.8859
+                    positionX (Float) : 910.9780
+                    positionY (Float) : -703.1567
+                    positionZ (Float) : 30.8884
                 415 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 28852
-                    positionX (Float) : 910.9690
-                    positionY (Float) : -703.1689
-                    positionZ (Float) : 30.8859
+                    positionX (Float) : 910.9780
+                    positionY (Float) : -703.1567
+                    positionZ (Float) : 30.8884
                 416 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 28928
-                    positionX (Float) : 911.0284
-                    positionY (Float) : -702.9435
-                    positionZ (Float) : 30.0571
+                    positionX (Float) : 911.0204
+                    positionY (Float) : -702.9528
+                    positionZ (Float) : 30.0666
                 417 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 28992
-                    positionX (Float) : 911.0284
-                    positionY (Float) : -702.9435
-                    positionZ (Float) : 30.0571
+                    positionX (Float) : 911.0204
+                    positionY (Float) : -702.9528
+                    positionZ (Float) : 30.0666
                 418 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29068
-                    positionX (Float) : 911.4642
-                    positionY (Float) : -701.2983
-                    positionZ (Float) : 29.7193
+                    positionX (Float) : 911.4674
+                    positionY (Float) : -701.2889
+                    positionZ (Float) : 29.7170
                 419 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29132
-                    positionX (Float) : 911.4642
-                    positionY (Float) : -701.2983
-                    positionZ (Float) : 29.7193
+                    positionX (Float) : 911.4674
+                    positionY (Float) : -701.2889
+                    positionZ (Float) : 29.7170
                 420 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29208
-                    positionX (Float) : 910.9612
-                    positionY (Float) : -703.5519
-                    positionZ (Float) : 29.9734
+                    positionX (Float) : 910.9631
+                    positionY (Float) : -703.5446
+                    positionZ (Float) : 29.9802
                 421 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29272
-                    positionX (Float) : 910.9612
-                    positionY (Float) : -703.5519
-                    positionZ (Float) : 29.9734
+                    positionX (Float) : 910.9631
+                    positionY (Float) : -703.5446
+                    positionZ (Float) : 29.9802
                 422 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -4599,57 +4599,57 @@
                 424 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29488
-                    positionX (Float) : 911.0550
-                    positionY (Float) : -703.1839
-                    positionZ (Float) : 29.8629
+                    positionX (Float) : 911.0575
+                    positionY (Float) : -703.1743
+                    positionZ (Float) : 29.8607
                 425 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29552
-                    positionX (Float) : 911.0550
-                    positionY (Float) : -703.1839
-                    positionZ (Float) : 29.8629
+                    positionX (Float) : 911.0575
+                    positionY (Float) : -703.1743
+                    positionZ (Float) : 29.8607
                 426 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29628
-                    positionX (Float) : 911.1827
-                    positionY (Float) : -702.1573
-                    positionZ (Float) : 29.7817
+                    positionX (Float) : 911.1813
+                    positionY (Float) : -702.1613
+                    positionZ (Float) : 29.7796
                 427 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29692
-                    positionX (Float) : 911.1827
-                    positionY (Float) : -702.1573
-                    positionZ (Float) : 29.7817
+                    positionX (Float) : 911.1813
+                    positionY (Float) : -702.1613
+                    positionZ (Float) : 29.7796
                 428 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29768
-                    positionX (Float) : 910.9619
-                    positionY (Float) : -703.5378
-                    positionZ (Float) : 29.7815
+                    positionX (Float) : 910.9602
+                    positionY (Float) : -703.5442
+                    positionZ (Float) : 29.7892
                 429 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 75
+                    radius (Uint16) : 100
                     offset (Uint64) : 29832
-                    positionX (Float) : 910.9619
-                    positionY (Float) : -703.5378
-                    positionZ (Float) : 29.7815
+                    positionX (Float) : 910.9602
+                    positionY (Float) : -703.5442
+                    positionZ (Float) : 29.7892
                 430 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 20
                     offset (Uint64) : 29908
-                    positionX (Float) : 911.1418
+                    positionX (Float) : 911.1419
                     positionY (Float) : -702.1157
                     positionZ (Float) : 28.6112
                 431 (CSectorDataObject)
@@ -4657,7 +4657,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 20
                     offset (Uint64) : 29972
-                    positionX (Float) : 911.1418
+                    positionX (Float) : 911.1419
                     positionY (Float) : -702.1157
                     positionZ (Float) : 28.6112
                 432 (CSectorDataObject)
@@ -4762,7 +4762,7 @@
                     radius (Uint16) : 20
                     offset (Uint64) : 30888
                     positionX (Float) : 910.7454
-                    positionY (Float) : -703.8504
+                    positionY (Float) : -703.8505
                     positionZ (Float) : 30.8315
                 445 (CSectorDataObject)
                     type (Uint8) : 2
@@ -4770,7 +4770,7 @@
                     radius (Uint16) : 20
                     offset (Uint64) : 30952
                     positionX (Float) : 910.7454
-                    positionY (Float) : -703.8504
+                    positionY (Float) : -703.8505
                     positionZ (Float) : 30.8315
                 446 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4855,35 +4855,35 @@
                 456 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 60
+                    radius (Uint16) : 80
                     offset (Uint64) : 31728
-                    positionX (Float) : 910.2389
-                    positionY (Float) : -713.0103
-                    positionZ (Float) : 29.4260
+                    positionX (Float) : 910.1962
+                    positionY (Float) : -713.0070
+                    positionZ (Float) : 29.5117
                 457 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 60
+                    radius (Uint16) : 80
                     offset (Uint64) : 31792
-                    positionX (Float) : 910.2389
-                    positionY (Float) : -713.0103
-                    positionZ (Float) : 29.4260
+                    positionX (Float) : 910.1962
+                    positionY (Float) : -713.0070
+                    positionZ (Float) : 29.5117
                 458 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 60
+                    radius (Uint16) : 80
                     offset (Uint64) : 31868
-                    positionX (Float) : 911.4515
-                    positionY (Float) : -713.7720
-                    positionZ (Float) : 29.2384
+                    positionX (Float) : 911.4711
+                    positionY (Float) : -713.7976
+                    positionZ (Float) : 29.3286
                 459 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 60
+                    radius (Uint16) : 80
                     offset (Uint64) : 31932
-                    positionX (Float) : 911.4515
-                    positionY (Float) : -713.7720
-                    positionZ (Float) : 29.2384
+                    positionX (Float) : 911.4711
+                    positionY (Float) : -713.7976
+                    positionZ (Float) : 29.3286
                 460 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -4905,33 +4905,33 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 32148
-                    positionX (Float) : 915.5426
-                    positionY (Float) : -709.9583
-                    positionZ (Float) : 29.0839
+                    positionX (Float) : 915.5304
+                    positionY (Float) : -709.9528
+                    positionZ (Float) : 29.0630
                 463 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 32212
-                    positionX (Float) : 915.5426
-                    positionY (Float) : -709.9583
-                    positionZ (Float) : 29.0839
+                    positionX (Float) : 915.5304
+                    positionY (Float) : -709.9528
+                    positionZ (Float) : 29.0630
                 464 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 32288
-                    positionX (Float) : 915.7300
-                    positionY (Float) : -708.4915
-                    positionZ (Float) : 29.2906
+                    positionX (Float) : 915.7318
+                    positionY (Float) : -708.5015
+                    positionZ (Float) : 29.2678
                 465 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 32352
-                    positionX (Float) : 915.7300
-                    positionY (Float) : -708.4915
-                    positionZ (Float) : 29.2906
+                    positionX (Float) : 915.7318
+                    positionY (Float) : -708.5015
+                    positionZ (Float) : 29.2678
                 466 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -4985,7 +4985,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 20
                     offset (Uint64) : 32824
-                    positionX (Float) : 915.1518
+                    positionX (Float) : 915.1517
                     positionY (Float) : -715.4954
                     positionZ (Float) : 29.7431
                 473 (CSectorDataObject)
@@ -5082,8 +5082,8 @@
                     radius (Uint16) : 100
                     offset (Uint64) : 33628
                     positionX (Float) : 907.7043
-                    positionY (Float) : -718.7815
-                    positionZ (Float) : 28.4683
+                    positionY (Float) : -718.7814
+                    positionZ (Float) : 28.4682
                 485 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -5091,7 +5091,7 @@
                     offset (Uint64) : 33692
                     positionX (Float) : 907.7295
                     positionY (Float) : -718.7421
-                    positionZ (Float) : 28.4709
+                    positionZ (Float) : 28.4708
                 486 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -5113,17 +5113,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 100
                     offset (Uint64) : 33896
-                    positionX (Float) : 908.8662
-                    positionY (Float) : -718.0533
-                    positionZ (Float) : 28.1565
+                    positionX (Float) : 908.8651
+                    positionY (Float) : -718.0531
+                    positionZ (Float) : 28.1567
                 489 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 33960
-                    positionX (Float) : 908.8662
-                    positionY (Float) : -718.0533
-                    positionZ (Float) : 28.1565
+                    positionX (Float) : 908.8651
+                    positionY (Float) : -718.0531
+                    positionZ (Float) : 28.1567
                 490 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -5207,7 +5207,7 @@
                 500 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 60
+                    radius (Uint16) : 1
                     offset (Uint64) : 34736
                     positionX (Float) : 915.3597
                     positionY (Float) : -715.2332
@@ -5215,7 +5215,7 @@
                 501 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
-                    radius (Uint16) : 60
+                    radius (Uint16) : 1
                     offset (Uint64) : 34800
                     positionX (Float) : 915.3597
                     positionY (Float) : -715.2332
@@ -5550,7 +5550,7 @@
                         bz (Float) : -0.9657
                         by (Float) : -0.1937
                         cz (Float) : -0.0170
-                        cy (Float) : -0.1938
+                        cy (Float) : -0.1937
                         cz (Float) : 0.9809
                     position (SVector3D)
                         X (Float) : 910.4376
@@ -5568,7 +5568,7 @@
                         bz (Float) : -0.9657
                         by (Float) : -0.1937
                         cz (Float) : -0.0170
-                        cy (Float) : -0.1938
+                        cy (Float) : -0.1937
                         cz (Float) : 0.9809
                     position (SVector3D)
                         X (Float) : 910.4376
@@ -14167,7 +14167,7 @@
                         cz (Float) : 0.0000
                     position (SVector3D)
                         X (Float) : 911.8785
-                        Y (Float) : -704.0377
+                        Y (Float) : -704.0376
                         Z (Float) : 30.5972
                     streamingRadius (Uint16) : 0
                     flags (Uint16) : 5
@@ -14185,7 +14185,7 @@
                         cz (Float) : 0.0000
                     position (SVector3D)
                         X (Float) : 911.8785
-                        Y (Float) : -704.0377
+                        Y (Float) : -704.0376
                         Z (Float) : 30.5972
                     streamingRadius (Uint16) : 0
                     flags (Uint16) : 0
