 1 file changed, 266 insertions(+), 266 deletions(-)

diff --git 1.32/dlc/bob/data/levels/bob/locations/vineyards/east/corvo_bianco/basement_deco .w2l.txt 4.00/dlc/bob/data/levels/bob/locations/vineyards/east/corvo_bianco/basement_deco .w2l.txt
index 8b4443b..84a4460 100644
--- 1.32/dlc/bob/data/levels/bob/locations/vineyards/east/corvo_bianco/basement_deco .w2l.txt	
+++ 4.00/dlc/bob/data/levels/bob/locations/vineyards/east/corvo_bianco/basement_deco .w2l.txt	
@@ -269,7 +269,7 @@
                         boundingBox (Box)
                             Min (Vector)
                                 X (Float) : -421.9349
-                                Y (Float) : -817.4035
+                                Y (Float) : -817.4034
                                 Z (Float) : 29.7708
                                 W (Float) : 1.0000
                             Max (Vector)
@@ -316,7 +316,7 @@
                                 Z (Float) : 29.4407
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : -417.2824
+                                X (Float) : -417.2825
                                 Y (Float) : -810.4747
                                 Z (Float) : 30.6321
                                 W (Float) : 1.0000
@@ -344,20 +344,20 @@
                     box5 (Float) : 0.0000
                     pathHash (String) : ""
                 1 (CSectorDataResource)
-                    box0 (Float) : -1.2176
-                    box1 (Float) : -1.2338
-                    box2 (Float) : -0.0020
-                    box3 (Float) : 1.2176
-                    box4 (Float) : 1.2338
-                    box5 (Float) : 2.4887
+                    box0 (Float) : -1.2308
+                    box1 (Float) : -1.2378
+                    box2 (Float) : -0.0117
+                    box3 (Float) : 1.2265
+                    box4 (Float) : 1.2442
+                    box5 (Float) : 2.4989
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/merchant/winery/poor_winery_barrel_big.w2mesh"
                 2 (CSectorDataResource)
-                    box0 (Float) : -0.4396
-                    box1 (Float) : -0.4414
+                    box0 (Float) : -0.4395
+                    box1 (Float) : -0.4361
                     box2 (Float) : -0.0003
-                    box3 (Float) : 0.4411
-                    box4 (Float) : 0.4355
-                    box5 (Float) : 1.0998
+                    box3 (Float) : 0.4410
+                    box4 (Float) : 0.4354
+                    box5 (Float) : 1.0997
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/containers/barrels/gen_barrel_closed_metal.w2mesh"
                 3 (CSectorDataResource)
                     box0 (Float) : -0.2852
@@ -840,28 +840,28 @@
                     box5 (Float) : 0.0315
                     pathHash (String) : "dlc/bob/data/environment/architecture/wooden_parts/bob_wooden_floor_planks_b.w2mesh"
                 63 (CSectorDataResource)
-                    box0 (Float) : -0.3729
-                    box1 (Float) : -0.4383
-                    box2 (Float) : -0.0172
-                    box3 (Float) : 0.3729
-                    box4 (Float) : 0.4383
-                    box5 (Float) : 0.2750
+                    box0 (Float) : -0.3707
+                    box1 (Float) : -0.4453
+                    box2 (Float) : -0.0097
+                    box3 (Float) : 0.3436
+                    box4 (Float) : 0.4404
+                    box5 (Float) : 0.2478
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/hunter/gen_hunter_rope_pile_b.w2mesh"
                 64 (CSectorDataResource)
-                    box0 (Float) : -0.3259
-                    box1 (Float) : -0.2963
-                    box2 (Float) : 0.0049
-                    box3 (Float) : 0.2934
-                    box4 (Float) : 0.2909
-                    box5 (Float) : 0.4113
+                    box0 (Float) : -0.2843
+                    box1 (Float) : -0.3382
+                    box2 (Float) : -0.0011
+                    box3 (Float) : 0.2767
+                    box4 (Float) : 0.2657
+                    box5 (Float) : 0.4038
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/hunter/gen_hunter_rope_pile_a.w2mesh"
                 65 (CSectorDataResource)
-                    box0 (Float) : -1.8039
-                    box1 (Float) : -3.1411
-                    box2 (Float) : -0.0163
-                    box3 (Float) : 1.9057
-                    box4 (Float) : 3.0913
-                    box5 (Float) : 0.3994
+                    box0 (Float) : -1.7482
+                    box1 (Float) : -3.0932
+                    box2 (Float) : -0.1200
+                    box3 (Float) : 1.8614
+                    box4 (Float) : 3.0475
+                    box5 (Float) : 0.3636
                     pathHash (String) : "dlc/bob/data/environment/terrain_surroundings/rock_debris/bob_rock_debris_section.w2mesh"
                 66 (CSectorDataResource)
                     box0 (Float) : -1.0500
@@ -901,7 +901,7 @@
                     box2 (Float) : 0.0008
                     box3 (Float) : 0.3390
                     box4 (Float) : 0.3335
-                    box5 (Float) : 0.0376
+                    box5 (Float) : 0.0373
                     pathHash (String) : "dlc/bob/data/environment/decorations/gameplay/generic/containers/barrels/gen_barrel_lid.w2mesh"
                 71 (CSectorDataResource)
                     box0 (Float) : -0.2755
@@ -966,175 +966,175 @@
                     radius (Uint16) : 80
                     offset (Uint64) : 0
                     positionX (Float) : -410.1661
-                    positionY (Float) : -808.4345
-                    positionZ (Float) : 25.0122
+                    positionY (Float) : -808.4384
+                    positionZ (Float) : 25.0124
                 1 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 64
                     positionX (Float) : -410.1661
-                    positionY (Float) : -808.4345
-                    positionZ (Float) : 25.0122
+                    positionY (Float) : -808.4384
+                    positionZ (Float) : 25.0124
                 2 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 140
-                    positionX (Float) : -415.8874
-                    positionY (Float) : -807.7238
+                    positionX (Float) : -415.8868
+                    positionY (Float) : -807.7212
                     positionZ (Float) : 27.0657
                 3 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 204
-                    positionX (Float) : -415.8874
-                    positionY (Float) : -807.7238
+                    positionX (Float) : -415.8868
+                    positionY (Float) : -807.7212
                     positionZ (Float) : 27.0657
                 4 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 280
-                    positionX (Float) : -418.8995
-                    positionY (Float) : -808.6874
+                    positionX (Float) : -418.8992
+                    positionY (Float) : -808.6848
                     positionZ (Float) : 24.3957
                 5 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 344
-                    positionX (Float) : -418.8995
-                    positionY (Float) : -808.6874
+                    positionX (Float) : -418.8992
+                    positionY (Float) : -808.6848
                     positionZ (Float) : 24.3957
                 6 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 420
-                    positionX (Float) : -408.3095
-                    positionY (Float) : -810.6759
-                    positionZ (Float) : 25.1122
+                    positionX (Float) : -408.3116
+                    positionY (Float) : -810.6791
+                    positionZ (Float) : 25.1124
                 7 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 484
-                    positionX (Float) : -408.3095
-                    positionY (Float) : -810.6759
-                    positionZ (Float) : 25.1122
+                    positionX (Float) : -408.3116
+                    positionY (Float) : -810.6791
+                    positionZ (Float) : 25.1124
                 8 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 560
-                    positionX (Float) : -418.9514
-                    positionY (Float) : -809.5646
+                    positionX (Float) : -418.9510
+                    positionY (Float) : -809.5620
                     positionZ (Float) : 24.3957
                 9 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 624
-                    positionX (Float) : -418.9514
-                    positionY (Float) : -809.5646
+                    positionX (Float) : -418.9510
+                    positionY (Float) : -809.5620
                     positionZ (Float) : 24.3957
                 10 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 700
-                    positionX (Float) : -419.7360
-                    positionY (Float) : -809.0270
+                    positionX (Float) : -419.7357
+                    positionY (Float) : -809.0244
                     positionZ (Float) : 24.3957
                 11 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 764
-                    positionX (Float) : -419.7360
-                    positionY (Float) : -809.0270
+                    positionX (Float) : -419.7357
+                    positionY (Float) : -809.0244
                     positionZ (Float) : 24.3957
                 12 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 840
-                    positionX (Float) : -416.7250
-                    positionY (Float) : -808.1254
+                    positionX (Float) : -416.7245
+                    positionY (Float) : -808.1228
                     positionZ (Float) : 27.0657
                 13 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 904
-                    positionX (Float) : -416.7250
-                    positionY (Float) : -808.1254
+                    positionX (Float) : -416.7245
+                    positionY (Float) : -808.1228
                     positionZ (Float) : 27.0657
                 14 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 980
-                    positionX (Float) : -417.4417
-                    positionY (Float) : -808.6505
+                    positionX (Float) : -417.4411
+                    positionY (Float) : -808.6481
                     positionZ (Float) : 27.0657
                 15 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 1044
-                    positionX (Float) : -417.4417
-                    positionY (Float) : -808.6505
+                    positionX (Float) : -417.4411
+                    positionY (Float) : -808.6481
                     positionZ (Float) : 27.0657
                 16 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 1120
-                    positionX (Float) : -420.5417
-                    positionY (Float) : -809.4894
+                    positionX (Float) : -420.5414
+                    positionY (Float) : -809.4868
                     positionZ (Float) : 24.3957
                 17 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 1184
-                    positionX (Float) : -420.5417
-                    positionY (Float) : -809.4894
+                    positionX (Float) : -420.5414
+                    positionY (Float) : -809.4868
                     positionZ (Float) : 24.3957
                 18 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 1260
-                    positionX (Float) : -414.9735
-                    positionY (Float) : -808.3098
+                    positionX (Float) : -414.9730
+                    positionY (Float) : -808.3074
                     positionZ (Float) : 27.0796
                 19 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 1324
-                    positionX (Float) : -414.9735
-                    positionY (Float) : -808.3098
+                    positionX (Float) : -414.9730
+                    positionY (Float) : -808.3074
                     positionZ (Float) : 27.0796
                 20 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 1400
-                    positionX (Float) : -414.0660
-                    positionY (Float) : -807.7520
+                    positionX (Float) : -414.0655
+                    positionY (Float) : -807.7495
                     positionZ (Float) : 27.0657
                 21 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 1464
-                    positionX (Float) : -414.0660
-                    positionY (Float) : -807.7520
+                    positionX (Float) : -414.0655
+                    positionY (Float) : -807.7495
                     positionZ (Float) : 27.0657
                 22 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1142,7 +1142,7 @@
                     radius (Uint16) : 60
                     offset (Uint64) : 1540
                     positionX (Float) : -398.5303
-                    positionY (Float) : -818.2548
+                    positionY (Float) : -818.2525
                     positionZ (Float) : 26.0966
                 23 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1150,24 +1150,24 @@
                     radius (Uint16) : 60
                     offset (Uint64) : 1604
                     positionX (Float) : -398.5303
-                    positionY (Float) : -818.2548
+                    positionY (Float) : -818.2525
                     positionZ (Float) : 26.0966
                 24 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 1680
-                    positionX (Float) : -389.1649
-                    positionY (Float) : -810.8811
-                    positionZ (Float) : 26.8696
+                    positionX (Float) : -389.1631
+                    positionY (Float) : -810.8845
+                    positionZ (Float) : 26.8699
                 25 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 1744
-                    positionX (Float) : -389.1649
-                    positionY (Float) : -810.8811
-                    positionZ (Float) : 26.8696
+                    positionX (Float) : -389.1631
+                    positionY (Float) : -810.8845
+                    positionZ (Float) : 26.8699
                 26 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -1189,40 +1189,40 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 1960
-                    positionX (Float) : -389.7003
+                    positionX (Float) : -389.7029
                     positionY (Float) : -802.7438
-                    positionZ (Float) : 27.1149
+                    positionZ (Float) : 27.1148
                 29 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 2024
-                    positionX (Float) : -389.7003
+                    positionX (Float) : -389.7029
                     positionY (Float) : -802.7438
-                    positionZ (Float) : 27.1149
+                    positionZ (Float) : 27.1148
                 30 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 2100
-                    positionX (Float) : -386.8005
-                    positionY (Float) : -802.5767
-                    positionZ (Float) : 27.1149
+                    positionX (Float) : -386.7997
+                    positionY (Float) : -802.5791
+                    positionZ (Float) : 27.1148
                 31 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 2164
-                    positionX (Float) : -386.8005
-                    positionY (Float) : -802.5767
-                    positionZ (Float) : 27.1149
+                    positionX (Float) : -386.7997
+                    positionY (Float) : -802.5791
+                    positionZ (Float) : 27.1148
                 32 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 2240
                     positionX (Float) : -396.3545
-                    positionY (Float) : -785.2540
+                    positionY (Float) : -785.2514
                     positionZ (Float) : 25.8368
                 33 (CSectorDataObject)
                     type (Uint8) : 2
@@ -1230,24 +1230,24 @@
                     radius (Uint16) : 60
                     offset (Uint64) : 2304
                     positionX (Float) : -396.3545
-                    positionY (Float) : -785.2540
+                    positionY (Float) : -785.2514
                     positionZ (Float) : 25.8368
                 34 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 2380
-                    positionX (Float) : -395.2784
-                    positionY (Float) : -783.0309
-                    positionZ (Float) : 26.5312
+                    positionX (Float) : -395.2758
+                    positionY (Float) : -783.0338
+                    positionZ (Float) : 26.5314
                 35 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 80
                     offset (Uint64) : 2444
-                    positionX (Float) : -395.2784
-                    positionY (Float) : -783.0309
-                    positionZ (Float) : 26.5312
+                    positionX (Float) : -395.2758
+                    positionY (Float) : -783.0338
+                    positionZ (Float) : 26.5314
                 36 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -1310,7 +1310,7 @@
                     radius (Uint16) : 15
                     offset (Uint64) : 2980
                     positionX (Float) : -398.7208
-                    positionY (Float) : -817.0439
+                    positionY (Float) : -817.0440
                     positionZ (Float) : 26.0738
                 44 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1381,7 +1381,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 15
                     offset (Uint64) : 3568
-                    positionX (Float) : -395.7296
+                    positionX (Float) : -395.7295
                     positionY (Float) : -817.3834
                     positionZ (Float) : 25.8632
                 53 (CSectorDataObject)
@@ -1429,7 +1429,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 15
                     offset (Uint64) : 3952
-                    positionX (Float) : -395.6832
+                    positionX (Float) : -395.6831
                     positionY (Float) : -817.6927
                     positionZ (Float) : 26.0790
                 59 (CSectorDataObject)
@@ -1558,7 +1558,7 @@
                     radius (Uint16) : 15
                     offset (Uint64) : 4988
                     positionX (Float) : -398.0378
-                    positionY (Float) : -818.3324
+                    positionY (Float) : -818.3325
                     positionZ (Float) : 26.7702
                 75 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1574,7 +1574,7 @@
                     radius (Uint16) : 15
                     offset (Uint64) : 5116
                     positionX (Float) : -397.5408
-                    positionY (Float) : -818.2823
+                    positionY (Float) : -818.2824
                     positionZ (Float) : 26.7716
                 77 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1590,7 +1590,7 @@
                     radius (Uint16) : 15
                     offset (Uint64) : 5244
                     positionX (Float) : -398.0596
-                    positionY (Float) : -818.3346
+                    positionY (Float) : -818.3347
                     positionZ (Float) : 26.5245
                 79 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1734,7 +1734,7 @@
                     radius (Uint16) : 15
                     offset (Uint64) : 6408
                     positionX (Float) : -396.7642
-                    positionY (Float) : -818.2097
+                    positionY (Float) : -818.2098
                     positionZ (Float) : 26.7716
                 97 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1758,7 +1758,7 @@
                     radius (Uint16) : 15
                     offset (Uint64) : 6600
                     positionX (Float) : -396.7652
-                    positionY (Float) : -818.2098
+                    positionY (Float) : -818.2099
                     positionZ (Float) : 26.5240
                 100 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1878,7 +1878,7 @@
                     radius (Uint16) : 15
                     offset (Uint64) : 7572
                     positionX (Float) : -396.4647
-                    positionY (Float) : -818.1853
+                    positionY (Float) : -818.1854
                     positionZ (Float) : 26.7702
                 115 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1902,7 +1902,7 @@
                     radius (Uint16) : 15
                     offset (Uint64) : 7764
                     positionX (Float) : -396.3071
-                    positionY (Float) : -818.1692
+                    positionY (Float) : -818.1693
                     positionZ (Float) : 26.7701
                 118 (CSectorDataObject)
                     type (Uint8) : 1
@@ -1910,7 +1910,7 @@
                     radius (Uint16) : 15
                     offset (Uint64) : 7828
                     positionX (Float) : -396.4865
-                    positionY (Float) : -818.1875
+                    positionY (Float) : -818.1876
                     positionZ (Float) : 26.5245
                 119 (CSectorDataObject)
                     type (Uint8) : 1
@@ -2925,16 +2925,16 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 16592
-                    positionX (Float) : -419.8085
-                    positionY (Float) : -810.1320
+                    positionX (Float) : -419.8106
+                    positionY (Float) : -810.1335
                     positionZ (Float) : 24.4357
                 246 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 16656
-                    positionX (Float) : -419.8085
-                    positionY (Float) : -810.1320
+                    positionX (Float) : -419.8106
+                    positionY (Float) : -810.1335
                     positionZ (Float) : 24.4357
                 247 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3029,16 +3029,16 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 17472
-                    positionX (Float) : -422.7893
-                    positionY (Float) : -814.5322
+                    positionX (Float) : -422.7918
+                    positionY (Float) : -814.5316
                     positionZ (Float) : 29.5263
                 259 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 17536
-                    positionX (Float) : -422.7893
-                    positionY (Float) : -814.5322
+                    positionX (Float) : -422.7918
+                    positionY (Float) : -814.5316
                     positionZ (Float) : 29.5263
                 260 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3077,32 +3077,32 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 17892
-                    positionX (Float) : -423.1947
-                    positionY (Float) : -815.5977
+                    positionX (Float) : -423.1972
+                    positionY (Float) : -815.5971
                     positionZ (Float) : 29.5263
                 265 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 17956
-                    positionX (Float) : -423.1947
-                    positionY (Float) : -815.5977
+                    positionX (Float) : -423.1972
+                    positionY (Float) : -815.5971
                     positionZ (Float) : 29.5263
                 266 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 18032
-                    positionX (Float) : -422.5210
-                    positionY (Float) : -816.4669
+                    positionX (Float) : -422.5235
+                    positionY (Float) : -816.4663
                     positionZ (Float) : 29.5263
                 267 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 18096
-                    positionX (Float) : -422.5210
-                    positionY (Float) : -816.4669
+                    positionX (Float) : -422.5235
+                    positionY (Float) : -816.4663
                     positionZ (Float) : 29.5263
                 268 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3205,16 +3205,16 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 19012
-                    positionX (Float) : -422.3048
-                    positionY (Float) : -815.5829
+                    positionX (Float) : -422.3073
+                    positionY (Float) : -815.5823
                     positionZ (Float) : 29.5263
                 281 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 19076
-                    positionX (Float) : -422.3048
-                    positionY (Float) : -815.5829
+                    positionX (Float) : -422.3073
+                    positionY (Float) : -815.5823
                     positionZ (Float) : 29.5263
                 282 (CSectorDataObject)
                     type (Uint8) : 1
@@ -3549,7 +3549,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 200
                     offset (Uint64) : 22016
-                    positionX (Float) : -422.1433
+                    positionX (Float) : -422.1434
                     positionY (Float) : -815.4186
                     positionZ (Float) : 28.9488
                 324 (CSectorDataObject)
@@ -3557,7 +3557,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 200
                     offset (Uint64) : 22080
-                    positionX (Float) : -422.1433
+                    positionX (Float) : -422.1434
                     positionY (Float) : -815.4186
                     positionZ (Float) : 28.9488
                 325 (CSectorDataObject)
@@ -4149,16 +4149,16 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 27236
-                    positionX (Float) : -422.5598
-                    positionY (Float) : -817.4080
+                    positionX (Float) : -422.5594
+                    positionY (Float) : -817.4054
                     positionZ (Float) : 29.5263
                 399 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 27300
-                    positionX (Float) : -422.5598
-                    positionY (Float) : -817.4080
+                    positionX (Float) : -422.5594
+                    positionY (Float) : -817.4054
                     positionZ (Float) : 29.5263
                 400 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4302,7 +4302,7 @@
                     radius (Uint16) : 50
                     offset (Uint64) : 28560
                     positionX (Float) : -415.9787
-                    positionY (Float) : -807.7452
+                    positionY (Float) : -807.7453
                     positionZ (Float) : 29.5207
                 418 (CSectorDataObject)
                     type (Uint8) : 2
@@ -4310,7 +4310,7 @@
                     radius (Uint16) : 50
                     offset (Uint64) : 28624
                     positionX (Float) : -415.9787
-                    positionY (Float) : -807.7452
+                    positionY (Float) : -807.7453
                     positionZ (Float) : 29.5207
                 419 (CSectorDataObject)
                     type (Uint8) : 1
@@ -4837,32 +4837,32 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 33244
-                    positionX (Float) : -423.5067
-                    positionY (Float) : -817.3313
-                    positionZ (Float) : 29.6664
+                    positionX (Float) : -423.5063
+                    positionY (Float) : -817.3287
+                    positionZ (Float) : 29.6663
                 485 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 33308
-                    positionX (Float) : -423.5067
-                    positionY (Float) : -817.3313
-                    positionZ (Float) : 29.6664
+                    positionX (Float) : -423.5063
+                    positionY (Float) : -817.3287
+                    positionZ (Float) : 29.6663
                 486 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 33384
-                    positionX (Float) : -423.5160
-                    positionY (Float) : -816.4441
+                    positionX (Float) : -423.5185
+                    positionY (Float) : -816.4435
                     positionZ (Float) : 29.6163
                 487 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 33448
-                    positionX (Float) : -423.5160
-                    positionY (Float) : -816.4441
+                    positionX (Float) : -423.5185
+                    positionY (Float) : -816.4435
                     positionZ (Float) : 29.6163
                 488 (CSectorDataObject)
                     type (Uint8) : 1
@@ -5165,16 +5165,16 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 36108
-                    positionX (Float) : -415.9296
-                    positionY (Float) : -808.9985
+                    positionX (Float) : -415.9271
+                    positionY (Float) : -808.9984
                     positionZ (Float) : 26.8819
                 526 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 36172
-                    positionX (Float) : -415.9296
-                    positionY (Float) : -808.9985
+                    positionX (Float) : -415.9271
+                    positionY (Float) : -808.9984
                     positionZ (Float) : 26.8819
                 527 (CSectorDataObject)
                     type (Uint8) : 1
@@ -5437,16 +5437,16 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 38488
-                    positionX (Float) : -414.9083
-                    positionY (Float) : -807.4468
+                    positionX (Float) : -414.9077
+                    positionY (Float) : -807.4443
                     positionZ (Float) : 27.0457
                 560 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 38552
-                    positionX (Float) : -414.9083
-                    positionY (Float) : -807.4468
+                    positionX (Float) : -414.9077
+                    positionY (Float) : -807.4443
                     positionZ (Float) : 27.0457
                 561 (CSectorDataObject)
                     type (Uint8) : 1
@@ -5894,7 +5894,7 @@
                     radius (Uint16) : 50
                     offset (Uint64) : 42452
                     positionX (Float) : -416.1264
-                    positionY (Float) : -814.4873
+                    positionY (Float) : -814.4874
                     positionZ (Float) : 23.4257
                 617 (CSectorDataObject)
                     type (Uint8) : 2
@@ -5902,7 +5902,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 42516
                     positionX (Float) : -416.1264
-                    positionY (Float) : -814.4873
+                    positionY (Float) : -814.4874
                     positionZ (Float) : 23.4257
                 618 (CSectorDataObject)
                     type (Uint8) : 1
@@ -5942,7 +5942,7 @@
                     radius (Uint16) : 60
                     offset (Uint64) : 42872
                     positionX (Float) : -415.1640
-                    positionY (Float) : -821.2869
+                    positionY (Float) : -821.2870
                     positionZ (Float) : 23.5026
                 623 (CSectorDataObject)
                     type (Uint8) : 1
@@ -5971,27 +5971,27 @@
                 626 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 30
+                    radius (Uint16) : 38
                     offset (Uint64) : 43128
-                    positionX (Float) : -410.6508
-                    positionY (Float) : -812.1223
-                    positionZ (Float) : 23.6285
+                    positionX (Float) : -410.6430
+                    positionY (Float) : -812.1110
+                    positionZ (Float) : 23.6211
                 627 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 30
+                    radius (Uint16) : 38
                     offset (Uint64) : 43192
-                    positionX (Float) : -416.3245
-                    positionY (Float) : -812.2467
-                    positionZ (Float) : 23.5739
+                    positionX (Float) : -416.3206
+                    positionY (Float) : -812.2598
+                    positionZ (Float) : 23.5663
                 628 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 40
                     offset (Uint64) : 43256
-                    positionX (Float) : -423.8588
-                    positionY (Float) : -813.5955
-                    positionZ (Float) : 24.1891
+                    positionX (Float) : -423.8463
+                    positionY (Float) : -813.6290
+                    positionZ (Float) : 24.1823
                 629 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6021,41 +6021,41 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 43524
-                    positionX (Float) : -417.4104
-                    positionY (Float) : -820.1022
-                    positionZ (Float) : 23.4744
+                    positionX (Float) : -417.4086
+                    positionY (Float) : -820.1030
+                    positionZ (Float) : 23.4512
                 633 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 43588
-                    positionX (Float) : -418.8710
-                    positionY (Float) : -821.0773
-                    positionZ (Float) : 23.5016
+                    positionX (Float) : -418.8699
+                    positionY (Float) : -821.0781
+                    positionZ (Float) : 23.4783
                 634 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 43652
-                    positionX (Float) : -420.1722
-                    positionY (Float) : -820.2994
-                    positionZ (Float) : 23.5272
+                    positionX (Float) : -420.1724
+                    positionY (Float) : -820.3013
+                    positionZ (Float) : 23.5039
                 635 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 43716
-                    positionX (Float) : -417.5873
-                    positionY (Float) : -812.6191
-                    positionZ (Float) : 23.5377
+                    positionX (Float) : -417.5875
+                    positionY (Float) : -812.6210
+                    positionZ (Float) : 23.5144
                 636 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 43780
-                    positionX (Float) : -412.4066
-                    positionY (Float) : -810.5451
-                    positionZ (Float) : 23.4989
+                    positionX (Float) : -412.4069
+                    positionY (Float) : -810.5470
+                    positionZ (Float) : 23.4757
                 637 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6093,9 +6093,9 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 44124
-                    positionX (Float) : -413.9755
-                    positionY (Float) : -819.7612
-                    positionZ (Float) : 23.4087
+                    positionX (Float) : -413.9757
+                    positionY (Float) : -819.7589
+                    positionZ (Float) : 23.3855
                 642 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6118,7 +6118,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 44328
                     positionX (Float) : -417.6529
-                    positionY (Float) : -809.2151
+                    positionY (Float) : -809.2150
                     positionZ (Float) : 23.6961
                 645 (CSectorDataObject)
                     type (Uint8) : 2
@@ -6126,7 +6126,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 44392
                     positionX (Float) : -417.6529
-                    positionY (Float) : -809.2151
+                    positionY (Float) : -809.2150
                     positionZ (Float) : 23.6961
                 646 (CSectorDataObject)
                     type (Uint8) : 1
@@ -6213,17 +6213,17 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 45156
-                    positionX (Float) : -417.9033
-                    positionY (Float) : -815.4666
-                    positionZ (Float) : 23.5220
+                    positionX (Float) : -417.9044
+                    positionY (Float) : -815.4659
+                    positionZ (Float) : 23.5072
                 657 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 45220
-                    positionX (Float) : -417.6400
-                    positionY (Float) : -813.8711
-                    positionZ (Float) : 23.5472
+                    positionX (Float) : -417.6410
+                    positionY (Float) : -813.8710
+                    positionZ (Float) : 23.5324
                 658 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6259,11 +6259,11 @@
                 662 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 40
+                    radius (Uint16) : 50
                     offset (Uint64) : 45564
                     positionX (Float) : -418.5504
                     positionY (Float) : -817.2009
-                    positionZ (Float) : 23.5436
+                    positionZ (Float) : 23.5435
                 663 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6302,7 +6302,7 @@
                     radius (Uint16) : 80
                     offset (Uint64) : 45908
                     positionX (Float) : -413.6852
-                    positionY (Float) : -826.0278
+                    positionY (Float) : -826.0277
                     positionZ (Float) : 23.7426
                 668 (CSectorDataObject)
                     type (Uint8) : 2
@@ -6310,7 +6310,7 @@
                     radius (Uint16) : 200
                     offset (Uint64) : 45972
                     positionX (Float) : -413.6852
-                    positionY (Float) : -826.0278
+                    positionY (Float) : -826.0277
                     positionZ (Float) : 23.7426
                 669 (CSectorDataObject)
                     type (Uint8) : 1
@@ -6333,9 +6333,9 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 46188
-                    positionX (Float) : -418.2551
-                    positionY (Float) : -824.0097
-                    positionZ (Float) : 23.5512
+                    positionX (Float) : -418.2521
+                    positionY (Float) : -824.0121
+                    positionZ (Float) : 23.5282
                 672 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6357,57 +6357,57 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 46392
-                    positionX (Float) : -413.6240
-                    positionY (Float) : -823.3692
-                    positionZ (Float) : 23.6567
+                    positionX (Float) : -413.6219
+                    positionY (Float) : -823.3681
+                    positionZ (Float) : 23.6336
                 675 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 46456
-                    positionX (Float) : -412.0638
-                    positionY (Float) : -819.3757
-                    positionZ (Float) : 23.5316
+                    positionX (Float) : -412.0626
+                    positionY (Float) : -819.3773
+                    positionZ (Float) : 23.5084
                 676 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 46520
-                    positionX (Float) : -418.2530
-                    positionY (Float) : -816.4178
-                    positionZ (Float) : 23.5312
+                    positionX (Float) : -418.2531
+                    positionY (Float) : -816.4182
+                    positionZ (Float) : 23.5210
                 677 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 46584
-                    positionX (Float) : -420.6678
-                    positionY (Float) : -814.1959
-                    positionZ (Float) : 23.7959
+                    positionX (Float) : -420.6688
+                    positionY (Float) : -814.1930
+                    positionZ (Float) : 23.7750
                 678 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 46648
-                    positionX (Float) : -420.3597
-                    positionY (Float) : -827.1234
-                    positionZ (Float) : 23.8279
+                    positionX (Float) : -420.3600
+                    positionY (Float) : -827.1252
+                    positionZ (Float) : 23.8133
                 679 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 46712
-                    positionX (Float) : -420.4872
-                    positionY (Float) : -826.7627
-                    positionZ (Float) : 23.8082
+                    positionX (Float) : -420.4865
+                    positionY (Float) : -826.7649
+                    positionZ (Float) : 23.7936
                 680 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
-                    radius (Uint16) : 30
+                    radius (Uint16) : 38
                     offset (Uint64) : 46776
-                    positionX (Float) : -408.8676
-                    positionY (Float) : -807.3226
-                    positionZ (Float) : 24.0630
+                    positionX (Float) : -408.8544
+                    positionY (Float) : -807.3187
+                    positionZ (Float) : 24.0556
                 681 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6669,7 +6669,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 40
                     offset (Uint64) : 48984
-                    positionX (Float) : -409.8046
+                    positionX (Float) : -409.8045
                     positionY (Float) : -809.7452
                     positionZ (Float) : 24.3740
                 714 (CSectorDataObject)
@@ -6677,7 +6677,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 40
                     offset (Uint64) : 49048
-                    positionX (Float) : -409.8046
+                    positionX (Float) : -409.8045
                     positionY (Float) : -809.7452
                     positionZ (Float) : 24.3740
                 715 (CSectorDataObject)
@@ -6829,23 +6829,23 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 50384
-                    positionX (Float) : -417.6996
-                    positionY (Float) : -807.7954
+                    positionX (Float) : -417.6993
+                    positionY (Float) : -807.7928
                     positionZ (Float) : 24.5039
                 734 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 50448
-                    positionX (Float) : -417.6996
-                    positionY (Float) : -807.7954
+                    positionX (Float) : -417.6993
+                    positionY (Float) : -807.7928
                     positionZ (Float) : 24.5039
                 735 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 50524
-                    positionX (Float) : -417.0463
+                    positionX (Float) : -417.0438
                     positionY (Float) : -807.3490
                     positionZ (Float) : 24.5039
                 736 (CSectorDataObject)
@@ -6853,7 +6853,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 50588
-                    positionX (Float) : -417.0463
+                    positionX (Float) : -417.0438
                     positionY (Float) : -807.3490
                     positionZ (Float) : 24.5039
                 737 (CSectorDataObject)
@@ -6861,7 +6861,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 50664
-                    positionX (Float) : -416.1414
+                    positionX (Float) : -416.1388
                     positionY (Float) : -807.5835
                     positionZ (Float) : 24.5039
                 738 (CSectorDataObject)
@@ -6869,7 +6869,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 50728
-                    positionX (Float) : -416.1414
+                    positionX (Float) : -416.1388
                     positionY (Float) : -807.5835
                     positionZ (Float) : 24.5039
                 739 (CSectorDataObject)
@@ -6878,7 +6878,7 @@
                     radius (Uint16) : 60
                     offset (Uint64) : 50804
                     positionX (Float) : -415.9096
-                    positionY (Float) : -806.6555
+                    positionY (Float) : -806.6581
                     positionZ (Float) : 24.5212
                 740 (CSectorDataObject)
                     type (Uint8) : 2
@@ -6886,24 +6886,24 @@
                     radius (Uint16) : 60
                     offset (Uint64) : 50868
                     positionX (Float) : -415.9096
-                    positionY (Float) : -806.6555
+                    positionY (Float) : -806.6581
                     positionZ (Float) : 24.5212
                 741 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 50944
-                    positionX (Float) : -415.1151
-                    positionY (Float) : -806.3795
-                    positionZ (Float) : 24.3907
+                    positionX (Float) : -415.1154
+                    positionY (Float) : -806.3796
+                    positionZ (Float) : 24.3882
                 742 (CSectorDataObject)
                     type (Uint8) : 2
                     flags (Uint8) : 0
                     radius (Uint16) : 60
                     offset (Uint64) : 51008
-                    positionX (Float) : -415.1151
-                    positionY (Float) : -806.3795
-                    positionZ (Float) : 24.3907
+                    positionX (Float) : -415.1154
+                    positionY (Float) : -806.3796
+                    positionZ (Float) : 24.3882
                 743 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
@@ -6957,7 +6957,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 40
                     offset (Uint64) : 51504
-                    positionX (Float) : -408.4356
+                    positionX (Float) : -408.4355
                     positionY (Float) : -808.6510
                     positionZ (Float) : 24.7711
                 750 (CSectorDataObject)
@@ -6965,7 +6965,7 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 40
                     offset (Uint64) : 51568
-                    positionX (Float) : -408.4356
+                    positionX (Float) : -408.4355
                     positionY (Float) : -808.6510
                     positionZ (Float) : 24.7711
                 751 (CSectorDataObject)
@@ -7141,33 +7141,33 @@
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 53060
-                    positionX (Float) : -426.1819
-                    positionY (Float) : -821.7280
-                    positionZ (Float) : 24.0054
+                    positionX (Float) : -426.1823
+                    positionY (Float) : -821.7294
+                    positionZ (Float) : 23.9907
                 773 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 53124
-                    positionX (Float) : -426.5166
-                    positionY (Float) : -818.9069
-                    positionZ (Float) : 24.0068
+                    positionX (Float) : -426.5171
+                    positionY (Float) : -818.9082
+                    positionZ (Float) : 23.9920
                 774 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 53188
-                    positionX (Float) : -424.4066
-                    positionY (Float) : -821.9136
-                    positionZ (Float) : 23.9553
+                    positionX (Float) : -424.4070
+                    positionY (Float) : -821.9149
+                    positionZ (Float) : 23.9406
                 775 (CSectorDataObject)
                     type (Uint8) : 1
                     flags (Uint8) : 0
                     radius (Uint16) : 50
                     offset (Uint64) : 53252
-                    positionX (Float) : -425.5014
-                    positionY (Float) : -818.7265
-                    positionZ (Float) : 23.9606
+                    positionX (Float) : -425.5007
+                    positionY (Float) : -818.7277
+                    positionZ (Float) : 23.9459
             blocksize (CVLQInt32) : 53316
             BlockData (CCompressedBuffer:SBlockData)
                 0 (SBlockData)
@@ -13556,7 +13556,7 @@
                         cy (Float) : 0.0000
                         cz (Float) : 1.0000
                     position (SVector3D)
-                        X (Float) : -422.1433
+                        X (Float) : -422.1434
                         Y (Float) : -815.4186
                         Z (Float) : 28.9488
                     streamingRadius (Uint16) : 0
@@ -13574,7 +13574,7 @@
                         cy (Float) : 0.0000
                         cz (Float) : 1.0000
                     position (SVector3D)
-                        X (Float) : -422.1433
+                        X (Float) : -422.1434
                         Y (Float) : -815.4186
                         Z (Float) : 28.9488
                     streamingRadius (Uint16) : 0
@@ -18583,7 +18583,7 @@
                 570 (SBlockData)
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : -0.0148
-                        ay (Float) : 0.1965
+                        ay (Float) : 0.1964
                         az (Float) : 4.1135
                         by (Float) : 1.9177
                         bz (Float) : 3.3335
@@ -18601,7 +18601,7 @@
                 571 (SBlockData)
                     rotationMatrix (CMatrix3x3)
                         ax (Float) : -0.0148
-                        ay (Float) : 0.1965
+                        ay (Float) : 0.1964
                         az (Float) : 4.1135
                         by (Float) : 1.9177
                         bz (Float) : 3.3335
