 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/noble_woman_corpse/model/corpse_05_noble_woman.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/noble_woman_corpse/model/corpse_05_noble_woman.w2mesh.txt
index ea5842f..55a147e 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/noble_woman_corpse/model/corpse_05_noble_woman.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/noble_woman_corpse/model/corpse_05_noble_woman.w2mesh.txt	
@@ -8,7 +8,7 @@
                     X (Float) : -0.7227
                     Y (Float) : -0.8923
                     Z (Float) : -0.0546
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.4128
                     Y (Float) : 0.9971
@@ -46,10 +46,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 139033300000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : -0.0641
+                    Y (Float) : -0.6851
+                    Z (Float) : 0.1174
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                     2 (Float) : 12.0000
@@ -69,6 +69,8 @@
                 vertexBufferSize (Uint32) : 632256
                 indexBufferSize (Uint32) : 279210
                 indexBufferOffset (Uint32) : 632832
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
