 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git 1.32/levels/skellige/env_global_pbr/shore_cave_sq209/2nd cave/env.w2l.txt 4.00/levels/skellige/env_global_pbr/shore_cave_sq209/2nd cave/env.w2l.txt
index 5ce78a4..5293dfa 100644
--- 1.32/levels/skellige/env_global_pbr/shore_cave_sq209/2nd cave/env.w2l.txt	
+++ 4.00/levels/skellige/env_global_pbr/shore_cave_sq209/2nd cave/env.w2l.txt	
@@ -35,7 +35,7 @@
                     W (Float) : 1.0000
                 Max (Vector)
                     X (Float) : 381.7312
-                    Y (Float) : -881.3879
+                    Y (Float) : -881.3880
                     Z (Float) : 29.0978
                     W (Float) : 1.0000
             terrainSide (EAreaTerrainSide) : ATS_OnlyBelowTerrain
@@ -132,18 +132,18 @@
                     Z (Float) : -67.7103
                     W (Float) : 1.0000
                 1 (Vector)
-                    X (Float) : -8.4850
-                    Y (Float) : -881.3879
+                    X (Float) : -8.4849
+                    Y (Float) : -881.3880
                     Z (Float) : -67.7103
                     W (Float) : 1.0000
                 2 (Vector)
-                    X (Float) : -96.0509
+                    X (Float) : -96.0508
                     Y (Float) : -924.2830
                     Z (Float) : -67.7103
                     W (Float) : 1.0000
                 3 (Vector)
                     X (Float) : -118.2461
-                    Y (Float) : -999.1137
+                    Y (Float) : -999.1138
                     Z (Float) : -67.7103
                     W (Float) : 1.0000
                 4 (Vector)
