 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse07.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse07.w2mesh.txt
index d58261d..f8db4e2 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse07.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse07.w2mesh.txt	
@@ -7,7 +7,7 @@
                     X (Float) : -0.6694
                     Y (Float) : -0.2318
                     Z (Float) : -0.1537
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.3552
                     Y (Float) : 0.9957
@@ -33,10 +33,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 75388010000000000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : -0.0005
+                    Y (Float) : 0.2170
+                    Z (Float) : 0.7280
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                     2 (Float) : 12.0000
@@ -56,6 +56,8 @@
                 vertexBufferSize (Uint32) : 444960
                 indexBufferSize (Uint32) : 156786
                 indexBufferOffset (Uint32) : 445440
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
