 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse08.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse08.w2mesh.txt
index 53308b7..ec7d034 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse08.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse08.w2mesh.txt	
@@ -7,9 +7,9 @@
                     X (Float) : -1.0483
                     Y (Float) : -0.3029
                     Z (Float) : -0.1118
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
-                    X (Float) : 0.3231
+                    X (Float) : 0.3230
                     Y (Float) : 0.9980
                     Z (Float) : 0.5661
                     W (Float) : 1.0000
@@ -33,10 +33,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 75395310000000000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : -0.6507
+                    Y (Float) : -0.0635
+                    Z (Float) : 0.3498
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                     2 (Float) : 12.0000
@@ -56,6 +56,8 @@
                 vertexBufferSize (Uint32) : 437088
                 indexBufferSize (Uint32) : 151392
                 indexBufferOffset (Uint32) : 437248
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
