 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse03.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse03.w2mesh.txt
index 3e7d0bf..75dc854 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse03.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/knight_corpses/model/knight_corpse03.w2mesh.txt	
@@ -7,11 +7,11 @@
                     X (Float) : -0.5910
                     Y (Float) : -0.9559
                     Z (Float) : -0.2188
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.6598
                     Y (Float) : 1.0130
-                    Z (Float) : 0.3477
+                    Z (Float) : 0.3476
                     W (Float) : 1.0000
             autoHideDistance (Float) : 80.0000
             isTwoSided (Bool) : true
@@ -33,10 +33,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 75388010000000000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : 0.3709
+                    Y (Float) : -0.7187
+                    Z (Float) : 0.1089
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                     2 (Float) : 12.0000
@@ -56,6 +56,8 @@
                 vertexBufferSize (Uint32) : 442456
                 indexBufferSize (Uint32) : 152988
                 indexBufferOffset (Uint32) : 443392
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
