 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/children.w2l.txt 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/children.w2l.txt
index 814f1fb..89d1a3f 100644
--- 1.32/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/children.w2l.txt	
+++ 4.00/dlc/bob/data/levels/bob/community/beauclair/lower_city/lower city/children.w2l.txt	
@@ -577,7 +577,7 @@
                             Min (Vector)
                                 X (Float) : -471.4232
                                 Y (Float) : -1208.7130
-                                Z (Float) : 71.1041
+                                Z (Float) : 71.1040
                                 W (Float) : 1.0000
                             Max (Vector)
                                 X (Float) : -471.2834
