 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/bob_guard_lvl1/model/corpse_02_bob_guard_lvl1.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/bob_guard_lvl1/model/corpse_02_bob_guard_lvl1.w2mesh.txt
index 5693d09..f3c27c1 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/bob_guard_lvl1/model/corpse_02_bob_guard_lvl1.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/bob_guard_lvl1/model/corpse_02_bob_guard_lvl1.w2mesh.txt	
@@ -7,7 +7,7 @@
                     X (Float) : -0.8470
                     Y (Float) : -0.9860
                     Z (Float) : -0.0825
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.4155
                     Y (Float) : 1.0644
@@ -32,10 +32,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 75350770000000000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : -0.0228
+                    Y (Float) : -0.8241
+                    Z (Float) : 0.1089
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                     2 (Float) : 12.0000
@@ -55,6 +55,8 @@
                 vertexBufferSize (Uint32) : 570168
                 indexBufferSize (Uint32) : 249312
                 indexBufferOffset (Uint32) : 570368
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
