 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/bob_guard_lvl1/model/corpse_03_bob_guard_lvl1.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/bob_guard_lvl1/model/corpse_03_bob_guard_lvl1.w2mesh.txt
index ad50a6d..4d8742c 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/bob_guard_lvl1/model/corpse_03_bob_guard_lvl1.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/bob_guard_lvl1/model/corpse_03_bob_guard_lvl1.w2mesh.txt	
@@ -7,7 +7,7 @@
                     X (Float) : -0.5902
                     Y (Float) : -0.8527
                     Z (Float) : -0.0950
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.7434
                     Y (Float) : 1.0341
@@ -32,10 +32,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 75328140000000000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : 0.3709
+                    Y (Float) : -0.7187
+                    Z (Float) : 0.1089
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                     2 (Float) : 12.0000
@@ -55,6 +55,8 @@
                 vertexBufferSize (Uint32) : 567736
                 indexBufferSize (Uint32) : 249318
                 indexBufferOffset (Uint32) : 568320
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
