 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git 1.32/characters/models/main_npc/talar/model/i_02_ma__talar .w2mesh.txt 4.00/characters/models/main_npc/talar/model/i_02_ma__talar .w2mesh.txt
index 150caa6..c55c4e9 100644
--- 1.32/characters/models/main_npc/talar/model/i_02_ma__talar .w2mesh.txt	
+++ 4.00/characters/models/main_npc/talar/model/i_02_ma__talar .w2mesh.txt	
@@ -9,11 +9,11 @@
                     X (Float) : -0.2069
                     Y (Float) : 0.0855
                     Z (Float) : 0.8560
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.1824
                     Y (Float) : 0.2350
-                    Z (Float) : 1.3842
+                    Z (Float) : 1.3841
                     W (Float) : 1.0000
             autoHideDistance (Float) : 25.0000
             generalizedMeshRadius (Float) : 0.2138
@@ -127,6 +127,8 @@
                 vertexBufferSize (Uint32) : 67104
                 indexBufferSize (Uint32) : 18684
                 indexBufferOffset (Uint32) : 67584
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
