 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__torso_01_censored.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__torso_01_censored.w2mesh.txt
index 3173e6a..cb23a94 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__torso_01_censored.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__torso_01_censored.w2mesh.txt	
@@ -8,7 +8,7 @@
                     X (Float) : -0.3024
                     Y (Float) : -0.9344
                     Z (Float) : -0.0033
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.2345
                     Y (Float) : 0.0930
@@ -29,10 +29,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 9412958000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : -0.0298
+                    Y (Float) : -0.8080
+                    Z (Float) : 0.1210
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                 renderChunks (array:Uint8) : # array:Uint8
@@ -51,6 +51,8 @@
                 vertexBufferSize (Uint32) : 90768
                 indexBufferSize (Uint32) : 37452
                 indexBufferOffset (Uint32) : 91136
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
