 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__torso_01.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__torso_01.w2mesh.txt
index 8a42d2e..539269d 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__torso_01.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__torso_01.w2mesh.txt	
@@ -9,7 +9,7 @@
                     X (Float) : -0.3316
                     Y (Float) : -0.9352
                     Z (Float) : -0.0087
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.3155
                     Y (Float) : 0.3175
@@ -48,10 +48,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 139341500000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : -0.0298
+                    Y (Float) : -0.8080
+                    Z (Float) : 0.1210
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                     2 (Float) : 12.0000
@@ -71,6 +71,8 @@
                 vertexBufferSize (Uint32) : 371536
                 indexBufferSize (Uint32) : 213426
                 indexBufferOffset (Uint32) : 371712
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
