 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__leg_02.w2mesh.txt 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__leg_02.w2mesh.txt
index 7e06a0a..4c71208 100644
--- 1.32/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__leg_02.w2mesh.txt	
+++ 4.00/dlc/bob/data/environment/decorations/corpses/human corpses/half_eaten_corpses/model/half_eaten_corpses__leg_02.w2mesh.txt	
@@ -9,7 +9,7 @@
                     X (Float) : -0.5106
                     Y (Float) : -0.2910
                     Z (Float) : -0.0141
-                    W (Float) : 1.0000
+                    W (Float) : 0.0000
                 Max (Vector)
                     X (Float) : 0.2249
                     Y (Float) : 0.3977
@@ -42,10 +42,10 @@
             isStatic (Bool) : true
             cookedData (SMeshCookedData)
                 collisionInitPositionOffset (Vector)
-                    X (Float) : 0.0000
-                    Y (Float) : 0.0000
-                    Z (Float) : 32284410000000000000000000000000000.0000
-                    W (Float) : 0.0000
+                    X (Float) : -0.0873
+                    Y (Float) : 0.0819
+                    Z (Float) : 0.0570
+                    W (Float) : 1.0000
                 renderLODs (array:Float)
                     1 (Float) : 5.0000
                 renderChunks (array:Uint8) : # array:Uint8
@@ -64,6 +64,8 @@
                 vertexBufferSize (Uint32) : 42224
                 indexBufferSize (Uint32) : 25248
                 indexBufferOffset (Uint32) : 43008
+                blasBuffer (DeferredDataBuffer)
+                    Bufferdata (Int16) : 0
             internalVersion (Uint8) : 2
             ChunkgroupIndeces (CBufferVLQInt32:CPaddedBuffer:CUInt16)
                 0 (CPaddedBuffer:CUInt16)
