 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git 1.32/levels/novigrad/gameplay/herbs/balisse fruit.w2l.txt 4.00/levels/novigrad/gameplay/herbs/balisse fruit.w2l.txt
index fc05adb..4f2d7c6 100644
--- 1.32/levels/novigrad/gameplay/herbs/balisse fruit.w2l.txt	
+++ 4.00/levels/novigrad/gameplay/herbs/balisse fruit.w2l.txt	
@@ -2858,10 +2858,10 @@
             transform (EngineTransform)
                 Pitch (Float) : 0.0000
                 Yaw (Float) : 0.0000
-                Roll (Float) : 232.9090
-                X (Float) : 2285.4690
-                Y (Float) : 2235.2750
-                Z (Float) : 21.6862
+                Roll (Float) : 246.8453
+                X (Float) : 2285.6890
+                Y (Float) : 2235.0940
+                Z (Float) : 21.6144
             guid (CGUID) : 540f6017-4e09-4822-972e-9042cd9e0df3
             AttachmentsReference (array:handle:IAttachment) : # array:handle:IAttachment
             AttachmentsChild (array:handle:IAttachment) : # array:handle:IAttachment
