 1 file changed, 19 insertions(+), 6 deletions(-)

diff --git 1.32/levels/novigrad/baron_lair/baron_castle/baron_castle_interior_lights_nn/entity light other.w2l.txt 4.00/levels/novigrad/baron_lair/baron_castle/baron_castle_interior_lights_nn/entity light other.w2l.txt
index e853c11..cce7db6 100644
--- 1.32/levels/novigrad/baron_lair/baron_castle/baron_castle_interior_lights_nn/entity light other.w2l.txt	
+++ 4.00/levels/novigrad/baron_lair/baron_castle/baron_castle_interior_lights_nn/entity light other.w2l.txt	
@@ -20,7 +20,18 @@
             BufferV1 (CCompressedBuffer:SEntityBufferType1)
                 0 (SEntityBufferType1)
                     ComponentName (CName) : 
-            BufferV2 (CBufferUInt32:SEntityBufferType2) : # CBufferUInt32:SEntityBufferType2
+            BufferV2 (CBufferUInt32:SEntityBufferType2)
+                0 (SEntityBufferType2)
+                    componentName (CName) : CSpotLightComponent0
+                    sizeofdata (Uint32) : 34
+                    variables (CBufferUInt32:CVariantSizeTypeName)
+                        0 (Float) : 2.8000
+                        1 (Float) : 10.0000
+                1 (SEntityBufferType2)
+                    componentName (CName) : CPointLightComponent0
+                    sizeofdata (Uint32) : 22
+                    variables (CBufferUInt32:CVariantSizeTypeName)
+                        0 (Float) : 5.0000
             idTag (IdTag) :  5757e62a-f2f4-4d2d-9fea-c78c6a67672f
     CGameplayEntity #2 (CGameplayEntity) : 
         CGameplayEntity (CGameplayEntity)
@@ -41,10 +52,12 @@
             BufferV2 (CBufferUInt32:SEntityBufferType2)
                 0 (SEntityBufferType2)
                     componentName (CName) : CSpotLightComponent0
-                    sizeofdata (Uint32) : 55
+                    sizeofdata (Uint32) : 79
                     variables (CBufferUInt32:CVariantSizeTypeName)
                         0 (Float) : 2.7500
-                        1 (EngineTransform)
+                        1 (Float) : 10.0000
+                        2 (Float) : 1.0000
+                        3 (EngineTransform)
                             Pitch (Float) : 0.0000
                             Yaw (Float) : 90.0000
                             Roll (Float) : 0.0000
@@ -55,9 +68,9 @@
                     componentName (CName) : CPointLightComponent0
                     sizeofdata (Uint32) : 43
                     variables (CBufferUInt32:CVariantSizeTypeName)
-                        0 (Float) : 2.2000
+                        0 (Float) : 2.0000
                         1 (EngineTransform)
-                            X (Float) : -0.8000
+                            X (Float) : 0.0000
                             Y (Float) : 0.8000
-                            Z (Float) : 0.4728
+                            Z (Float) : 0.5000
             idTag (IdTag) :  fbd1091e-02e6-4af5-bc28-9725298d38bf
