 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git 1.32/levels/skellige/ard_skellig/location indicators/forest_ruins_dynamic.w2l.txt 4.00/levels/skellige/ard_skellig/location indicators/forest_ruins_dynamic.w2l.txt
index 4eb12de..eaa355f 100644
--- 1.32/levels/skellige/ard_skellig/location indicators/forest_ruins_dynamic.w2l.txt	
+++ 4.00/levels/skellige/ard_skellig/location indicators/forest_ruins_dynamic.w2l.txt	
@@ -210,7 +210,7 @@
                                 Z (Float) : 34.3143
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : 743.9097
+                                X (Float) : 743.9096
                                 Y (Float) : -148.6637
                                 Z (Float) : 37.8957
                                 W (Float) : 1.0000
@@ -585,7 +585,7 @@
                                 Z (Float) : 33.4003
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : 751.4641
+                                X (Float) : 751.4642
                                 Y (Float) : -146.2273
                                 Z (Float) : 36.0741
                                 W (Float) : 1.0000
@@ -630,7 +630,7 @@
                                 Z (Float) : 34.2513
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : 753.0372
+                                X (Float) : 753.0373
                                 Y (Float) : -147.4981
                                 Z (Float) : 36.6606
                                 W (Float) : 1.0000
@@ -670,7 +670,7 @@
                         graphPositionY (Int16) : 209
                         boundingBox (Box)
                             Min (Vector)
-                                X (Float) : 753.3900
+                                X (Float) : 753.3901
                                 Y (Float) : -152.6513
                                 Z (Float) : 32.9002
                                 W (Float) : 1.0000
@@ -1054,7 +1054,7 @@
                                 Z (Float) : 33.3958
                                 W (Float) : 1.0000
                             Max (Vector)
-                                X (Float) : 744.4351
+                                X (Float) : 744.4350
                                 Y (Float) : -150.2515
                                 Z (Float) : 35.0024
                                 W (Float) : 1.0000
@@ -1113,12 +1113,12 @@
                     box5 (Float) : 0.4540
                     pathHash (String) : "environment/decorations/housework/dishes/jugs/jug_red_a.w2mesh"
                 3 (CSectorDataResource)
-                    box0 (Float) : -0.1134
-                    box1 (Float) : -0.1117
-                    box2 (Float) : 0.0008
-                    box3 (Float) : 0.1101
-                    box4 (Float) : 0.1117
-                    box5 (Float) : 0.4004
+                    box0 (Float) : -0.1137
+                    box1 (Float) : -0.1131
+                    box2 (Float) : -0.0010
+                    box3 (Float) : 0.1119
+                    box4 (Float) : 0.1120
+                    box5 (Float) : 0.4005
                     pathHash (String) : "environment/decorations/housework/dishes/jugs/jug_green_a.w2mesh"
             Objects (CBufferVLQInt32:CSectorDataObject)
                 0 (CSectorDataObject)
@@ -1148,11 +1148,11 @@
                 3 (CSectorDataObject)
                     type (Uint8) : 7
                     flags (Uint8) : 0
-                    radius (Uint16) : 30
+                    radius (Uint16) : 60
                     offset (Uint64) : 236
-                    positionX (Float) : 753.9838
-                    positionY (Float) : -146.6099
-                    positionZ (Float) : 32.6300
+                    positionX (Float) : 753.9845
+                    positionY (Float) : -146.6110
+                    positionZ (Float) : 32.6298
             blocksize (CVLQInt32) : 332
             BlockData (CCompressedBuffer:SBlockData)
                 0 (SBlockData)
